QIIME[1] (an abbreviation for _Quantitative Insights Into Microbial Ecology_) is a bioinformatic pipeline designated for the task of analysing microbial communities that were sampled through marker gene (e.g. 16S or 18S rRNA genes) amplicon sequencing. In its heart the pipeline performs quality control over the input sequencing reads, clusters the marker gene nucleotide sequences at a requested phylogenetic level (e.g. 97% for species level) into OTUs (operational taxonomic units) or sequence variants and taxonomically annotates them by looking for similar sequences in a reference taxonomic database. The main output from the QIIME pipeline is a feature table, which describes the abundances of each OTU or sequence variant in each sample. Additional tools that are relevant to ecological aspects of the samples being investigated are also provided within the pipeline, including rarefaction, alpha diversity and beta diversity calculations, visualizations such as principle coordinates analysis(PCoA) and much more. QIIME presents a very modular approach and allows multiple methods to be applied in many stages of the analysis. For example, the stage of sequences clustering can be performed using either UCLUST, CD-HIT, BLAST and more. QIIME is under active development since its release in 2010.
