The ALPHA HELIX (Α-HELIX) is a common motif in the secondary structure of proteins and is a right hand-helix conformation in which every backbone N−H group donates a hydrogen bond to the backbone C=O group of the amino acid located three or four residues earlier along the protein sequence.

The alpha helix is also called a classic PAULING–COREY–BRANSON Α-HELIX. The name 3.6₁₃-HELIX is also used for this type of helix, denoting the average number of residues per helical turn, with 13 atoms being involved in the ring formed by the hydrogen bond.

Among types of local structure in proteins, the α-helix is the most regular and the most predictable from sequence, as well as the most prevalent.


Discovery

In the early 1930s, William Astbury showed that there were drastic changes in the X-ray fiber diffraction of moist wool or hair fibers upon significant stretching. The data suggested that the unstretched fibers had a coiled molecular structure with a characteristic repeat of ≈.

Astbury initially proposed a kinked-chain structure for the fibers. He later joined other researchers (notably the American chemist Maurice Huggins) in proposing that:

-   the unstretched protein molecules formed a helix (which he called the α-form)
-   the stretching caused the helix to uncoil, forming an extended state (which he called the β-form).

Although incorrect in their details, Astbury's models of these forms were correct in essence and correspond to modern elements of secondary structure, the α-helix and the β-strand (Astbury's nomenclature was kept), which were developed by Linus Pauling, Robert Corey and Herman Branson in 1951 (see below); that paper showed both right- and left-handed helices, although in 1960 the crystal structure of myoglobin[1] showed that the right-handed form is the common one. Hans Neurath was the first to show that Astbury's models could not be correct in detail, because they involved clashes of atoms.[2] Neurath's paper and Astbury's data inspired H. S. Taylor,[3] Maurice Huggins[4] and Bragg and collaborators[5] to propose models of keratin that somewhat resemble the modern α-helix.

Two key developments in the modeling of the modern α-helix were: the correct bond geometry, thanks to the crystal structure determinations of amino acids and peptides and Pauling's prediction of _planar_ peptide bonds; and his relinquishing of the assumption of an integral number of residues per turn of the helix. The pivotal moment came in the early spring of 1948, when Pauling caught a cold and went to bed. Being bored, he drew a polypeptide chain of roughly correct dimensions on a strip of paper and folded it into a helix, being careful to maintain the planar peptide bonds. After a few attempts, he produced a model with physically plausible hydrogen bonds. Pauling then worked with Corey and Branson to confirm his model before publication.[6] In 1954, Pauling was awarded his first Nobel Prize "for his research into the nature of the chemical bond and its application to the elucidation of the structure of complex substances"[7] (such as proteins), prominently including the structure of the α-helix.


Structure

Geometry and hydrogen bonding

The amino acids in an α-helix are arranged in a right-handed helical structure where each amino acid residue corresponds to a 100° turn in the helix (i.e., the helix has 3.6 residues per turn), and a translation of along the helical axis. Dunitz[8] describes how Pauling's first article on the theme in fact shows a left-handed helix, the enantiomer of the true structure. Short pieces of left-handed helix sometimes occur with a large content of achiral glycine amino acids, but are unfavorable for the other normal, biological [[amino_acids|

The α-helix is tightly packed; there is almost no free space within the helix. The amino-acid side-chains are on the outside of the helix, and point roughly "downward" (i.e., toward the N-terminus), like the branches of an evergreen tree (Christmas tree effect). This directionality is sometimes used in preliminary, low-resolution electron-density maps to determine the direction of the protein backbone.[9]

2-dimensional diagrams for representing α-helices

Three differently arranged styles of 2D diagrams are used to represent different aspects of the sequence and structure relationships that confer specific physical and interaction properties on individual α-helices. Two of these emphasize circular placement around the cylindrical cross-section: The first-developed such diagram is called the "helical wheel",[10] and a more recent version is called the "wenxiang diagram".[11] The latter name came from the fact that it resembles a coil-like incense used in China to repel mosquitos ().

The helical wheel represents a helix by a projection of the C^(α) backbone structure down the helix axis, while the wenxiang diagram represents it more abstractly as a smooth spiral coiled on the plane of the page. Both label the sequence with one-letter amino-acid code (see amino acid) at each C^(α) position, using different colors or symbols to code the amino-acid properties. Hydrophobic vs hydrophilic amino acids are always distinguished, as the most important property governing helix interactions. Sometimes positively vs negatively charged hydrophilics are distinguished, and sometimes ambiguous amino acids such as glycine (G) are distinguished. Color-coding conventions are various. The helical wheel does not change representation along the helix, while the wenxiang diagram is able to show the relative locations of the amino acids in an α-helix regardless of how long it is.

Either circular style of diagram can provide an intuitive and easily visualizable 2D picture that characterizes the disposition of hydrophobic and hydrophilic residues in α-helices,[12][13] and can be used to study helix–helix interactions,[14] helix-membrane interactions as quantified by the helical hydrophobic moment,[15] or protein-protein interactions.[16] [17] Various utilities and web sites are available to generate helical wheels, such as the page by Kael Fischer.[18]

The third style of 2D diagram is called a "helical net". It is generated by opening the cylindrical surface of each helix along a line parallel to the axis and laying the result out vertically. The helix net is not suitable for studying helix–helix packing interactions, but it has become the dominant means of representing the sequence arrangement for integral membrane proteins because it shows important relationships of the helical sequence to vertical positioning within the membrane even without knowledge of how the helices are arranged in 3D.

Stability

Helices observed in proteins can range from four to over forty residues long, but a typical helix contains about ten amino acids (about three turns). In general, short polypeptides do not exhibit much α-helical structure in solution, since the entropic cost associated with the folding of the polypeptide chain is not compensated for by a sufficient amount of stabilizing interactions. In general, the backbone hydrogen bonds of α-helices are considered slightly weaker than those found in β-sheets, and are readily attacked by the ambient water molecules. However, in more hydrophobic environments such as the plasma membrane, or in the presence of co-solvents such as trifluoroethanol (TFE), or isolated from solvent in the gas phase,[19] oligopeptides readily adopt stable α-helical structure. Furthermore, crosslinks can be incorporated into peptides to conformationally stabilize helical folds. Crosslinks stabilize the helical state by entropically destabilizing the unfolded state and by removing enthalpically stabilized "decoy" folds that compete with the fully helical state.[20] It has been shown that α-helices are more stable, robust to mutations and designable than β-strands in natural proteins,[21] and also in artificial designed proteins.[22]

Helix_electron_density_myoglobin_2nrl_17-32.jpg


Experimental determination

Since the α-helix is defined by its hydrogen bonds and backbone conformation, the most detailed experimental evidence for α-helical structure comes from atomic-resolution X-ray crystallography such as the example shown at right. It is clear that all the backbone carbonyl oxygens point downward (toward the C-terminus) but splay out slightly, and the H-bonds are approximately parallel to the helix axis. Protein structures from NMR spectroscopy also show helices well, with characteristic observations of nuclear Overhauser effect (NOE) couplings between atoms on adjacent helical turns. In some cases, the individual hydrogen bonds can be observed directly as a small scalar coupling in NMR.

There are several lower-resolution methods for assigning general helical structure. The NMR chemical shifts (in particular of the C^(α), C^(β) and C′) and residual dipolar couplings are often characteristic of helices. The far-UV (170–250 nm) circular dichroism spectrum of helices is also idiosyncratic, exhibiting a pronounced double minimum at around 208 and 222 nm. Infrared spectroscopy is rarely used, since the α-helical spectrum resembles that of a random coil (although these might be discerned by, e.g., hydrogen-deuterium exchange). Finally, cryo electron microscopy is now capable of discerning individual α-helices within a protein, although their assignment to residues is still an active area of research.

Long homopolymers of amino acids often form helices if soluble. Such long, isolated helices can also be detected by other methods, such as dielectric relaxation, flow birefringence, and measurements of the diffusion constant. In stricter terms, these methods detect only the characteristic prolate (long cigar-like) hydrodynamic shape of a helix, or its large dipole moment.


Amino-acid propensities

Different amino-acid sequences have different propensities for forming α-helical structure. Methionine, alanine, leucine, glutamate, and lysine uncharged ("MALEK" in the amino-acid 1-letter codes) all have especially high helix-forming propensities, whereas proline and glycine have poor helix-forming propensities.[23] Proline either breaks or kinks a helix, both because it cannot donate an amide hydrogen bond (having no amide hydrogen), and also because its sidechain interferes sterically with the backbone of the preceding turn inside a helix, this forces a bend of about 30° in the helix's axis.[24] However, proline is often seen as the _first_ residue of a helix, it is presumed due to its structural rigidity. At the other extreme, glycine also tends to disrupt helices because its high conformational flexibility makes it entropically expensive to adopt the relatively constrained α-helical structure.

Table of standard amino acid alpha-helical propensities

Estimated differences in free energy, Δ(Δ_G_), estimated in kcal/mol per residue in an α-helical configuration, relative to alanine arbitrarily set as zero. Higher numbers (more positive free energies) are less favoured. Significant deviations from these average numbers are possible, depending on the identities of the neighbouring residues.

    {| class="wikitable sortable"

|+Differences in free energy per residue[25] !rowspan=2| Amino acid !rowspan=2 class="unsortable"| 3-
letter !rowspan=2 class="unsortable"| 1-
letter !colspan=2| Helical penalty |- !kcal/mol !kJ/mol |- | Alanine | Ala | A | |- | Arginine | Arg | R | |- | Asparagine | Asn | N | |- | Aspartic acid | Asp | D | |- | Cysteine | Cys | C | |- | Glutamic acid | Glu | E | |- | Glutamine | Gln | Q | |- | Glycine | Gly | G | |- | Histidine | His | H | |- | Isoleucine | Ile | I | |- | Leucine | Leu | L | |- | Lysine | Lys | K | |- | Methionine | Met | M | |- | Phenylalanine | Phe | F | |- | Proline | Pro | P | |- | Serine | Ser | S | |- | Threonine | Thr | T | |- | Tryptophan | Trp | W | |- | Tyrosine | Tyr | Y | |- | Valine | Val | V | |}


Dipole moment

A helix has an overall dipole moment due to the aggregate effect of the individual microdipoles from the carbonyl groups of the peptide bond pointing along the helix axis.[26] The effects of this macrodipole are a matter of some controversy. α-helices often occur with the N-terminal end bound by a negatively charged group, sometimes an amino acid side chain such as glutamate or aspartate, or sometimes a phosphate ion. Some regard the helix macrodipole as interacting electrostatically with such groups. Others feel that this is misleading and it is more realistic to say that the hydrogen bond potential of the free NH groups at the N-terminus of an α-helix can be satisfied by hydrogen bonding; this can also be regarded as set of interactions between local microdipoles such as .[27][28]


Coiled coils

Coiled-coil α helices are highly stable forms in which two or more helices wrap around each other in a "supercoil" structure. Coiled coils contain a highly characteristic sequence motif known as a HEPTAD REPEAT, in which the motif repeats itself every seven residues along the sequence (_amino acid_ residues, not DNA base-pairs). The first and especially the fourth residues (known as the _a_ and _d_ positions) are almost always hydrophobic; the fourth residue is typically leucine this gives rise to the name of the structural motif called a _leucine zipper_, which is a type of coiled-coil. These hydrophobic residues pack together in the interior of the helix bundle. In general, the fifth and seventh residues (the _e_ and _g_ positions) have opposing charges and form a salt bridge stabilized by electrostatic interactions. Fibrous proteins such as keratin or the "stalks" of myosin or kinesin often adopt coiled-coil structures, as do several dimerizing proteins. A pair of coiled-coils a four-helix bundle is a very common structural motif in proteins. For example, it occurs in human growth hormone and several varieties of cytochrome. The Rop protein, which promotes plasmid replication in bacteria, is an interesting case in which a single polypeptide forms a coiled-coil and two monomers assemble to form a four-helix bundle.


Facial Arrangements

The amino acids that make up a particular helix can be plotted on a helical wheel, a representation that illustrates the orientations of the constituent amino acids (see the article for leucine zipper for such a diagram). Often in globular proteins, as well as in specialized structures such as coiled-coils and leucine zippers, an α-helix will exhibit two "faces" one containing predominantly hydrophobic amino acids oriented toward the interior of the protein, in the hydrophobic core, and one containing predominantly polar amino acids oriented toward the solvent-exposed surface of the protein.

Changes in binding orientation also occur for facially-organized oligopeptides. This pattern is especially common in antimicrobial peptides, and many models have been devised to describe how this relates to their function. Common to many of them is that the hydrophobic face of the antimicrobial peptide forms pores in the plasma membrane after associating with the fatty chains at the membrane core.[29][30]


Larger-scale assemblies

1GZX_Haemoglobin.png molecule has four heme-binding subunits, each made largely of α-helices.]] Myoglobin and hemoglobin, the first two proteins whose structures were solved by X-ray crystallography, have very similar folds made up of about 70% α-helix, with the rest being non-repetitive regions, or "loops" that connect the helices. In classifying proteins by their dominant fold, the Structural Classification of Proteins database maintains a large category specifically for all-α proteins.

Hemoglobin then has an even larger-scale quaternary structure, in which the functional oxygen-binding molecule is made up of four subunits.


Functional roles

Coiled-coil_TF_Max_on_DNA.jpg: transcription factor Max (PDB file 1HLO)]] 1gzm_opm.png file 1GZM), with a bundle of seven helices crossing the membrane (membrane surfaces marked by horizontal lines)]]

DNA binding

α-Helices have particular significance in DNA binding motifs, including helix-turn-helix motifs, leucine zipper motifs and zinc finger motifs. This is because of the convenient structural fact that the diameter of an α-helix is about including an average set of sidechains, about the same as the width of the major groove in B-form DNA, and also because coiled-coil (or leucine zipper) dimers of helices can readily position a pair of interaction surfaces to contact the sort of symmetrical repeat common in double-helical DNA.[31] An example of both aspects is the transcription factor Max (see image at left), which uses a helical coiled coil to dimerize, positioning another pair of helices for interaction in two successive turns of the DNA major groove.

Membrane spanning

α-Helices are also the most common protein structure element that crosses biological membranes (transmembrane protein),[32] it is presumed because the helical structure can satisfy all backbone hydrogen-bonds internally, leaving no polar groups exposed to the membrane if the sidechains are hydrophobic. Proteins are sometimes anchored by a single membrane-spanning helix, sometimes by a pair, and sometimes by a helix bundle, most classically consisting of seven helices arranged up-and-down in a ring such as for rhodopsins (see image at right) or for G protein–coupled receptors (GPCRs).

Mechanical properties

α-Helices under axial tensile deformation, a characteristic loading condition that appears in many alpha-helix-rich filaments and tissues, results in a characteristic three-phase behavior of stiff-soft-stiff tangent modulus.[33] Phase I corresponds to the small-deformation regime during which the helix is stretched homogeneously, followed by phase II, in which alpha-helical turns break mediated by the rupture of groups of H-bonds. Phase III is typically associated with large-deformation covalent bond stretching.


Dynamical features

Alpha-helices in proteins may have low-frequency accordion-like motion as observed by the Raman spectroscopy[34] and analyzed via the quasi-continuum model.[35][36] Helices not stabilized by tertiary interactions show dynamic behavior, which can be mainly attributed to helix fraying from the ends.[37]


Helix–coil transition

Homopolymers of amino acids (such as polylysine) can adopt α-helical structure at low temperature that is "melted out" at high temperatures. This HELIX–COIL TRANSITION was once thought to be analogous to protein denaturation. The statistical mechanics of this transition can be modeled using an elegant transfer matrix method, characterized by two parameters: the propensity to initiate a helix and the propensity to extend a helix.


In art

AlphaHelixForLinusPauling.jpg's _Alpha Helix for Linus Pauling_ (2004), powder coated steel, height . The sculpture stands in front of Pauling's childhood home on 3945 SE Hawthorne Boulevard in Portland, Oregon, USA.]]

At least five artists have made explicit reference to the α-helix in their work: Julie Newdoll in painting and Julian Voss-Andreae, Bathsheba Grossman, Byron Rubin, and Mike Tyka in sculpture.

San Francisco area artist Julie Newdoll,[38] who holds a degree in Microbiology with a minor in art, has specialized in paintings inspired by microscopic images and molecules since 1990. Her painting "Rise of the Alpha Helix" (2003) features human figures arranged in an α helical arrangement. According to the artist, "the flowers reflect the various types of sidechains that each amino acid holds out to the world".[39] This same metaphor is also echoed from the scientist's side: "β sheets do not show a stiff repetitious regularity but flow in graceful, twisting curves, and even the α-helix is regular more in the manner of a flower stem, whose branching nodes show the influence of environment, developmental history, and the evolution of each part to match its own idiosyncratic function."[40]

Julian Voss-Andreae is a German-born sculptor with degrees in experimental physics and sculpture. Since 2001 Voss-Andreae creates "protein sculptures"[41] based on protein structure with the α-helix being one of his preferred objects. Voss-Andreae has made α-helix sculptures from diverse materials including bamboo and whole trees. A monument Voss-Andreae created in 2004 to celebrate the memory of Linus Pauling, the discoverer of the α-helix, is fashioned from a large steel beam rearranged in the structure of the α-helix. The , bright-red sculpture stands in front of Pauling's childhood home in Portland, Oregon.

Ribbon diagrams of α-helices are a prominent element in the laser-etched crystal sculptures of protein structures created by artist Bathsheba Grossman, such as those of insulin, hemoglobin, and DNA polymerase.[42] Byron Rubin is a former protein crystallographer now professional sculptor in metal of proteins, nucleic acids, and drug molecules many of which featuring α-helices, such as subtilisin, human growth hormone, and phospholipase A2.[43]

Mike Tyka is a computational biochemist at the University of Washington working with David Baker. Tyka has been making sculptures of protein molecules since 2010 from copper and steel, including ubiquitin and a potassium channel tetramer.[44]


See also

-   3₁₀ helix
-   Pi helix
-   Beta sheet
-   Davydov soliton
-   Folding (chemistry)
-   Knobs into holes packing
-


References


Further reading

-   .

-   -   -   -   -   -   -   -   -   -   -   -   -


External links

-   NetSurfP ver. 1.1 - Protein Surface Accessibility and Secondary Structure Predictions
-   α-helix rotational angle calculator
-   Artist Julie Newdoll's website
-   Artist Julian Voss-Andreae's website

Category:Protein structural motifs Category:Helices

[1]

[2]

[3]

[4]

[5]

[6]

[7] 1

[8]

[9]

[10]

[11]

[12]

[13]

[14]

[15]

[16]

[17]

[18] 2 .

[19]

[20]

[21]

[22]

[23]

[24]

[25]

[26]

[27]

[28]

[29]

[30]

[31] Branden & Tooze, chapter 10

[32] Branden & Tooze, chapter 12.

[33]

[34]

[35]

[36]

[37]

[38]

[39]

[40]

[41]

[42]

[43]

[44]