ASPARAGINE PEPTIDE LYASE are one of the seven groups in which proteases, also termed proteolytic enzymes, peptidases, or proteinases, are classified according to their catalytic residue. The catalytic mechanism of the asparagine peptide lyases involves an asparagine residue acting as nucleophile to perform a nucleophilic elimination reaction, rather than hydrolysis, to catalyse the breaking of a peptide bond.[1]

The existence of this seventh catalytic type of proteases, in which the peptide bond cleavage occurs by self-processing instead of hydrolysis, was demonstrated with the discovery of the crystal structure of the self-cleaving precursor of the Tsh autotransporter from _E. coli_.[2]


Synthesis

General_asparagine_self-cleaving_mechanism_in_asparagine_peptide_lyases.jpg These enzymes are synthesized as precursors or propeptides, which cleave themselves by an autoproteolytic reaction.[3]

The self-cleaving nature of asparagine peptide lyases contradicts the general definition of an enzyme given that the enzymatic activity destroys the enzyme. However, the self-processing is the action of a proteolytic enzyme, notwithstanding the enzyme is not recoverable from the reaction.[4]


Active site and catalytic mechanism

All the proteolytic activity of the asparagine peptide lyases is only self-cleavages, then no further peptidase activity occurs.[5]

The main residue of the active site is the asparagine and there are other residues involved in the catalytic mechanism, which are different between the different families of asparagine peptide lyases.[6][7][8]

The cleavage mechanism consists in the cyclization of the asparagine, assisted by other active site residues. In certain conditions, the asparagine cyclic structure nucleophilically attacks its C-terminal peptide bond to the main chain forming a new bond to create a stable succinimide, cleaving itself from the main chain and consequently releasing the two halves of the product.[9][10]


Inhibition

No inhibitors are known.[11]


Classification

The MEROPS protease database includes the following ten families of asparagine peptide lyases, which are included in 6 different clans of proteases.[12]

Proteolytic enzymes are classified into families based on sequence similarity. Each family includes proteolytic enzymes with homologous sequences and common catalytic type. Clans are groups of proteolytic enzymes families with related structures, where catalytic type is not conserved.

  ------ -------- ------------------------- ------------------------------------------------------------------------------- ----------- ------------
  CLAN   FAMILY   MEROPS ID                 PEPTIDASES AND HOMOLOGUES                                                       NC-IUBMB    PDB ID
  NA     N1       N01.001                   nodavirus coat protein                                                          3.4.23.44   2BBV
                  unassigned                family N1 unassigned peptide lyases                                             *           -
         N2       N02.001                   tetravirus coat protein                                                         *           1OHF
                  non-peptidase homologue   family N2 non-lyase homologues                                                  *           -
                  unassigned                family N2 unassigned peptide lyases                                             *           -
         N8       N08.001                   picornavirus capsid VP0-type self-cleaving protein                              *           1NCQ
                  non-peptidase homologue   family N8 non-lyase homologues                                                  *           -
                  unassigned                family N8 unassigned peptide lyases                                             *           -
  NB     N6       N06.001                   YscU protein (_Yersinia pseudotuberculosis_)                                    *           2JLJ
                  N06.002                   SpaS protein (_Salmonella_ sp.)                                                 *           3C01; 2VT1
                  N06.003                   EscU protein (_Escherichia coli_)                                               *           3BZO
                  N06.004                   HrcU protein (_Xanthomonas_ sp.)                                                *           -
                  N06.A01                   FlhB protein (_Escherichia coli_)                                               *           -
                  non-peptidase homologue   family N6 non-lyase homologues                                                  *           -
                  unassigned                family N6 unassigned peptide lyases                                             *           -
  NC     N7       N07.001                   reovirus type 1 coat protein                                                    *           1JMU
                  N07.002                   aquareovirus coat protein                                                       *           -
                  unassigned                family N7 unassigned peptide lyases                                             *           -
  ND     N4       N04.001                   Tsh-associated self-cleaving domain (_Escherichia coli_) and similar            *           3AEH
                  N04.002                   EspP gamma protein autotransporter domain (_Escherichi_-type)                   *           2QOM
                  non-peptidase homologue   family N4 non-lyase homologues                                                  *           -
                  unassigned                family N4 unassigned peptide lyases                                             *           -
  NE     N5       N05.001                   picobirnavirus self-cleaving protein                                            *           2VF1
                  unassigned                family N5 unassigned peptide lyases                                             *           -
  PD     N9       N09.001                   intein-containing V-type proton ATPase catalytic subunit A                      3.6.3.14    1VDE
                  non-peptidase homologue   family N9 non-lyase homologues                                                  *           -
                  unassigned                family N9 unassigned peptide lyases                                             *           -
         N10      N10.001                   intein-containing DNA gyrase subunit A precursor                                *           -
                  N10.002                   intein-containing replicative DNA helicase precursor                            *           1MI8
                  N10.003                   intein-containing DNA polymerase III subunit alpha precursor                    2.7.7.7     2KEQ
                  N10.004                   intein-containing translation initiation factor IF-2 precursor                              -
                  N10.005                   intein-containing DNA polymerase II large subunit DP2 precursor Mername-AA281   *           -
                  N10.006                   intein-containing DNA polymerase II large subunit DP2 precursor Mername-AA282   2.7.7.7     -
                  N10.007                   intein-containing DNA-dependent DNA polymerase precursor                        *           2CW7; 2CW8
                  N10.008                   intein-containing DNA gyrase subunit A (_Mycobacterium xenopi_)                 *           1AM2; 4OZ6
                  N10.009                   Mtu recA intein (_Mycobacterium_ sp.)                                           *           2IN9
                  non-peptidase homologue   family N10 non-lyase homologues                                                 *           -
                  unassigned                family N10 unassigned peptide lyases                                            *           -
         N11      N11.001                   intein-containing chloroplast ATP-dependent peptide lyase                       *           -
                  non-peptidase homologue   family N11 non-lyase homologues                                                 *           -
                  unassigned                family N11 unassigned peptide lyases                                            *           -
  ------ -------- ------------------------- ------------------------------------------------------------------------------- ----------- ------------

*Not yet included in IUBMB recommendations.


Distribution and types

The ten different families of asparagine peptide lyases are distributed in three different types:

-   Viral coat proteins
-   Autotransporter proteins
-   Intein-containing proteins

There are five families of viral coat proteins (N1, N2, N8, N7 and N5), two families of autotransporter proteins (N6 and N4) and three families of intein-containing proteins (N9, N10 and N11).

Viral coat proteins

There are five families of viral coat proteins in which processing occurs at an asparagine residue. These five families are included in three clans: Clan NA (Families N1, N2 and N8), clan NC (Family N7) and clan NE (Family N5).[13]

Family N1: The known autolytic cleavage is mediated by the nodavirus endopeptidase, from the C-terminus of the coat protein and only occurs within the assembled virion.Reddy, A., Schneemann, A. & Johnson, J.E.

Nodavirus endopeptidase.

In Handbook of Proteolytic Enzymes, 2 edn (Barrett, A.J., Rawlings, N.D. & Woessner, J.F. eds), p.197-201, Elsevier, London (2004)

Family N2: Includes tetraviruses endopeptidases. The known autolytic cleavage is from the C-terminus of the coat protein. The cleavage occurs during the late stages of virion assembly.Taylor, D.J. & Johnson, J.E.

Folding and particle assembly are disrupted by single-point mutations near the autocatalytic cleavage site of Nudaurelia capensis omega virus capsid protein.

Protein Sci (2005) 14, 401-408

Family N8: The known autolytic cleavage is in poliovirus VP0 viral capsid protein into VP2 and Vp4 in the provirion.[14]

Family N7: The known autolytic cleavage is from the N-terminus of the coat protein.[15]

Family N5: The known autolytic cleavage is from the N-terminus of the coat protein.[16]

Autotransporter proteins

Tsh-associated_self-cleaving_domain_(Escherichia_coli)_and_similar.jpg Autotransporter proteins are outer membrane or secreted proteins found in a broad variety of Gram-negative bacteria. These proteins contain three structural motifs: a signal sequence, a passenger domain located at the N-terminal, and a translocator or autotransporter domain located at the C-terminal, forming a beta barrel structure. These structures promote the protein self-transport. Autotransporter proteins are usually related to virulence functions. This fact, their interaction with host cells and the broad occurrence of autotransporter encoding genes, bring up the possibility to represent therapeutic targets for the design of vaccines against Gram-negative pathogens.[17]

Two of the families in which the MEROPS database classifies asparagine peptide lyases are autotransporter proteins, families N4 and N6.[18]

Family N4 includes secreted virulence factors, or autotransporters, from enterobacteria. Their only proteolytic activity is releasing the virulence factor from the precursor, enabling it to be secreted. The active site residues in family N4 asparagine peptide lyases are N1100, Y1227, E1249 and R1282.

Family N6 includes autoprocessing endopeptidases involved in type III protein secretion system, in which autoproteolysis is essential for mediating the secretion of proteins. Type III secretion system secretes proteins directly into host cells by an injectisome, a hollow tubular structure that penetrates into the host cell. Secreted proteins can pass through the injectisome into the host cell cytoplasm. The conserved active site residue in family N6 asparagine peptide lyases is N263.

Intein-containing proteins

An intein is a protein contained within another protein, the extein. Parasitic DNA infects an intein gene, which encodes an endonuclease. The resulting cDNA (complementary DNA) encodes the extein along with the intein. The intein contains a self-cleaving domain, which has the endonuclease nested within it. The intein domain performs two proteolytic cleavages at its own N-terminus and C-terminus and releases from the extein, separating it in two fragments. This two fragments are then spliced together and the extein remains as a completely functional protein.

The N-terminal residue of the intein domain must be a serine, threonine or cysteine, and it attacks its preceding peptide bond in order to form an ester or a thioester. The first residue of the second portion of the extein must be a serine, threonine or cysteine as well, and this second nucleophile forms a branched intermediary. The C-terminal residue of the intein domain is always an asparagine, which cyclizes to form a succinimide, cleaving its own peptide bond and releasing the intein from the extein. Finally, in the extein the ester or thioester bond is rearranged to form a normal peptide bond.[19]

There are three known families of intein-containing proteins (N9, N10 and N11) all of them included in the PD clan, which contains proteolytic enzymes of different catalytic types. The tertiary structure has been solved for the intein V type proton ATPase catalytic subunit (_Saccharomyces cerevisiae_), a member of family N9 and for several inteins from family N10.


See also

-   Asparagine
-   Lyase
-   Protein precursor
-   Proteolysis
-   Nucleophilic substitution
-   Protease
    -   cysteine-
    -   serine-
    -   threonine-
    -   aspartic-
    -   glutamic-
    -   metallo-


References


Further reading

-   Rawlings ND, Barrett AJ, Bateman A. _Asparagine peptide lyases: a seventh catalytic type of proteolytic enzymes_. 2011 Nov 4;286(44):38321-8.
-   Alan J. Barrett, Neil D. Rawlings, J. Fred (2012). _Handbook of Proteolytic Enzymes_. Third edition.
-   Guoyao Wu (2013) _Amino Acids: Biochemistry and Nutrition_.
-   Klaudia Brix, Walter Stöcker (Jan 21, 2014). _Proteases: Structure and Function_.
-   Jin Zhang, Sohum Mehta, Carsten Schultz (2016). _Optical Probes in Biology_.


External links

-   International Proteolysis Society
-   Protease cut sites graphical interface
-   Merops - the peptidase database
-   The Proteolysis Map
-   TopFIND protease database covering cut sites, substrates and protein termini
-   List of proteases and their specificities (see also 1)
-

Category:Enzymes Peptidase

[1]

[2]

[3] Tajima, N., Kawai, F., Park, S. Y., Tame, J. R. A novel intein-like autoproteolytic mechanism in autotransporter proteins. (2010) _J. Mol. Biol_. 402(4), 645-656 PMID 20615416

[4] Rawlings ND, Barret AJ, Bateman A. Asparagine peptide lyases: a seventh catalytic type of proteolytic enzymes. (2011 Nov 4) _J Biol Chem._ 286(44), 38321-8 PMID 21832066

[5]

[6]

[7] Dautin, N., Barnard, T. J., Anderson, D. E., and Bernstein, H. D. (2007) EMBO J. 26, 1942-1952

[8] J. March, Advanced Organic Chemistry, 4th ed., Wiley, New York, 1992

[9] Dehart, M. P., and Anderson, B. D. (2007) J. Pharm. Sci. 96, 2667-2685

[10] R. A. Rossi, R. H. de Rossi, Aromatic Substitution by the SRN1 Mechanism, ACS Monograph Series No. 178, American Chemical Society, 1983

[11]

[12]

[13]

[14]

[15]

[16]

[17] Wells TJ, Tree JJ, Ulett GC, Schembri MA. Autotransporter proteins: novel targets at the bacterial cell surface. (2007) 274(2), 163-72

[18]

[19] Alan J. Barrett, Neil D. Rawlings, J. Fred Woessner. Handbook of Proteolytic Enzymes. Third edition. (2013) (pp. 14-16)