PhageExterior.svg bacteriophage]] 11_Hegasy_Phage_T4_Wiki_E_CCBYSA.png

A BACTERIOPHAGE (), also known informally as a _PHAGE_ (), is a virus that infects and replicates within bacteria and archaea. The term was derived from "bacteria" and the Greek φαγεῖν (__), "to devour". Bacteriophages are composed of proteins that encapsulate a DNA or RNA genome, and may have structures that are either simple or elaborate. Their genomes may encode as few as four genes and as many as hundreds of genes. Phages replicate within the bacterium following the injection of their genome into its cytoplasm.

Bacteriophages are among the most common and diverse entities in the biosphere.[1] Bacteriophages are ubiquitous viruses, found wherever bacteria exist. It is estimated there are more than 10³¹ bacteriophages on the planet, more than every other organism on Earth, including bacteria, combined.[2] One of the densest natural sources for phages and other viruses is seawater, where up to 9x10⁸ virions per millilitre have been found in microbial mats at the surface,[3] and up to 70% of marine bacteria may be infected by phages.[4]

Phages have been used for more than 90 years as an alternative to antibiotics in the former Soviet Union and Central Europe, as well as in France.[5] They are seen as a possible therapy against multi-drug-resistant strains of many bacteria (see phage therapy).[6] Phages of _Inoviridae_ have been shown to complicate biofilms involved in pneumonia and cystic fibrosis and to shelter the bacteria from drugs meant to eradicate disease, thus promoting persistent infection.[7]


Classification

Bacteriophages occur abundantly in the biosphere, with different genomes, and lifestyles. Phages are classified by the International Committee on Taxonomy of Viruses (ICTV) according to morphology and nucleic acid.

Nineteen families are currently recognized by the ICTV that infect bacteria and archaea. Of these, only two families have RNA genomes, and only five families are surrounded by an envelope. Of the viral families with DNA genomes, only two have single-stranded genomes. Eight of the viral families with DNA genomes have circular genomes, while nine have linear genomes. Nine families infect bacteria only, nine infect archaea only, and one (_Tectiviridae_) infects both bacteria and archaea.

Bacteriophage_P22_Casjens_Lenk.png'' by morphology due to its short, non-contractile tail]]

  Order                  Family                                      Morphology                                        Nucleic acid                               Examples
  ---------------------- ------------------------------------------- ------------------------------------------------- ------------------------------------------ -------------------------------
  _Caudovirales_         _Ackermannviridae_                          Nonenveloped, contractile tail                    Linear dsDNA
  _Myoviridae_           Nonenveloped, contractile tail              Linear dsDNA                                      T4, Mu, P1, P2
  _Siphoviridae_         Nonenveloped, noncontractile tail (long)    Linear dsDNA                                      λ, T5, HK97, N15
  _Podoviridae_          Nonenveloped, noncontractile tail (short)   Linear dsDNA                                      T7, T3, Φ29, P22
  _Ligamenvirales_       _Lipothrixviridae_                          Enveloped, rod-shaped                             Linear dsDNA                               Acidianus filamentous virus 1
  _Rudiviridae_          Nonenveloped, rod-shaped                    Linear dsDNA                                      Sulfolobus islandicus rod-shaped virus 1
  Unassigned             _Ampullaviridae_                            Enveloped, bottle-shaped                          Linear dsDNA
  _Bicaudaviridae_       Nonenveloped, lemon-shaped                  Circular dsDNA
  _Clavaviridae_         Nonenveloped, rod-shaped                    Circular dsDNA
  _Corticoviridae_       Nonenveloped, isometric                     Circular dsDNA                                    PM2
  _Cystoviridae_         Enveloped, spherical                        Segmented dsRNA
  _Fuselloviridae_       Nonenveloped, lemon-shaped                  Circular dsDNA
  _Globuloviridae_       Enveloped, isometric                        Linear dsDNA
  _Guttaviridae_         Nonenveloped, ovoid                         Circular dsDNA
  _Inoviridae_           Nonenveloped, filamentous                   Circular ssDNA                                    M13
  _Leviviridae_          Nonenveloped, isometric                     Linear ssRNA                                      MS2, Qβ
  _Microviridae_         Nonenveloped, isometric                     Circular ssDNA                                    ΦX174
  _Plasmaviridae_        Enveloped, pleomorphic                      Circular dsDNA
  _Pleolipoviridae_      Enveloped, pleomorphic                      Circular ssDNA, circular dsDNA, or linear dsDNA
  _Portogloboviridae_    Enveloped, isometric                        Circular dsDNA
  _Sphaerolipoviridae_   Enveloped, isometric                        Linear dsDNA
  _Spiraviridae_         Nonnveloped, rod-shaped                     Circular ssDNA
  _Tectiviridae_         Nonenveloped, isometric                     Linear dsDNA
  _Tristromaviridae_     Enveloped, rod-shaped                       Linear dsDNA
  _Turriviridae_         Enveloped, isometric                        Linear dsDNA

  : ICTV CLASSIFICATION OF PROKARYOTIC (BACTERIAL AND ARCHAEAL) VIRUSES[8]

It has been suggested that members of _Picobirnaviridae_ infect bacteria, but not mammals.[9]


History

In 1896, Ernest Hanbury Hankin reported that something in the waters of the Ganges and Yamuna rivers in India had a marked antibacterial action against cholera and it could pass through a very fine porcelain filter.[10] In 1915, British bacteriologist Frederick Twort, superintendent of the Brown Institution of London, discovered a small agent that infected and killed bacteria. He believed the agent must be one of the following:

1.  a stage in the life cycle of the bacteria
2.  an enzyme produced by the bacteria themselves, or
3.  a virus that grew on and destroyed the bacteria [11]

Twort's research was interrupted by the onset of World War I and a shortage of funding.

Félix_d'Hérelle.jpg]] Independently, French-Canadian microbiologist Félix d'Hérelle, working at the Pasteur Institute in Paris, announced on 3 September 1917, that he had discovered "an invisible, antagonistic microbe of the dysentery bacillus". For d’Hérelle, there was no question as to the nature of his discovery: "In a flash I had understood: what caused my clear spots was in fact an invisible microbe … a virus parasitic on bacteria."[12] D'Hérelle called the virus a bacteriophage, a bacteria-eater (from the Greek __ meaning "to devour"). He also recorded a dramatic account of a man suffering from dysentery who was restored to good health by the bacteriophages.[13] It was D'Herelle who conducted much research into bacteriophages and introduced the concept of phage therapy.[14]

More than a half a century later, in 1969, Max Delbrück, Alfred Hershey, and Salvador Luria were awarded the Nobel Prize in Physiology or Medicine for their discoveries of the replication of viruses and their genetic structure.[15]


Uses

Phage therapy

Phages were discovered to be antibacterial agents and were used in the former Soviet Republic of Georgia (pioneered there by Giorgi Eliava with help from the co-discoverer of bacteriophages, Félix d'Herelle) during the 1920s and 1930s for treating bacterial infections. They had widespread use, including treatment of soldiers in the Red Army. However, they were abandoned for general use in the West for several reasons:

-   Antibiotics were discovered and marketed widely. They were easier to make, store, and to prescribe.
-   Medical trials of phages were carried out, but a basic lack of understanding raised questions about the validity of these trials.[16]
-   Publication of research in the Soviet Union was mainly in the Russian or Georgian languages and for many years, was not followed internationally.

The use of phages has continued since the end of the Cold War in Georgia and elsewhere in Central and Eastern Europe. The first regulated, randomized, double-blind clinical trial was reported in the Journal of Wound Care in June 2009, which evaluated the safety and efficacy of a bacteriophage cocktail to treat infected venous ulcers of the leg in human patients.[17] The FDA approved the study as a Phase I clinical trial. The study's results demonstrated the safety of therapeutic application of bacteriophages, but did not show efficacy. The authors explained that the use of certain chemicals that are part of standard wound care (e.g. lactoferrin or silver) may have interfered with bacteriophage viability.[18] Shortly after that, another controlled clinical trial in Western Europe (treatment of ear infections caused by _Pseudomonas aeruginosa_) was reported in the journal, Clinical Otolaryngology in August 2009.[19] The study concludes that bacteriophage preparations were safe and effective for treatment of chronic ear infections in humans. Additionally, there have been numerous animal and other experimental clinical trials evaluating the efficacy of bacteriophages for various diseases, such as infected burns and wounds, and cystic fibrosis associated lung infections, among others.[20]

Meanwhile, bacteriophage researchers have been developing engineered viruses to overcome antibiotic resistance, and engineering the phage genes responsible for coding enzymes that degrade the biofilm matrix, phage structural proteins, and the enzymes responsible for lysis of the bacterial cell wall.[21][22][23] There have been results showing that T4 phages that are small in size and short-tailed, can be helpful in detecting _E.coli_ in the human body.[24]

Therapeutic efficacy of a phage cocktail was evaluated in a mice model with nasal infection of multidrug-resistant (MDR) _A. baumannii_. Mice treated with the phage cocktail showed a 2.3-fold higher survival rate than those untreated in seven days post infection.[25] In 2017 a patient with a pancreas compromised by MDR _A. baumannii_ was put on several antibiotics, despite this the patient’s health continued to deteriorate during a four-month period. Without effective antibiotics the patient was subjected to phage therapy using a phage cocktail containing nine different phages that had been demonstrated to be effective against MDR _A. baumannii_. Once on this therapy the patient’s downward clinical trajectory reversed, and returned to health.[26]

D'Herelle "quickly learned that bacteriophages are found wherever bacteria thrive: in sewers, in rivers that catch waste runoff from pipes, and in the stools of convalescent patients."[27] This includes rivers traditionally thought to have healing powers, including India's Ganges River.[28]

Other

FOOD INDUSTRY - Since 2006, the United States Food and Drug Administration (FDA) and United States Department of Agriculture (USDA) have approved several bacteriophage products. LMP-102 (Intralytix) was approved for treating ready-to-eat (RTE) poultry and meat products. In that same year, the FDA approved LISTEX (developed and produced by Micreos) using bacteriophages on cheese to kill _Listeria monocytogenes_ bacteria, in order to give them generally recognized as safe (GRAS) status.[29] In July 2007, the same bacteriophage were approved for use on all food products.[30] In 2011 USDA confirmed that LISTEX is a clean label processing aid and is included in USDA.[31] Research in the field of food safety is continuing to see if lytic phages are a viable option to control other food-borne pathogens in various food products.

Dairy industry - Bacteriophages present in the environment can cause fermentation failures of cheese starter cultures. In order to avoid this, mixed-strain starter cultures and culture rotation regimes can be used.[32]

DIAGNOSTICS - In 2011, the FDA cleared the first bacteriophage-based product for in vitro diagnostic use.[33] The KeyPath MRSA/MSSA Blood Culture Test uses a cocktail of bacteriophage to detect _Staphylococcus aureus_ in positive blood cultures and determine methicillin resistance or susceptibility. The test returns results in about five hours, compared to two to three days for standard microbial identification and susceptibility test methods. It was the first accelerated antibiotic-susceptibility test approved by the FDA.[34]

COUNTERACTING BIOWEAPONS AND TOXINS - Government agencies in the West have for several years been looking to Georgia and the former Soviet Union for help with exploiting phages for counteracting bioweapons and toxins, such as anthrax and botulism.[35] Developments are continuing among research groups in the U.S. Other uses include spray application in horticulture for protecting plants and vegetable produce from decay and the spread of bacterial disease. Other applications for bacteriophages are as biocides for environmental surfaces, e.g., in hospitals, and as preventative treatments for catheters and medical devices before use in clinical settings. The technology for phages to be applied to dry surfaces, e.g., uniforms, curtains, or even sutures for surgery now exists. Clinical trials reported in _Clinical Otolaryngology_[36] show success in veterinary treatment of pet dogs with otitis.

The SEPTIC bacterium sensing and identification method uses the ion emission and its dynamics during phage infection and offers high specificity and speed for detection.[37]

PHAGE DISPLAY is a different use of phages involving a library of phages with a variable peptide linked to a surface protein. Each phage genome encodes the variant of the protein displayed on its surface (hence the name), providing a link between the peptide variant and its encoding gene. Variant phages from the library may be selected through their binding affinity to an immobilized molecule (e.g., botulism toxin) to neutralize it. The bound, selected phages can be multiplied by reinfecting a susceptible bacterial strain, thus allowing them to retrieve the peptides encoded in them for further study.[38]

ANTIMICROBIAL DRUG DISCOVERY - Phage proteins often have antimicrobial activity and may serve as leads for peptidomimetics, i.e. drugs that mimic peptides.[39] Phage-ligand technology makes use of phage proteins for various applications, such as binding of bacteria and bacterial components (e.g. endotoxin) and lysis of bacteria.[40]

BASIC RESEARCH - Bacteriophages are important model organisms for studying principles of evolution and ecology.[41]


Replication

Phage_injection.png Bacteriophages may have a lytic cycle or a lysogenic cycle, and a few viruses are capable of carrying out both. With _lytic phages_ such as the T4 phage, bacterial cells are broken open (lysed) and destroyed after immediate replication of the virion. As soon as the cell is destroyed, the phage progeny can find new hosts to infect. Lytic phages are more suitable for phage therapy. Some lytic phages undergo a phenomenon known as lysis inhibition, where completed phage progeny will not immediately lyse out of the cell if extracellular phage concentrations are high. This mechanism is not identical to that of temperate phage going dormant and usually, is temporary.

In contrast, the _lysogenic cycle_ does not result in immediate lysing of the host cell. Those phages able to undergo lysogeny are known as temperate phages. Their viral genome will integrate with host DNA and replicate along with it, relatively harmlessly, or may even become established as a plasmid. The virus remains dormant until host conditions deteriorate, perhaps due to depletion of nutrients, then, the endogenous phages (known as prophages) become active. At this point they initiate the reproductive cycle, resulting in lysis of the host cell. As the lysogenic cycle allows the host cell to continue to survive and reproduce, the virus is replicated in all offspring of the cell. An example of a bacteriophage known to follow the lysogenic cycle and the lytic cycle is the phage lambda of _E. coli._[42]

Sometimes prophages may provide benefits to the host bacterium while they are dormant by adding new functions to the bacterial genome, in a phenomenon called lysogenic conversion. Examples are the conversion of harmless strains of _Corynebacterium diphtheriae_ or _Vibrio cholerae_ by bacteriophages, to highly virulent ones that cause diphtheria or cholera, respectively.[43][44] Strategies to combat certain bacterial infections by targeting these toxin-encoding prophages have been proposed.[45]

Attachment and penetration

Phage.jpg of bacteriophages attached to a bacterial cell, the viruses are the size and shape of coliphage T1 ]] To enter a host cell, bacteriophages attach to specific receptors on the surface of bacteria, including lipopolysaccharides, teichoic acids, proteins, or even flagella. This specificity means a bacteriophage can infect only certain bacteria bearing receptors to which they can bind, which in turn, determines the phage's host range. Host growth conditions also influence the ability of the phage to attach and invade them.[46] As phage virions do not move independently, they must rely on random encounters with the correct receptors when in solution, such as blood, lymphatic circulation, irrigation, soil water, etc.

Myovirus bacteriophages use a hypodermic syringe-like motion to inject their genetic material into the cell. After making contact with the appropriate receptor, the tail fibers flex to bring the base plate closer to the surface of the cell. This is known as reversible binding. Once attached completely, irreversible binding is initiated and the tail contracts, possibly with the help of ATP, present in the tail,[47] injecting genetic material through the bacterial membrane. The injection is acomplished through a sort of bending motion in the shaft by going to the side, contracting closer to the cell and pushing back up. Podoviruses lack an elongated tail sheath similar to that of a myovirus, so instead, they use their small, tooth-like tail fibers enzymatically to degrade a portion of the cell membrane before inserting their genetic material.

Synthesis of proteins and nucleic acid

Within minutes, bacterial ribosomes start translating viral mRNA into protein. For RNA-based phages, RNA replicase is synthesized early in the process. Proteins modify the bacterial RNA polymerase so it preferentially transcribes viral mRNA. The host’s normal synthesis of proteins and nucleic acids is disrupted, and it is forced to manufacture viral products instead. These products go on to become part of new virions within the cell, helper proteins that contribute to the assemblage of new virions, or proteins involved in cell lysis. In 1972, Walter Fiers (University of Ghent, Belgium) was the first to establish the complete nucleotide sequence of a gene and in 1976, of the viral genome of bacteriophage MS2.[48] Some dsDNA bacteriophages encode ribosomal proteins, which are thought to modulate protein translation during phage infection.[49]

Virion assembly

In the case of the T4 phage, the construction of new virus particles involves the assistance of helper proteins. The base plates are assembled first, with the tails being built upon them afterward. The head capsids, constructed separately, will spontaneously assemble with the tails. The DNA is packed efficiently within the heads. The whole process takes about 15 minutes. Tevenphage.svg

Release of virions

Phages may be released via cell lysis, by extrusion, or, in a few cases, by budding. Lysis, by tailed phages, is achieved by an enzyme called endolysin, which attacks and breaks down the cell wall peptidoglycan. An altogether different phage type, the filamentous phage, make the host cell continually secrete new virus particles. Released virions are described as free, and, unless defective, are capable of infecting a new bacterium. Budding is associated with certain _Mycoplasma_ phages. In contrast to virion release, phages displaying a lysogenic cycle do not kill the host but, rather, become long-term residents as prophage.


Genome structure

Given the millions of different phages in the environment, phage genomes come in a variety of forms and sizes. RNA phage such as MS2 have the smallest genomes, of only a few kilobases. However, some DNA phage such as T4 may have large genomes with hundreds of genes; the size and shape of the capsid varies along with the size of the genome.[50]

Bacteriophage genomes can be highly mosaic, i.e. the genome of many phage species appear to be composed of numerous individual modules. These modules may be found in other phage species in different arrangements. Mycobacteriophages, bacteriophages with mycobacterial hosts, have provided excellent examples of this mosaicism. In these mycobacteriophages, genetic assortment may be the result of repeated instances of site-specific recombination and illegitimate recombination (the result of phage genome acquisition of bacterial host genetic sequences).[51] Evolutionary mechanisms shaping the genomes of bacterial viruses vary between different families and depend upon the type of the nucleic acid, characteristics of the virion structure, as well as the mode of the viral life cycle.[52]


Systems biology

Phage often have dramatic effects on their hosts. As a consequence, the transcription pattern of the infected bacterium may change considerably. For instance, infection of _Pseudomonas aeruginosa_ by the temperate phage PaP3 changed the expression of 38% (2160/5633) of its host's genes. Many of these effects are probably indirect, hence the challenge becomes to identify the direct interactions among bacteria and phage.[53]

Several attempts have been made to map protein–protein interactions among phage and their host. For instance, bacteriophage lambda was found to interact with its host, E. coli, by 31 interactions. However, a large-scale study revealed 62 interactions, most of which were new. Again, the significance of many of these interactions remains unclear, but these studies suggest that there most likely are several key interactions and many indirect interactions whose role remains uncharacterized.[54]


In the environment

Metagenomics has allowed the in-water detection of bacteriophages that was not possible previously.[55]

Also, bacteriophages have been used in hydrological tracing and modelling in river systems, especially where surface water and groundwater interactions occur. The use of phages is preferred to the more conventional dye marker because they are significantly less absorbed when passing through ground waters and they are readily detected at very low concentrations.[56] Non-polluted water may contain approximately 2×10⁸ bacteriophages per mL.[57]

Bacteriophages are thought to contribute extensively to horizontal gene transfer in natural environments, principally via transduction, but also via transformation.[58] Metagenomics-based studies also have revealed that viromes from a variety of environments harbor antibiotic-resistance genes, including those that could confer multidrug resistance.[59]


Model bacteriophages

The following bacteriophages are extensively studied:

-   186 phage
-   λ phage
-   Φ6 phage
-   Φ29 phage
-   ΦX174
-   G4 phage
-   M13 phage
-   MS2 phage (23–28 nm in size)[60]
-   N4 phage
-   P1 phage
-   P2 phage
-   P4 phage
-   R17 phage
-   T2 phage
-   T4 phage (169 kbp genome,[61] 200 nm long[62])
-   T7 phage
-   T12 phage


See also

-   Bacterivore
-   CrAssphage
-   DNA viruses
-   Phage ecology
-   Phage monographs (a comprehensive listing of phage and phage-associated monographs, 1921 – present)
-   Polyphage
-   RNA viruses
-   Transduction
-   Viriome
-   CRISPR


References


External links

-   Häusler, T. (2006) "Viruses vs. Superbugs", Macmillan
-   Animation of bacteriophage targeting E. coli bacteria
-   Phage.org general information on bacteriophages
-   bacteriophages illustrations and genomics
-   Bacteriophages get a foothold on their prey
-   NPR Science Friday podcast, "Using 'Phage' Viruses to Help Fight Infection", April 2008
-   Animation by _Hybrid Animation Medical_ for a T4 Bacteriophage targeting _E. coli_ bacteria
-

Bacteriophages

[1]

[2]

[3]

[4] Prescott, L. (1993). Microbiology, Wm. C. Brown Publishers,

[5] BBC Horizon (1997): _The Virus that Cures_ – Documentary about the history of phage medicine in Russia and the West

[6]

[7]

[8]

[9]

[10]

[11]

[12]

[13]

[14]

[15]

[16]

[17]

[18]

[19]

[20]

[21]

[22]

[23]

[24]

[25]

[26]

[27]

[28]

[29] U.S. FDA/CFSAN: Agency Response Letter, GRAS Notice No. 000198

[30] (U.S. FDA/CFSAN: Agency Response Letter, GRAS Notice No. 000218)

[31] FSIS Directive 7120

[32]

[33] FDA 510(k) Premarket Notification

[34] FDA clears first test to quickly diagnose and distinguish MRSA and MSSA. FDA (May 6, 2011)

[35] Vaisman, Daria (May 25, 2007) Studying anthrax in a Soviet-era lab – with Western funding. _The New York Times_

[36]

[37]

[38]

[39]

[40] Technological background Phage-ligand technology

[41]

[42] Mason, Kenneth A., Jonathan B. Losos, Susan R. Singer, Peter H Raven, and George B. Johnson. (2011). _Biology_, p. 533. McGraw-Hill, New York. .

[43]

[44]

[45]

[46]

[47]

[48]

[49]

[50]

[51]

[52]

[53]

[54]

[55]

[56]

[57]

[58]

[59]

[60]

[61]

[62]