s (top) and cephalosporins (bottom). β-lactam ring in red.]] '' bacteria on the right are sensitive to two beta-lactam antibiotics, and do not grow in the semi-circular regions surrounding antibiotics. _E. coli_ bacteria on the left are resistant to beta-lactam antibiotics, and grow next to one antibiotic (bottom) and are less inhibited by another antibiotic (top).]]

BETA-LACTAMASES are enzymes () produced by bacteria that provide multi-resistance to β-lactam antibiotics such as penicillins, cephalosporins, cephamycins, and carbapenems (ertapenem), although carbapenems are relatively resistant to beta-lactamase. Beta-lactamase provides antibiotic resistance by breaking the antibiotics' structure. These antibiotics all have a common element in their molecular structure: a four-atom ring known as a β-lactam. Through hydrolysis, the lactamase enzyme breaks the β-lactam ring open, deactivating the molecule's antibacterial properties.

Beta-lactam antibiotics are typically used to treat a broad spectrum of Gram-positive and Gram-negative bacteria.

Beta-lactamases produced by Gram-negative organisms are usually secreted, especially when antibiotics are present in the environment.[1]


Structure

The structure of a _Streptomyces_ β-lactamase is given by .


Penicillinase

Penicillinase is a specific type of β-lactamase, showing specificity for penicillins, again by hydrolysing the β-lactam ring. Molecular weights of the various penicillinases tend to cluster near 50 kiloDaltons.

Penicillinase was the first β-lactamase to be identified. It was first isolated by Abraham and Chain in 1940 from Gram-negative _E. coli_ even before penicillin entered clinical use,[2] but penicillinase production quickly spread to bacteria that previously did not produce it or produced it only rarely. Penicillinase-resistant beta-lactams such as methicillin were developed, but there is now widespread resistance to even these.


Resistance in Gram-negative bacteria

Among Gram-negative bacteria, the emergence of resistance to expanded-spectrum cephalosporins has been a major concern. It appeared initially in a limited number of bacterial species (_E. cloacae_, _C. freundii_, _S. marcescens_, and _P. aeruginosa_) that could mutate to hyperproduce their chromosomal class C β-lactamase. A few years later, resistance appeared in bacterial species not naturally producing AmpC enzymes (_K. pneumoniae_, _Salmonella_ spp., _P. mirabilis_) due to the production of TEM- or SHV-type ESBLs (extended spectrum beta lactamases). Characteristically, such resistance has included oxyimino- (for example ceftizoxime, cefotaxime, ceftriaxone, and ceftazidime, as well as the oxyimino-monobactam aztreonam), but not 7-alpha-methoxy-cephalosporins (cephamycins; in other words, cefoxitin and cefotetan); has been blocked by inhibitors such as clavulanate, sulbactam or tazobactam and did not involve carbapenems and temocillin. Chromosomal-mediated AmpC β-lactamases represent a new threat, since they confer resistance to 7-alpha-methoxy-cephalosporins (cephamycins) such as cefoxitin or cefotetan but are not affected by commercially available β-lactamase inhibitors, and can, in strains with loss of outer membrane porins, provide resistance to carbapenems.[3]

Extended-spectrum beta-lactamase (ESBL)

Members of the family commonly express plasmid-encoded β-lactamases (e.g., TEM-1, TEM-2, and SHV-1), which confer resistance to penicillins but not to expanded-spectrum cephalosporins. In the mid-1980s, a new group of enzymes, the extended-spectrum β-lactamases (ESBLs), was detected (first detected in 1979).[4] The prevalence of ESBL-producing bacteria have been gradually increasing in acute care hospitals.[5] ESBLs are beta-lactamases that hydrolyze extended-spectrum cephalosporins with an oxyimino side chain. These cephalosporins include cefotaxime, ceftriaxone, and ceftazidime, as well as the oxyimino-monobactam aztreonam. Thus ESBLs confer multi-resistance to these antibiotics and related oxyimino-beta lactams. In typical circumstances, they derive from genes for TEM-1, TEM-2, or SHV-1 by mutations that alter the amino acid configuration around the active site of these β-lactamases. A broader set of β-lactam antibiotics are susceptible to hydrolysis by these enzymes. An increasing number of ESBLs not of TEM or SHV lineage have recently been described.[6] The ESBLs are frequently plasmid encoded. Plasmids responsible for ESBL production frequently carry genes encoding resistance to other drug classes (for example, aminoglycosides). Therefore, antibiotic options in the treatment of ESBL-producing organisms are extremely limited. Carbapenems are the treatment of choice for serious infections due to ESBL-producing organisms, yet carbapenem-resistant (primarily ertapenem resistant) isolates have recently been reported. ESBL-producing organisms may appear susceptible to some extended-spectrum cephalosporins. However, treatment with such antibiotics has been associated with high failure rates.

Types

TEM beta-lactamases (class A)

TEM-1 is the most commonly encountered beta-lactamase in Gram-negative bacteria. Up to 90% of ampicillin resistance in _E. coli_ is due to the production of TEM-1.[7] Also responsible for the ampicillin and penicillin resistance that is seen in _H. influenzae_ and _N. gonorrhoeae_ in increasing numbers. Although TEM-type beta-lactamases are most often found in _E. coli_ and _K. pneumoniae_, they are also found in other species of Gram-negative bacteria with increasing frequency. The amino acid substitutions responsible for the extended-spectrum beta lactamase (ESBL) phenotype cluster around the active site of the enzyme and change its configuration, allowing access to oxyimino-beta-lactam substrates. Opening the active site to beta-lactam substrates also typically enhances the susceptibility of the enzyme to β-lactamase inhibitors, such as clavulanic acid. Single amino acid substitutions at positions 104, 164, 238, and 240 produce the ESBL phenotype, but ESBLs with the broadest spectrum usually have more than a single amino acid substitution. Based upon different combinations of changes, currently 140 TEM-type enzymes have been described. TEM-10, TEM-12, and TEM-26 are among the most common in the United States.[8][9][10] The term TEM comes from the name of the Athenian patient (Temoniera) from which the isolate was recovered in 1963.[11]

SHV beta-lactamases (class A)

SHV-1 shares 68 percent of its amino acids with TEM-1 and has a similar overall structure. The SHV-1 beta-lactamase is most commonly found in _K. pneumoniae_ and is responsible for up to 20% of the plasmid-mediated ampicillin resistance in this species. ESBLs in this family also have amino acid changes around the active site, most commonly at positions 238 or 238 and 240. More than 60 SHV varieties are known. SHV-5 and SHV-12 are among the most common.[12]

CTX-M beta-lactamases (class A)

These enzymes were named for their greater activity against cefotaxime than other oxyimino-beta-lactam substrates (e.g., ceftazidime, ceftriaxone, or cefepime). Rather than arising by mutation, they represent examples of plasmid acquisition of beta-lactamase genes normally found on the chromosome of _Kluyvera_ species, a group of rarely pathogenic commensal organisms. These enzymes are not very closely related to TEM or SHV beta-lactamases in that they show only approximately 40% identity with these two commonly isolated beta-lactamases. More than 80 CTX-M enzymes are currently known. Despite their name, a few are more active on ceftazidime than cefotaxime. They have mainly been found in strains of _Salmonella enterica_ serovar _Typhimurium_ and _E. coli_, but have also been described in other species of Enterobacteriaceae and are the predominant ESBL type in parts of South America. (They are also seen in eastern Europe) CTX-M-14, CTX-M-3, and CTX-M-2 are the most widespread. CTX-M-15 is currently (2006) the most widespread type in _E. coli_ the UK and is widely prevalent in the community.[13] An example of beta-lactamase CTX-M-15, along with IS_Ecp1_, has been found to have recently transposed onto the chromosome of _Klebsiella pneumoniae_ ATCC BAA-2146.[14]

OXA beta-lactamases (class D)

OXA beta-lactamases were long recognized as a less common but also plasmid-mediated beta-lactamase variety that could hydrolyze oxacillin and related anti-staphylococcal penicillins. These beta-lactamases differ from the TEM and SHV enzymes in that they belong to molecular class D and functional group 2d . The OXA-type beta-lactamases confer resistance to ampicillin and cephalothin and are characterized by their high hydrolytic activity against oxacillin and cloxacillin and the fact that they are poorly inhibited by clavulanic acid. Amino acid substitutions in OXA enzymes can also give the ESBL phenotype. While most ESBLs have been found in _E. coli_, _K. pneumoniae_, and other Enterobacteriaceae, the OXA-type ESBLs have been found mainly in _P. aeruginosa_. OXA-type ESBLs have been found mainly in _Pseudomonas aeruginosa_ isolates from Turkey and France. The OXA beta-lactamase family was originally created as a phenotypic rather than a genotypic group for a few beta-lactamases that had a specific hydrolysis profile. Therefore, there is as little as 20% sequence homology among some of the members of this family. However, recent additions to this family show some degree of homology to one or more of the existing members of the OXA beta-lactamase family. Some confer resistance predominantly to ceftazidime, but OXA-17 confers greater resistance to cefotaxime and cefepime than it does resistance to ceftazidime.

Others

Other plasmid-mediated ESBLs, such as PER, VEB, GES, and IBC beta-lactamases, have been described but are uncommon and have been found mainly in _P. aeruginosa_ and at a limited number of geographic sites. PER-1 in isolates in Turkey, France, and Italy; VEB-1 and VEB-2 in strains from Southeast Asia; and GES-1, GES-2, and IBC-2 in isolates from South Africa, France, and Greece. PER-1 is also common in multiresistant acinetobacter species in Korea and Turkey. Some of these enzymes are found in Enterobacteriaceae as well, whereas other uncommon ESBLs (such as BES-1, IBC-1, SFO-1, and TLA-1) have been found only in Enterobacteriaceae.

Treatment

While ESBL-producing organisms were previously associated with hospitals and institutional care, these organisms are now increasingly found in the community. CTX-M-15-positive E. coli are a cause of community-acquired urinary infections in the UK,[15] and tend to be resistant to all oral β-lactam antibiotics, as well as quinolones and sulfonamides. Treatment options may include nitrofurantoin, fosfomycin, mecillinam and chloramphenicol. In desperation, once-daily ertapenem or gentamicin injections may also be used.

Inhibitor-resistant β-lactamases

Although the inhibitor-resistant β-lactamases are not ESBLs, they are often discussed with ESBLs because they are also derivatives of the classical TEM- or SHV-type enzymes. These enzymes were at first given the designation IRT for inhibitor-resistant TEM β-lactamase; however, all have subsequently been renamed with numerical TEM designations. There are at least 19 distinct inhibitor-resistant TEM β-lactamases. Inhibitor-resistant TEM β-lactamases have been found mainly in clinical isolates of _E. coli_, but also some strains of _K. pneumoniae_, _Klebsiella oxytoca_, _P. mirabilis_, and _Citrobacter freundii_. Although the inhibitor-resistant TEM variants are resistant to inhibition by clavulanic acid and sulbactam, thereby showing clinical resistance to the beta-lactam—lactamase inhibitor combinations of amoxicillin-clavulanate (co-amoxiclav), ticarcillin-clavulanate (co-ticarclav), and ampicillin/sulbactam, they normally remain susceptible to inhibition by tazobactam and subsequently the combination of piperacillin/tazobactam, although resistance has been described. This is no longer a primarily European epidemiology, it is found in northern parts of America often and should be tested for with complex UTI's.[16]

AmpC-type β-lactamases (Class C)

AmpC type β-lactamases are commonly isolated from extended-spectrum cephalosporin-resistant Gram-negative bacteria. AmpC β-lactamases (also termed class C or group 1) are typically encoded on the chromosome of many Gram-negative bacteria including _Citrobacter_, _Serratia_ and _Enterobacter_ species where its expression is usually inducible; it may also occur on _Escherichia coli_ but is not usually inducible, although it can be hyperexpressed. AmpC type β-lactamases may also be carried on plasmids.[17] AmpC β-lactamases, in contrast to ESBLs, hydrolyse broad and extended-spectrum cephalosporins (cephamycins as well as to oxyimino-β-lactams) but are not inhibited by β-lactamase inhibitors such as clavulanic acid. AmpC-type β-lactamase organisms are often clinically grouped through the acronym, "SPACE": _Serratia, Pseudomonas_ or _Proteus, Acinetobacter, Citrobacter_, and _Enterobacter_.

Carbapenemases

Carbapenems are famously stable to AmpC β-lactamases and extended-spectrum-β-lactamases. Carbapenemases are a diverse group of β-lactamases that are active not only against the oxyimino-cephalosporins and cephamycins but also against the carbapenems. Aztreonam is stable to the metallo-β-lactamases, but many IMP and VIM producers are resistant, owing to other mechanisms. Carbapenemases were formerly believed to derive only from classes A, B, and D, but a class C carbapenemase has been described.

IMP-type carbapenemases (metallo-β-lactamases) (Class B)

Plasmid-mediated IMP-type carbapenemases, 19 varieties of which are currently known, became established in Japan in the 1990s both in enteric Gram-negative organisms and in _Pseudomonas_ and _Acinetobacter_ species. IMP enzymes spread slowly to other countries in the Far East, were reported from Europe in 1997, and have been found in Canada and Brazil.

VIM (Verona integron-encoded metallo-β-lactamase) (Class B)

A second growing family of carbapenemases, the VIM family, was reported from Italy in 1999 and now includes 10 members, which have a wide geographic distribution in Europe, South America, and the Far East and have been found in the United States. VIM-1 was discovered in _P. aeruginosa_ in Italy in 1996; since then, VIM-2 - now the predominant variant - was found repeatedly in Europe and the Far East; VIM-3 and -4 are minor variants of VIM-2 and -1, respectively. VIM enzymes occur mostly in _P. aeruginosa_, also _P. putida_ and, very rarely, Enterobacteriaceae.

Amino acid sequence diversity is up to 10% in the VIM family, 15% in the IMP family, and 70% between VIM and IMP. Enzymes of both the families, nevertheless, are similar. Both are integron-associated, sometimes within plasmids. Both hydrolyse all β-lactams except monobactams, and evade all β-lactam inhibitors. The VIM enzymes are among the most widely distributed MBLs, with >40 VIM variants having been reported. Biochemical and biophysical studies revealed that VIM variants have only small variations in their kinetic parameters but substantial differences in their thermal stabilities and inhibition profiles.[18]

OXA (oxacillinase) group of β-lactamases (Class D)

The OXA group of β-lactamases occur mainly in Acinetobacter species and are divided into two clusters. OXA carbapenemases hydrolyse carbapenems very slowly _in vitro_, and the high MICs seen for some Acinetobacter hosts (>64 mg/L) may reflect secondary mechanisms. They are sometimes augmented in clinical isolates by additional resistance mechanisms, such as impermeability or efflux. OXA carbapenemases also tend to have a reduced hydrolytic efficiency towards penicillins and cephalosporins.[19]

KPC (_K. pneumoniae_ carbapenemase) (Class A)

A few class A enzymes, most noted the plasmid-mediated KPC enzymes, are effective carbapenemases as well. Ten variants, KPC-2 through KPC-11 are known, and they are distinguished by one or two amino acid substitutions (KPC-1 was re-sequenced in 2008 and found to be 100% homologous to published sequences of KPC-2). KPC-1 was found in North Carolina, KPC-2 in Baltimore and KPC-3 in New York. They have only 45% homology with SME and NMC/IMI enzymes and, unlike them, can be encoded by self-transmissible plasmids.

, the class A _Klebsiella pneumoniae_ carbapenemase (KPC) globally has been the most common carbapenemase, and was first detected in 1996 in North Carolina, USA.[20] A 2010 publication indicated that KPC producing Enterobacteriaceae were becoming common in the United States.[21]

CMY (Class C)

The first class C carbapenemase was described in 2006 and was isolated from a virulent strain of _Enterobacter aerogenes_.[22] It is carried on a plasmid, pYMG-1, and is therefore transmissible to other bacterial strains.[23]

SME (Serratia marcescens Enzymes), IMI (IMIpenem-hydrolysing β-lactamase), NMC and CcrA

In general, these are of little clinical significance.

CcrA (CfiA). Its gene occurs in ca. 1-3% of _B. fragilis_ isolates, but fewer produce the enzyme since expression demands appropriate migration of an insertion sequence. CcrA was known before imipenem was introduced, and producers have shown little subsequent increase.

NDM-1 (New Delhi metallo-β-lactamase) (Class B)

Originally described from New Delhi in 2009, this gene is now widespread in _Escherichia coli_ and _Klebsiella pneumoniae_ from India and Pakistan. As of mid-2010, NDM-1 carrying bacteria have been introduced to other countries (including the United States and UK), most probably due to the large number of tourists travelling the globe, who may have picked up the strain from the environment, as strains containing the NDM-1 gene have been found in environmental samples in India.[24] NDM have several variants which share different properties.[25]


Treatment of ESBL/AmpC/carbapenemases

General overview

In general, an isolate is suspected to be an ESBL producer when it shows _in vitro_ susceptibility to the second-generation cephalosporins (cefoxitin, cefotetan) but resistance to the third-generation cephalosporins and to aztreonam. Moreover, one should suspect these strains when treatment with these agents for Gram-negative infections fails despite reported _in vitro_ susceptibility. Once an ESBL-producing strain is detected, the laboratory should report it as "resistant" to all penicillins, cephalosporins, and aztreonam, even if it is tested (in vitro) as susceptible. Associated resistance to aminoglycosides and trimethoprim-sulfamethoxazole, as well as high frequency of co-existence of fluoroquinolone resistance, creates problems. Beta-lactamase inhibitors such as clavulanate, sulbactam, and tazobactam _in vitro_ inhibit most ESBLs, but the clinical effectiveness of beta-lactam/beta-lactamase inhibitor combinations cannot be relied on consistently for therapy. Cephamycins (cefoxitin and cefotetan) are not hydrolyzed by majority of ESBLs, but are hydrolyzed by associated AmpC-type β-lactamase. Also, β-lactam/β-lactamase inhibitor combinations may not be effective against organisms that produce AmpC-type β-lactamase. Sometimes these strains decrease the expression of outer membrane proteins, rendering them resistant to cephamycins. _In vivo_ studies have yielded mixed results against ESBL-producing _K. pneumoniae_. (Cefepime, a fourth-generation cephalosporin, has demonstrated _in vitro_ stability in the presence of many ESBL/AmpC strains.) Currently, carbapenems are, in general, regarded as the preferred agent for treatment of infections due to ESBL-producing organisms. Carbapenems are resistant to ESBL-mediated hydrolysis and exhibit excellent _in vitro_ activity against strains of Enterobacteriaceae expressing ESBLs.

According to genes

ESBLs

Strains producing only ESBLs are susceptible to cephamycins and carbapenems _in vitro_ and show little if any inoculum effect with these agents.

For organisms producing TEM and SHV type ESBLs, apparent _in vitro_ sensitivity to cefepime and to piperacillin/tazobactam is common, but both drugs show an inoculum effect, with diminished susceptibility as the size of the inoculum is increased from 10⁵ to 10⁷ organisms.

Strains with some CTX-M–type and OXA-type ESBLs are resistant to cefepime on testing, despite the use of a standard inoculum.

Inhibitor-Resistant β-lactamases

Although the inhibitor-resistant TEM variants are resistant to inhibition by clavulanic acid and sulbactam, thereby showing clinical resistance to the beta-lactam—beta lactamase inhibitor combinations of amoxicillin-clavulanate (Co-amoxiclav), ticarcillin-clavulanate, and ampicillin/sulbactam, they remain susceptible to inhibition by tazobactam and subsequently the combination of piperacillin/tazobactam.

AmpC

AmpC-producing strains are typically resistant to oxyimino-beta lactams and to cephamycins and are susceptible to carbapenems; however, diminished porin expression can make such a strain carbapenem-resistant as well.

Carbapenemases

Strains with IMP-, VIM-, AND OXA-type carbapenemases usually remain susceptible. Resistance to non-beta-lactam antibiotics is common in strains making any of these enzymes, such that alternative options for non-beta-lactam therapy need to be determined by direct susceptibility testing. Resistance to fluoroquinolones and aminoglycosides is especially high.

According to species

_Escherichia coli_ or _Klebsiella_

For infections caused by ESBL-producing _Escherichia coli_ or _Klebsiella_ species, treatment with imipenem or meropenem has been associated with the best outcomes in terms of survival and bacteriologic clearance. Cefepime and piperacillin/tazobactam have been less successful. Ceftriaxone, cefotaxime, and ceftazidime have failed even more often, despite the organism's susceptibility to the antibiotic _in vitro_. Several reports have documented failure of cephamycin therapy as a result of resistance due to porin loss. Some patients have responded to aminoglycoside or quinolone therapy, but, in a recent comparison of ciprofloxacin and imipenem for bacteremia involving an ESBL-producing _K. pneumoniae_, imipenem produced the better outcome

_Pseudomonas aeruginosa_

There have been few clinical studies to define the optimal therapy for infections caused by ESBL producing _Pseudomonas aeruginosa_ strains.


Detection

Beta-lactamase enzymatic activity can be detected using nitrocefin, a chromogenic cephalosporin substrate which changes color from yellow to red upon beta-lactamase mediated hydrolysis.[26]


Evolution

Beta-lactamases are ancient bacterial enzymes. The class B beta-lactamases (the metallo-beta-lactamases) are divided into three subclasses: B1, B2 and B3. Subclasses B1 and B2 are theorized to have evolved about one billion years ago and subclass B3s is theorized to have evolved before the divergence of the Gram-positive and Gram-negative eubacteria about two billion years ago.[27]

The other three groups are serine enzymes that show little homology to each other. Structural studies have shown that groups A and D are sister taxa and that group C diverged before A and D.[28] These serine-based enzymes, like the group B betalactamases, are of ancient origin and are theorized to have evolved about two billion years ago.[29]

The OXA group (in class D) in particular is theorized to have evolved on chromosomes and moved to plasmids on at least two separate occasions.[30]


Etymology

The "β" (beta) refers to the nitrogen's position on the second carbon in the ring. _Lactam_ is a portmanteau of _lactone_ (from the Latin _lactis_, _milk_, since lactic acid was isolated from soured milk) and _amide_. The suffix _-ase_, indicating an enzyme, is derived from _diastase_ (from the Greek _diastasis_, "separation"), the first enzyme discovered in 1833 by Payen and Persoz.[31]


See also

-   ESBL-producing _E. coli_
-   Nitrocefin
-   β-lactamase inhibitor


References


External links

-   Online ESBL genotyping tool (EGT)
-   Online Amino Acid Sequences for ESBL enzymes
-

Category:EC 3.5.2 Category:Beta-lactam antibiotics Category:Enzymes of known structure

[1]

[2]

[3]

[4]

[5]

[6]

[7]

[8]

[9]

[10]

[11]

[12]

[13]

[14]

[15]

[16]

[17]

[18]

[19]

[20]

[21]

[22]

[23]

[24]

[25]

[26]

[27]

[28]

[29]

[30]

[31]