CLADISTICS (, from Greek , _kládos_, "branch")[1] is an approach to biological classification in which organisms are categorized in groups ("clades") based on the most recent common ancestor. Hypothesized relationships are typically based on shared derived characteristics (synapomorphies_)_ that can be traced to the most recent common ancestor and are not present in more distant groups and ancestors. A key feature of a clade is that a common ancestor and all its descendants are part of the clade. Importantly, all descendants stay in their overarching ancestral clade. For example, if within a _strict_ cladistic framework the terms _animals_, _bilateria/worms_, _fishes/vertebrata_, or _monkeys/anthropoidea_ were used, these terms would include humans. Many of these terms are normally used paraphyletically, outside of cladistics, e.g. as a 'grade'. Radiation results in the generation of new subclades by bifurcation.[2][3][4][5]

The techniques and nomenclature of cladistics have been applied to other disciplines. (See phylogenetic nomenclature.)

Cladistics is now the most commonly used method to classify organisms.[6]


History

Willi_Hennig2.jpg Peter_Chalmers_Mitchell_1920.jpg CSIRO_ScienceImage_2955_Robert_J_Tillyard_18811937.jpg

The original methods used in cladistic analysis and the school of taxonomy derived from the work of the German entomologist Willi Hennig, who referred to it as PHYLOGENETIC SYSTEMATICS (also the title of his 1966 book); the terms "cladistics" and "clade" were popularized by other researchers. Cladistics in the original sense refers to a particular set of methods used in phylogenetic analysis, although it is now sometimes used to refer to the whole field.

What is now called the cladistic method appeared as early as 1901 with a work by Peter Chalmers Mitchell for birds[7][8] and subsequently by Robert John Tillyard (for insects) in 1921,[9] and W. Zimmermann (for plants) in 1943.[10] The term "clade" was introduced in 1958 by Julian Huxley after having been coined by Lucien Cuénot in 1940,[11] "cladogenesis" in 1958,[12] "cladistic" by Cain and Harrison in 1960,[13] "cladist" (for an adherent of Hennig's school) by Mayr in 1965,[14] and "cladistics" in 1966.[15] Hennig referred to his own approach as "phylogenetic systematics". From the time of his original formulation until the end of the 1970s, cladistics competed as an analytical and philosophical approach to systematics with phenetics and so-called evolutionary taxonomy. Phenetics was championed at this time by the numerical taxonomists Peter Sneath and Robert Sokal, and evolutionary taxonomy by Ernst Mayr.

Originally conceived, if only in essence, by Willi Hennig in a book published in 1950, cladistics did not flourish until its translation into English in 1966 (Lewin 1997). Today, cladistics is the most popular method for constructing phylogenies from morphological data.

In the 1990s, the development of effective polymerase chain reaction techniques allowed the application of cladistic methods to biochemical and molecular genetic traits of organisms, vastly expanding the amount of data available for phylogenetics. At the same time, cladistics rapidly became popular in evolutionary biology, because computers made it possible to process large quantities of data about organisms and their characteristics.


Methodology

The cladistic method interprets each character state transformation implied by the distribution of shared character states among taxa (or other terminals) as a potential piece of evidence for grouping. The outcome of a cladistic analysis is a cladogram – a tree-shaped diagram (dendrogram)[16] that is interpreted to represent the best hypothesis of phylogenetic relationships. Although traditionally such cladograms were generated largely on the basis of morphological characters and originally calculated by hand, genetic sequencing data and computational phylogenetics are now commonly used in phylogenetic analyses, and the parsimony criterion has been abandoned by many phylogeneticists in favor of more "sophisticated" but less parsimonious evolutionary models of character state transformation. Cladists contend that these models are unjustified.

Every cladogram is based on a particular dataset analyzed with a particular method. Datasets are tables consisting of molecular, morphological, ethological[17] and/or other characters and a list of operational taxonomic units (OTUs), which may be genes, individuals, populations, species, or larger taxa that are presumed to be monophyletic and therefore to form, all together, one large clade; phylogenetic analysis infers the branching pattern within that clade. Different datasets and different methods, not to mention violations of the mentioned assumptions, often result in different cladograms. Only scientific investigation can show which is more likely to be correct.

Until recently, for example, cladograms like the following have generally been accepted as accurate representations of the ancestral relations among turtles, lizards, crocodilians, and birds:[18]

If this phylogenetic hypothesis is correct, then the last common ancestor of turtles and birds, at the branch near the lived earlier than the last common ancestor of lizards and birds, near the . Most molecular evidence, however, produces cladograms more like this:[19]

If this is accurate, then the last common ancestor of turtles and birds lived later than the last common ancestor of lizards and birds. Since the cladograms provide competing accounts of real events, at most one of them is correct.

Monophyly,_paraphyly,_polyphyly.pngs, showing a monophyletic taxon (a clade: the simians or Anthropoidea, in yellow), a paraphyletic taxon (the prosimians, in blue, including the red patch), and a polyphyletic taxon (the nocturnal primates – the lorises and the tarsiers – in red)]]

The cladogram to the right represents the current universally accepted hypothesis that all primates, including strepsirrhines like the lemurs and lorises, had a common ancestor all of whose descendants were primates, and so form a clade; the name Primates is therefore recognized for this clade. Within the primates, all anthropoids (monkeys, apes and humans) are hypothesized to have had a common ancestor all of whose descendants were anthropoids, so they form the clade called Anthropoidea. The "prosimians", on the other hand, form a paraphyletic taxon. The name Prosimii is not used in phylogenetic nomenclature, which names only clades; the "prosimians" are instead divided between the clades Strepsirhini and Haplorhini, where the latter contains Tarsiiformes and Anthropoidea.


Terminology for character states

The following terms, coined by Hennig, are used to identify shared or distinct character states among groups:[20][21][22]

-   A PLESIOMORPHY ("close form") or ANCESTRAL STATE is a character state that a taxon has retained from its ancestors. When two or more taxa that are not nested within each other share a plesiomorphy, it is a SYMPLESIOMORPHY (from _syn-_, "together"). Symplesiomorphies do not mean that the taxa that exhibit that character state are necessarily closely related. For example, Reptilia is traditionally characterized by (among other things) being cold-blooded (i.e., not maintaining a constant high body temperature), whereas birds are warm-blooded. Since cold-bloodedness is a plesiomorphy, inherited from the common ancestor of traditional reptiles and birds, and thus a symplesiomorphy of turtles, snakes and crocodiles (among others), it does not mean that turtles, snakes and crocodiles form a clade that excludes the birds.

-   An APOMORPHY ("separate form") or DERIVED STATE is an innovation. It can thus be used to diagnose a clade – or even to help define a clade name in phylogenetic nomenclature. Features that are derived in individual taxa (a single species or a group that is represented by a single terminal in a given phylogenetic analysis) are called AUTAPOMORPHIES (from _auto-_, "self"). Autapomorphies express nothing about relationships among groups; clades are identified (or defined) by SYNAPOMORPHIES (from _syn-_, "together"). For example, the possession of digits that are homologous with those of _Homo sapiens_ is a synapomorphy within the vertebrates. The tetrapods can be singled out as consisting of the first vertebrate with such digits homologous to those of _Homo sapiens_ together with all descendants of this vertebrate (an apomorphy-based phylogenetic definition).[23] Importantly, snakes and other tetrapods that do not have digits are nonetheless tetrapods: other characters, such as amniotic eggs and diapsid skulls, indicate that they descended from ancestors that possessed digits which are homologous with ours.

-   A character state is HOMOPLASTIC or "an instance of HOMOPLASY" if it is shared by two or more organisms but is absent from their common ancestor or from a later ancestor in the lineage leading to one of the organisms. It is therefore inferred to have evolved by convergence or reversal. Both mammals and birds are able to maintain a high constant body temperature (i.e., they are warm-blooded). However, the accepted cladogram explaining their significant features indicates that their common ancestor is in a group lacking this character state, so the state must have evolved independently in the two clades. Warm-bloodedness is separately a synapomorphy of mammals (or a larger clade) and of birds (or a larger clade), but it is not a synapomorphy of any group including both these clades. Hennig's Auxiliary Principle [24] states that shared character states should be considered evidence of grouping unless they are contradicted by the weight of other evidence; thus, homoplasy of some feature among members of a group may only be inferred after a phylogenetic hypothesis for that group has been established.

The terms plesiomorphy and apomorphy are relative; their application depends on the position of a group within a tree. For example, when trying to decide whether the tetrapods form a clade, an important question is whether having four limbs is a synapomorphy of the earliest taxa to be included within Tetrapoda: did all the earliest members of the Tetrapoda inherit four limbs from a common ancestor, whereas all other vertebrates did not, or at least not homologously? By contrast, for a group within the tetrapods, such as birds, having four limbs is a plesiomorphy. Using these two terms allows a greater precision in the discussion of homology, in particular allowing clear expression of the hierarchical relationships among different homologous features.

It can be difficult to decide whether a character state is in fact the same and thus can be classified as a synapomorphy, which may identify a monophyletic group, or whether it only appears to be the same and is thus a homoplasy, which cannot identify such a group. There is a danger of circular reasoning: assumptions about the shape of a phylogenetic tree are used to justify decisions about character states, which are then used as evidence for the shape of the tree.[25] Phylogenetics uses various forms of parsimony to decide such questions; the conclusions reached often depend on the dataset and the methods. Such is the nature of empirical science, and for this reason, most cladists refer to their cladograms as hypotheses of relationship. Cladograms that are supported by a large number and variety of different kinds of characters are viewed as more robust than those based on more limited evidence.


Terminology for taxa

Mono-, para- and polyphyletic taxa can be understood based on the shape of the tree (as done above), as well as based on their character states.[26][27][28] These are compared in the table below.

  Term        Node-based definition                                                                                                                                                                                                                                         Character-based definition
  ----------- ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Monophyly   A clade, a monophyletic taxon, is a taxon that includes all descendants of an inferred ancestor.                                                                                                                                                              A clade is characterized by one or more APOMORPHIES: derived character states present in the first member of the taxon, inherited by its descendants (unless secondarily lost), and not inherited by any other taxa.
  Paraphyly   A paraphyletic assemblage is one that is constructed by taking a clade and removing one or more smaller clades.[29] (Removing one clade produces a singly paraphyletic assemblage, removing two produces a doubly paraphylectic assemblage, and so on.)[30]   A paraphyletic assemblage is characterized by one or more PLESIOMORPHIES: character states inherited from ancestors but not present in all of their descendants. As a consequence, a paraphyletic assemblage is truncated, in that it excludes one or more clades from an otherwise monophyletic taxon. An alternative name is _evolutionary grade_, referring to an ancestral character state within the group. While paraphyletic assemblages are popular among paleontologists and evolutionary taxonomists, cladists do not recognize paraphyletic assemblages as having any formal information content – they are merely parts of clades.
  Polyphyly   A polyphyletic assemblage is one which is neither monophyletic nor paraphyletic.                                                                                                                                                                              A polyphyletic assemblage is characterized by one or more HOMOPLASIES: character states which have converged or reverted so as to be the same but which have not been inherited from a common ancestor. No systematist recognizes polyphyletic assemblages as taxonomically meaningful entities, although ecologists sometimes consider them meaningful labels for functional participants in ecological communities (e. g., primary producers, detritivores, etc.).


Criticism

Cladistics, either generally or in specific applications, has been criticized from its beginnings. Decisions as to whether particular character states are homologous, a precondition of their being synapomorphies, have been challenged as involving circular reasoning and subjective judgements.[31] Transformed cladistics arose in the late 1970s in an attempt to resolve some of these problems by removing phylogeny from cladistic analysis, but it has remained unpopular.

However, homology is usually determined from analysis of the results that are evaluated with homology measures, mainly the consistency index (CI) and retention index (RI), which, it has been claimed, makes the process objective. Also, homology can be equated to synapomorphy, which is what Patterson has done.[32]


Issues

In organisms with sexual reproduction, incomplete lineage sorting may result in inconsistent phylogenetic trees, depending on which genes are assessed.[33] It is also possible that multiple surviving lineages are generated while interbreeding is still significantly occurring (polytomy). Interbreeding is possible over periods of about 10 million years.[34][35] Typically speciation occurs over only about 1 million years,[36] which makes it less likely multiple long surviving lineages developed "simultaneously". Even so, interbreeding can result in a lineage being overwhelmed and absorbed by a related more numerous lineage.

The cladistic method does not identify fossils as actual ancestors.[37] Instead, they are identified as separate extinct branches, which could be argued to be fine to take as the default position.


In disciplines other than biology

The comparisons used to acquire data on which cladograms can be based are not limited to the field of biology.[38] Any group of individuals or classes that are hypothesized to have a common ancestor, and to which a set of common characteristics may or may not apply, can be compared pairwise. Cladograms can be used to depict the hypothetical descent relationships within groups of items in many different academic realms. The only requirement is that the items have characteristics that can be identified and measured.

Anthropology and archaeology:[39] Cladistic methods have been used to reconstruct the development of cultures or artifacts using groups of cultural traits or artifact features.

Comparative mythology and folktale use cladistic methods to reconstruct the protoversion of many myths. Mythological phylogenies constructed with mythemes clearly support low horizontal transmissions (borrowings), historical (sometimes Palaeolithic) diffusions and punctuated evolution.[40] They also are a powerful way to test hypotheses about cross-cultural relationships among folktales.[41][42]

Literature: Cladistic methods have been used in the classification of the surviving manuscripts of the _Canterbury Tales_,[43] and the manuscripts of the Sanskrit _Charaka Samhita_.[44]

Historical linguistics:[45] Cladistic methods have been used to reconstruct the phylogeny of languages using linguistic features. This is similar to the traditional comparative method of historical linguistics, but is more explicit in its use of parsimony and allows much faster analysis of large datasets (computational phylogenetics).

Textual criticism or stemmatics:[46][47] Cladistic methods have been used to reconstruct the phylogeny of manuscripts of the same work (and reconstruct the lost original) using distinctive copying errors as apomorphies. This differs from traditional historical-comparative linguistics in enabling the editor to evaluate and place in genetic relationship large groups of manuscripts with large numbers of variants that would be impossible to handle manually. It also enables parsimony analysis of contaminated traditions of transmission that would be impossible to evaluate manually in a reasonable period of time.

Astrophysics[48] infers the history of relationships between galaxies to create branching diagram hypotheses of galaxy diversification.


See also

-   Bioinformatics
-   Biomathematics
-   Coalescent theory
-   Common descent
-   Glossary of scientific naming
-   Language family
-   Maximum parsimony
-   Patrocladogram
-   Phylogenetic network
-   Scientific classification
-   Stratocladistics
-   Subclade
-   Systematics
-   Three-taxon analysis
-   Tree model
-   Tree structure


Notes and references


Bibliography

-   -   -   -   -   -   -   Available free online at http://gallica.bnf.fr (No direct URL). This is the paper credited by for the first use of the term 'clade'.

-   -   -   -   -   -   -   -   -   responding to .

-   Translated from manuscript in German eventually published in 1982 (Phylogenetische Systematik, Verlag Paul Parey, Berlin).

-   -   -   -   d'Huy, Julien (2012b), "Le motif de Pygmalion : origine afrasienne et diffusion en Afrique". _Sahara_, 23: 49-59 1.
-   d'Huy, Julien (2013a), "Polyphemus (Aa. Th. 1137). "A phylogenetic reconstruction of a prehistoric tale". _Nouvelle Mythologie Comparée / New Comparative Mythology_ 1, 2
-   -   d'Huy, Julien (2013c) "Les mythes évolueraient par ponctuations". _Mythologie française_, 252, 2013c: 8-12. 3
-   d'Huy, Julien (2013d) "A Cosmic Hunt in the Berber sky : a phylogenetic reconstruction of Palaeolithic mythology". _Les Cahiers de l'AARS_, 15, 2013d: 93-106. 4
-   -   -   -   -   -   -   Reissued 1997 in paperback. Includes a reprint of Mayr's 1974 anti-cladistics paper at pp. 433–476, "Cladistic analysis or cladistic classification." This is the paper to which is a response.

-   -   -   -   .

-   -   -   -   -   -   -   -   Tehrani, Jamshid J., 2013, "The Phylogeny of Little Red Riding Hood", _PLOS ONE_, November 13.5
-   -   -   -   -


External links

-   Willi Hennig Society
-   Cladistics (scholarly journal of the Willi Hennig Society)
-   -   -   -   -

Category:Phylogenetics Category:Evolutionary biology Category:Zoology Category:Philosophy of biology

[1]

[2] Columbia Encyclopedia

[3]

[4] Oxford Dictionary of English

[5] Oxford English Dictionary

[6]

[7] Schuh, Randall. 2000. Biological Systematics: Principles and Applications, p.7 (citing Nelson and Platnick, 1981). Cornell University Press (books.google)

[8] Folinsbee, Kaila et al. 2007. 5 Quantitative Approaches to Phylogenetics, p. 172. Rev. Mex. Div. 225-52 (kfolinsb.public.iastate.edu)

[9]

[10] Schuh, Randall. 2000. Biological Systematics: Principles and Applications, p.7. Cornell U. Press

[11]

[12] Webster's 9th New Collegiate Dictionary

[13]

[14]

[15]

[16]

[17]

[18]

[19]

[20]

[21]

[22]

[23]

[24]

[25] : "Synapomorphies are invoked to defend the hypothesis; the hypothesis is invoked to defend the synapomorphies."

[26]

[27]

[28]

[29] Many sources give a verbal definition of 'paraphyletic' that does not require the missing groups to be monophyletic. However, when diagrams are presented representing paraphyletic groups, these invariably show the missing groups as monophyletic. See e.g.

[30]

[31]

[32] Forey, Peter et al. 1992. Cladistics,1st ed., p. 9, Oxford U. Press.

[33]

[34]

[35]

[36]

[37]

[38]

[39]

[40] d'Huy 2012a, b; d'Huy 2013a, b, c, d

[41] Ross and al. 2013

[42] Tehrani 2013

[43]

[44]

[45]

[46]

[47]

[48]