DNA METHYLATION IN CANCER plays a variety of roles, helping to change the healthy regulation of gene expression to a disease pattern.

All mammalian cells descended from a fertilized egg (a zygote) share a common DNA sequence (except for new mutations in some lineages). However, during development and formation of different tissues epigenetic factors change. The changes include histone modifications, CpG island methylations and chromatin reorganizations which can cause the stable silencing or activation of particular genes.[1] Once differentiated tissues are formed, CpG island methylation is generally stably inherited from one cell division to the next through the DNA methylation maintenance machinery.[2]

In cancer, a number of mutational changes are found in protein coding genes. Colorectal cancers typically have 3 to 6 driver mutations and 33 to 66 hitchhiker or passenger mutations that silence protein expression in the genes affected.[3] However, transcriptional silencing may be more important than mutation in causing gene silencing in progression to cancer.[4] In colorectal cancers about 600 to 800 genes are transcriptionally silenced, compared to adjacent normal-appearing tissues, by CpG island methylation. Transcriptional repression in cancer can also occur by other epigenetic mechanisms, such as altered expression of microRNAs.[5]


CpG islands are frequent control elements

CpG islands are commonly 200 to 2000 base pairs long, have a C:G base pair content >50%, and have frequent 5' â†’ 3' CpG sequences. About 70% of human promoters located near the transcription start site of a gene contain a CpG island.[6][7]

Promoters located at a distance from the transcription start site of a gene also frequently contain CpG islands. The promoter of the DNA repair gene _ERCC1_, for instance, was identified and located about 5,400 nucleotides upstream of its coding region.[8] CpG islands also occur frequently in promoters for functional noncoding RNAs such as microRNAs and Long non-coding RNAs (lncRNAs).


Methylation of CpG islands in promoters stably silences genes

Genes can be silenced by multiple methylation of CpG sites in the CpG islands of their promoters.[9] Even if silencing of a gene is initiated by another mechanism, this often is followed by methylation of CpG sites in the promoter CpG island to stabilize the silencing of the gene.[10] On the other hand, hypomethylation of CpG islands in promoters can result in gene over-expression.


Promoter CpG hyper/hypo-methylation in cancer

In cancers, loss of expression of genes occurs about 10 times more frequently by hypermethylation of promoter CpG islands than by mutations. For instance, in colon tumors compared to adjacent normal-appearing colonic mucosa, about 600 to 800 heavily methylated CpG islands occur in promoters of genes in the tumors while these CpG islands are not methylated in the adjacent mucosa.[11][12][13] In contrast, as Vogelstein et al.[14] point out, in a colorectal cancer there are typically only about 3 to 6 driver mutations and 33 to 66 hitchhiker or passenger mutations.

DNA repair gene silencing in cancer

In sporadic cancers, a DNA repair deficiency is occasionally found to be due to a mutation in a DNA repair gene. However, much more frequently, reduced or absent expression of a DNA repair gene in cancer is due to methylation of its promoter. For example, of 113 colorectal cancers examined, only four had a missense mutation in the DNA repair gene _MGMT_, while the majority had reduced _MGMT_ expression due to methylation of the _MGMT_ promoter region.[15] Similarly, among 119 cases of mismatch repair-deficient colorectal cancers that lacked DNA repair gene _PMS2_ expression, 6 had a mutation in the _PMS2_ gene, while for 103 PMS2 was deficient because its pairing partner MLH1 was repressed due to promoter methylation (PMS2 protein is unstable in the absence of MLH1).[16] In the remaining 10 cases, loss of PMS2 expression was likely due to epigenetic overexpression of the microRNA, miR-155, which down-regulates MLH1.[17]

Frequency of hypermethylation of DNA repair genes in cancer

Twenty-two DNA repair genes with hypermethylated promoters, and reduced or absent expression, were found to occur among 17 types of cancer, as listed in two review articles.[18][19] As listed in one of the reviews,[20] promoter hypermethylation of _MGMT_ occurs frequently in a number of cancers including 93% of bladder cancers, 88% of stomach cancers, 74% of thyroid cancers, 40%-90% of colorectal cancers and 50% of brain cancers. That review also indicated promoter hypermethylation of _LIG4_, _NEIL1_, _ATM_, _MLH1_ or _FANCB_ occurs at frequencies between 33% to 82% in one or more of head and neck cancers, non-small-cell lung cancers or non-small-cell lung cancer squamous cell carcinomas. The article Werner syndrome ATP-dependent helicase indicates the DNA repair gene _WRN_ has a promoter that is frequently hypermethylated in a number of cancers, with hypermethylation occurring in 11% to 38% of colorectal, head and neck, stomach, prostate, breast, thyroid, non-Hodgkin lymphoma, chondrosarcoma and osteosarcoma cancers (see WRN).


Likely role of hypermethylation of DNA repair genes in cancer

As discussed by Jin and Roberston in their review,[21] silencing of a DNA repair gene by hypermethylation may be a very early step in progression to cancer. Such silencing is proposed to act similarly to a germ-line mutation in a DNA repair gene, and predisposes the cell and its descendants to progression to cancer. Another review[22] also indicated an early role for hypermethylation of DNA repair genes in cancer. If a gene necessary for DNA repair is hypermethylated, resulting in deficient DNA repair, DNA damages will accumulate. Increased DNA damage tends to cause increased errors during DNA synthesis, leading to mutations that can give rise to cancer.

If hypermethylation of a DNA repair gene is an early step in carcinogenesis, then it may also occur in the normal-appearing tissues surrounding the cancer from which the cancer arose (the field defect). See the table below.

  Cancer                       Gene   Frequency in Cancer   Frequency in Field Defect   Ref.
  ---------------------------- ------ --------------------- --------------------------- ------
  Colorectal                   MGMT   55%                   54%                         [23]
  Colorectal                   MSH2   13%                   5%                          [24]
  Colorectal                   WRN    29%                   13%                         [25]
  Head and Neck                MGMT   54%                   38%                         [26]
  Head and Neck                MLH1   33%                   25%                         [27]
  Non-small cell lung cancer   ATM    69%                   59%                         [28]
  Non-small cell lung cancer   MLH1   69%                   72%                         [29]
  Stomach                      MGMT   88%                   78%                         [30]
  Stomach                      MLH1   73%                   20%                         [31]
  Esophagus                    MLH1   77%-100%              23%-79%                     [32]

  : Frequencies of hypermethylated promoters in DNA repair genes in sporadic cancers and in adjacent field defects

While DNA damages may give rise to mutations through error prone translesion synthesis, DNA damages can also give rise to epigenetic alterations during faulty DNA repair processes.[33][34][35][36] The DNA damages that accumulate due to hypermethylation of the promoters of DNA repair genes can be a source of the increased epigenetic alterations found in many genes in cancers.

In an early study, looking at a limited set of transcriptional promoters, Fernandez et al.[37] examined the DNA methylation profiles of 855 primary tumors. Comparing each tumor type with its corresponding normal tissue, 729 CpG island sites (55% of the 1322 CpG island sites evaluated) showed differential DNA methylation. Of these sites, 496 were hypermethylated (repressed) and 233 were hypomethylated (activated). Thus, there is a high level of promoter methylation alterations in tumors. Some of these alterations may contribute to cancer progression.


DNA methylation of microRNAs in cancer

In mammals, microRNAs (miRNAs) regulate the transcriptional activity of about 60% of protein-encoding genes.[38] Individual miRNAs can each target, and repress transcription of, on average, roughly 200 messenger RNAs of protein coding genes.[39] The promoters of about one third of the 167 miRNAs evaluated by Vrba et al.[40] in normal breast tissues were differentially hyper/hypo-methylated in breast cancers. A more recent study pointed out that the 167 miRNAs evaluated by Vrba et al. were only 10% of the miRNAs found expressed in breast tissues.[41] This later study found that 58% of the miRNAs in breast tissue had differentially methylated regions in their promoters in breast cancers, including 278 hypermethylated miRNAs and 802 hypomethylated miRNAs.

One miRNA that is over-expressed about 100-fold in breast cancers is miR-182.[42] MiR-182 targets the BRCA1 messenger RNA and may be a major cause of reduced BRCA1 protein expression in many breast cancers[43] (also see BRCA1).


microRNAs that control DNA methyltransferase genes in cancer

Some miRNAs target the messenger RNAs for DNA methyltransferase genes _DNMT1_, _DNMT3A_ and _DNMT3B_, whose gene products are needed for initiating and stabilizing promoter methylations. As summarized in three reviews,[44][45][46] miRNAs miR-29a, miR-29b and miR-29c target DNMT3A and DNMT3B; miR-148a and miR-148b target DNMT3B; and miR-152 and miR-301 target DNMT1. In addition, miR-34b targets DNMT1 and the promoter of miR-34b itself is hypermethylated and under-expressed in the majority of prostate cancers.[47] When expression of these microRNAs is altered, they may also be a source of the hyper/hypo-methylation of the promoters of protein-coding genes in cancers.


References

Category:Gene expression Category:Non-coding RNA Category:Epigenetics Category:Cancer epigenetics Category:DNA

[1]

[2]

[3]

[4]

[5]

[6]

[7]

[8]

[9]

[10]

[11]

[12]

[13]

[14]

[15]

[16]

[17]

[18] Carol Bernstein and Harris Bernstein (2015). Epigenetic Reduction of DNA Repair in Progression to Cancer, Advances in DNA Repair, Prof. Clark Chen (Ed.), , InTech, Available from: http://www.intechopen.com/books/advances-in-dna-repair/epigenetic-reduction-of-dna-repair-in-progression-to-cancer

[19]

[20]

[21]

[22]

[23]

[24]

[25]

[26]

[27]

[28]

[29]

[30]

[31]

[32]

[33]

[34]

[35]

[36]

[37]

[38]

[39]

[40]

[41]

[42]

[43]

[44]

[45]

[46]

[47]