Northern_blot_diagram.png The NORTHERN BLOT, or RNA blot,[1] is a technique used in molecular biology research to study gene expression by detection of RNA (or isolated mRNA) in a sample.[2][3]

With northern blotting it is possible to observe cellular control over structure and function by determining the particular gene expression rates during differentiation and morphogenesis, as well as in abnormal or diseased conditions.[4] Northern blotting involves the use of electrophoresis to separate RNA samples by size, and detection with a hybridization probe complementary to part of or the entire target sequence. The term 'northern blot' actually refers specifically to the capillary transfer of RNA from the electrophoresis gel to the blotting membrane. However, the entire process is commonly referred to as northern blotting.[5] The northern blot technique was developed in 1977 by James Alwine, David Kemp, and George Stark at Stanford University,[6] with contributions from Gerhard Heinrich. Northern blotting takes its name from its similarity to the first blotting technique, the Southern blot, named for biologist Edwin Southern.[7] The major difference is that RNA, rather than DNA, is analyzed in the northern blot.[8]


Procedure

A general blotting procedure[9] starts with extraction of total RNA from a homogenized tissue sample or from cells. Eukaryotic mRNA can then be isolated through the use of oligo (dT) cellulose chromatography to isolate only those RNAs with a poly(A) tail.[10][11] RNA samples are then separated by gel electrophoresis. Since the gels are fragile and the probes are unable to enter the matrix, the RNA samples, now separated by size, are transferred to a nylon membrane through a capillary or vacuum blotting system. capillary_blot_setup.svg A nylon membrane with a positive charge is the most effective for use in northern blotting since the negatively charged nucleic acids have a high affinity for them. The transfer buffer used for the blotting usually contains formamide because it lowers the annealing temperature of the probe-RNA interaction, thus eliminating the need for high temperatures, which could cause RNA degradation.[12] Once the RNA has been transferred to the membrane, it is immobilized through covalent linkage to the membrane by UV light or heat. After a probe has been labeled, it is hybridized to the RNA on the membrane. Experimental conditions that can affect the efficiency and specificity of hybridization include ionic strength, viscosity, duplex length, mismatched base pairs, and base composition.[13] The membrane is washed to ensure that the probe has bound specifically and to prevent background signals from arising. The hybrid signals are then detected by X-ray film and can be quantified by densitometry. To create controls for comparison in a northern blot, samples not displaying the gene product of interest can be used after determination by microarrays or RT-PCR.[14]

Gels

RNA_agarose_gel.svg The RNA samples are most commonly separated on agarose gels containing formaldehyde as a denaturing agent for the RNA to limit secondary structure.[15][16] The gels can be stained with ethidium bromide (EtBr) and viewed under UV light to observe the quality and quantity of RNA before blotting.[17] Polyacrylamide gel electrophoresis with urea can also be used in RNA separation but it is most commonly used for fragmented RNA or microRNAs.[18] An RNA ladder is often run alongside the samples on an electrophoresis gel to observe the size of fragments obtained but in total RNA samples the ribosomal subunits can act as size markers.[19] Since the large ribosomal subunit is 28S (approximately 5kb) and the small ribosomal subunit is 18S (approximately 2kb) two prominent bands appear on the gel, the larger at close to twice the intensity of the smaller.[20][21]

Probes

Probes for northern blotting are composed of nucleic acids with a complementary sequence to all or part of the RNA of interest, they can be DNA, RNA, or oligonucleotides with a minimum of 25 complementary bases to the target sequence.[22] RNA probes (riboprobes) that are transcribed in vitro are able to withstand more rigorous washing steps preventing some of the background noise.[23] Commonly cDNA is created with labelled primers for the RNA sequence of interest to act as the probe in the northern blot.[24] The probes must be labelled either with radioactive isotopes (³²P) or with chemiluminescence in which alkaline phosphatase or horseradish peroxidase (HRP) break down chemiluminescent substrates producing a detectable emission of light.[25] The chemiluminescent labelling can occur in two ways: either the probe is attached to the enzyme, or the probe is labelled with a ligand (e.g. biotin) for which the ligand (e.g., avidin or streptavidin) is attached to the enzyme (e.g. HRP).[26] X-ray film can detect both the radioactive and chemiluminescent signals and many researchers prefer the chemiluminescent signals because they are faster, more sensitive, and reduce the health hazards that go along with radioactive labels.[27] The same membrane can be probed up to five times without a significant loss of the target RNA.[28]


Applications

Northern blotting allows one to observe a particular gene's expression pattern between tissues, organs, developmental stages, environmental stress levels, pathogen infection, and over the course of treatment.[29][30][31] The technique has been used to show overexpression of oncogenes and downregulation of tumor-suppressor genes in cancerous cells when compared to 'normal' tissue,[32] as well as the gene expression in the rejection of transplanted organs.[33] If an upregulated gene is observed by an abundance of mRNA on the northern blot the sample can then be sequenced to determine if the gene is known to researchers or if it is a novel finding.[34] The expression patterns obtained under given conditions can provide insight into the function of that gene. Since the RNA is first separated by size, if only one probe type is used variance in the level of each band on the membrane can provide insight into the size of the product, suggesting alternative splice products of the same gene or repetitive sequence motifs.[35][36] The variance in size of a gene product can also indicate deletions or errors in transcript processing. By altering the probe target used along the known sequence it is possible to determine which region of the RNA is missing.[37]

BlotBase is an online database publishing northern blots. BlotBase has over 700 published northern blots of human and mouse samples, in over 650 genes across more than 25 different tissue types.[38] Northern blots can be searched by a blot ID, paper reference, gene identifier, or by tissue.[39] The results of a search provide the blot ID, species, tissue, gene, expression level, blot image (if available), and links to the publication that the work originated from.[40] This new database provides sharing of information between members of the science community that was not previously seen in northern blotting as it was in sequence analysis, genome determination, protein structure, etc.


Advantages and disadvantages

Analysis of gene expression can be done by several different methods including RT-PCR, RNase protection assays, microarrays, RNA-Seq, serial analysis of gene expression (SAGE), as well as northern blotting.[41][42] Microarrays are quite commonly used and are usually consistent with data obtained from northern blots; however, at times northern blotting is able to detect small changes in gene expression that microarrays cannot.[43] The advantage that microarrays have over northern blots is that thousands of genes can be visualized at a time, while northern blotting is usually looking at one or a small number of genes.[44][45]

A problem in northern blotting is often sample degradation by RNases (both endogenous to the sample and through environmental contamination), which can be avoided by proper sterilization of glassware and the use of RNase inhibitors such as DEPC (diethylpyrocarbonate).[46] The chemicals used in most northern blots can be a risk to the researcher, since formaldehyde, radioactive material, ethidium bromide, DEPC, and UV light are all harmful under certain exposures.[47] Compared to RT-PCR, northern blotting has a low sensitivity, but it also has a high specificity, which is important to reduce false positive results.[48]

The advantages of using northern blotting include the detection of RNA size, the observation of alternate splice products, the use of probes with partial homology, the quality and quantity of RNA can be measured on the gel prior to blotting, and the membranes can be stored and reprobed for years after blotting.[49]

For northern blotting for the detection of acetylcholinesterase mRNA the nonradioactive technique was compared to a radioactive technique and found as sensitive as the radioactive one, but requires no protection against radiation and is less time consuming.[50]


Reverse northern blot

Researchers occasionally use a variant of the procedure known as the reverse northern blot. In this procedure, the substrate nucleic acid (that is affixed to the membrane) is a collection of isolated DNA fragments, and the probe is RNA extracted from a tissue and radioactively labelled. The use of DNA microarrays that have come into widespread use in the late 1990s and early 2000s is more akin to the reverse procedure, in that they involve the use of isolated DNA fragments affixed to a substrate, and hybridization with a probe made from cellular RNA. Thus the reverse procedure, though originally uncommon, enabled northern analysis to evolve into gene expression profiling, in which many (possibly all) of the genes in an organism may have their expression monitored.


See also

-   Southern blot
-   Western blot
-   Northwestern blot
-   Far-Eastern blotting
-   Far-western blotting
-   Eastern blot
-   Differential display


References


External links

-   OpenWetWare

Category:Molecular biology techniques

[1] Gilbert, S. F. (2000) Developmental Biology, 6th Ed. Sunderland MA, Sinauer Associates.

[2]

[3] Kevil, C. G., Walsh, L., Laroux, F. S., Kalogeris, T., Grisham, M. B., Alexander, J. S. (1997) An Improved, Rapid Northern Protocol. Biochem. and Biophys. Research Comm. 238:277–279.

[4]

[5] Trayhurn, P. (1996) Northern Blotting. Pro. Nutrition Soc. 55:583–589.

[6]

[7] Alberts, B., Johnson, A., Lewis, J. Raff, M., Roberts, K., Walter, P. 2008. Molecular Biology of the Cell, 5th ed. Garland Science, Taylor & Francis Group, NY, pp 538–539.

[8]

[9]

[10]

[11]

[12]

[13]

[14]

[15]

[16]

[17]

[18] Valoczi, A., Hornyik, C., Varga, N., Burgyan, J., Kauppinen, S., Havelda, Z. (2004) Sensitive and specific detection of microRNAs by northern blot analysis using LNA-modified oligonucleotide probes. Nuc. Acids Research. 32: e175.

[19]

[20]

[21]

[22]

[23]

[24] Liang, P. Pardee, A. B. (1995) Recent advances in differential display. Current Opinion Immunol. 7: 274–280.

[25]

[26]

[27]

[28]

[29]

[30]

[31] Baldwin, D., Crane, V., Rice, D. (1999) A comparison of gel-based, nylon filter and microarray techniques to detect differential RNA expression in plants. Current Opinion in Plant Biol. 2: 96–103.

[32]

[33]

[34]

[35]

[36]

[37]

[38]

[39]

[40]

[41]

[42]

[43]

[44]

[45]

[46]

[47]

[48]

[49]

[50] Kreft, K., Kreft, S., Komel, R., Grubič, Z. (2000). Nonradioactive northern blotting for the determination of acetylcholinesterase mRNA. Pflügers Arch – Eur J Physiol, 439:R66-R67