PEPTIDOGLYCAN(MUREIN) is a polymer consisting of sugars and amino acids that forms a mesh-like layer outside the plasma membrane of most bacteria, forming the cell wall. The sugar component consists of alternating residues of β-(1,4) linked _N_-acetylglucosamine (NAG) and _N_-acetylmuramic acid (NAM). Attached to the _N_-acetylmuramic acid is a peptide chain of three to five amino acids. The peptide chain can be cross-linked to the peptide chain of another strand forming the 3D mesh-like layer.[1] Peptidoglycan serves a structural role in the bacterial cell wall, giving structural strength, as well as counteracting the osmotic pressure of the cytoplasm. Peptidoglycan is also involved in binary fission during bacterial cell reproduction.

The peptidoglycan layer is substantially thicker in Gram-positive bacteria (20 to 80 nanometers) than in Gram-negative bacteria (7 to 8 nanometers). Peptidoglycan forms around 90% of the dry weight of Gram-positive bacteria but only 10% of Gram-negative strains. Thus, presence of high levels of peptidoglycan is the primary determinant of the characterisation of bacteria as Gram-positive.[2] In Gram-positive strains, it is important in attachment roles and serotyping purposes.[3] For both Gram-positive and Gram-negative bacteria, particles of approximately 2 nm can pass through the peptidoglycan.[4]


Structure

The peptidoglycan layer in the bacterial cell wall is a crystal lattice structure formed from linear chains of two alternating amino sugars, namely _N_-acetylglucosamine (GlcNAc or NAGA) and _N_-acetylmuramic acid (MurNAc or NAMA). The alternating sugars are connected by a β-(1,4)-glycosidic bond. Each MurNAc is attached to a short (4- to 5-residue) amino acid chain, containing L-alanine, D-glutamic acid, _meso_-diaminopimelic acid, and D-alanine in the case of _Escherichia coli_ (a Gram-negative bacterium) or L-alanine, D-glutamine, L-lysine, and D-alanine with a 5-glycine interbridge between tetrapeptides in the case of _Staphylococcus aureus_ (a Gram-positive bacterium). Peptidoglycan is one of the most important sources of D-amino acids in nature.

Cross-linking between amino acids in different linear amino sugar chains occurs with the help of the enzyme DD-transpeptidase and results in a 3-dimensional structure that is strong and rigid. The specific amino acid sequence and molecular structure vary with the bacterial species.[5]

File:Mureine.svg%7CThe structure of peptidoglycan. NAG = _N_-acetylglucosamine (also called GlcNAc or NAGA), NAM = _N_-acetylmuramic acid (also called MurNAc or NAMA). File:Gram-positive cellwall-schematic.png|Gram-positive cell wall File:PBP catalysis.svg|Penicillin binding protein forming cross-links in newly formed bacterial cell wall.


Biosynthesis

The peptidoglycan monomers are synthesized in the cytosol and are then attached to a membrane carrier bactoprenol. Bactoprenol transports peptidoglycan monomers across the cell membrane where they are inserted into the existing peptidoglycan.[6]

In the first step of peptidoglycan synthesis, glutamine, which is an amino acid, donates an amino group to a sugar, fructose 6-phosphate. This turns fructose 6-phosphate into glucosamine-6-phosphate. In step two, an acetyl group is transferred from acetyl CoA to the amino group on the glucosamine-6-phosphate creating _N_-acetyl-glucosamine-6-phosphate.[7] In step three of the synthesis process, the _N_-acetyl-glucosamine-6-phosphate is isomerized, which will change _N_-acetyl-glucosamine-6-phosphate to _N_-acetyl-glucosamine-1-phosphate.[8]

In step 4, the _N_-acetyl-glucosamine-1-phosphate, which is now a monophosphate, attacks UTP. Uridine triphosphate, which is a pyrimidine nucleotide, has the ability to act as an energy source. In this particular reaction, after the monophosphate has attacked the UTP, an inorganic pyrophosphate is given off and is replaced by the monophosphate, creating UDP-N-acetylglucosamine (2,4). (When UDP is used as an energy source, it gives off an inorganic phosphate.) This initial stage, is used to create the precursor for the NAG in peptidoglycan.

In step 5, some of the UDP-N-acetylglucosamine (UDP-GlcNAc) is converted to UDP-MurNAc (UDP-N-acetylmuramic acid) by the addition of a lactyl group to the glucosamine. Also in this reaction, the C3 hydroxyl group will remove a phosphate from the alpha carbon of phosphoenolpyruvate. This creates what is called an enol derivative that will be reduced to a “lactyl moiety” by NADPH in step six.[9]

In step 7, the UDP–MurNAc is converted to UDP-MurNAc pentapeptide by the addition of five amino acids, usually including the dipeptide D-alanyl-D-alanine.[10] Each of these reactions requires the energy source ATP.[11] This is all referred to as Stage one.

Stage two occurs in the cytoplasmic membrane. It is in the membrane where a lipid carrier called bactoprenol carries peptidoglycan precursors through the cell membrane. Bactoprenol will attack the UDP-MurNAc penta, creating a PP-MurNac penta, which is now a lipid. UDP-GlcNAc is then transported to MurNAc, creating Lipid-PP-MurNAc penta-GlcNAc, a disaccharide, also a precursor to peptidoglycan.[12] How this molecule is transported through the membrane is still not understood. However, once it is there, it is added to the growing glycan chain.[13] The next reaction is known as tranglycosylation. In the reaction, the hydroxyl group of the GlcNAc will attach to the MurNAc in the glycan, which will displace the lipid-PP from the glycan chain. The enzyme responsible for this is transglycosylase.[14]


Inhibition

Some antibacterial drugs such as penicillin interfere with the production of peptidoglycan by binding to bacterial enzymes known as penicillin-binding proteins or DD-transpeptidases.[15] Penicillin-binding proteins form the bonds between oligopeptide crosslinks in peptidoglycan. For a bacterial cell to reproduce through binary fission, more than a million peptidoglycan subunits (NAM-NAG+oligopeptide) must be attached to existing subunits.[16] Mutations in genes coding for transpeptidases that lead to reduced interactions with an antibiotic are a significant source of emerging antibiotic resistance.[17]

Lysozyme, which is found in tears and constitutes part of the body's innate immune system exerts its antibacterial effect by breaking the β-(1,4)-glycosidic bonds in peptidoglycan (see above).


Similarity to pseudopeptidoglycan

Some archaea have a similar layer of pseudopeptidoglycan (also known as pseudomurein), in which the sugar residues are β-(1,3) linked _N_-acetylglucosamine and _N_-acetyltalosaminuronic acid. This makes the cell walls of such archaea insensitive to lysozyme.[18]


References


External links

-   Diagrammatic representation of peptidoglycan structures.
-   Structure of MurNAc 6-Phosphate Hydrolase (MurQ) from Haemophilus influenzae with a Bound Inhibitor.

Category:Membrane biology Category:Glycobiology

[1] Animation of Synthesis of Peptidoglycan Layer

[2] C.Michael Hogan. 2010. _Bacteria_. Encyclopedia of Earth. eds. Sidney Draggan and C.J.Cleveland, National Council for Science and the Environment, Washington DC

[3]

[4]

[5]

[6]

[7]

[8]

[9]

[10]

[11]

[12]

[13]

[14]

[15]

[16]

[17]

[18] Madigan, M. T., J. M. Martinko, P. V. Dunlap, and D. P. Clark. Brock biology of microorganisms. 12th ed. San Francisco, CA: Pearson/Benjamin Cummings, 2009.