In biology, PHENETICS ( - to appear) , also known as TAXIMETRICS, is an attempt to classify organisms based on overall similarity, usually in morphology or other observable traits, regardless of their phylogeny or evolutionary relation. It is closely related to numerical taxonomy which is concerned with the use of numerical methods for taxonomic classification. Many people contributed to the development of phenetics, but the most influential were Peter Sneath and Robert R. Sokal. Their books are still primary references for this sub-discipline, although now out of print.[1]

Phenetics has largely been superseded by cladistics for research into evolutionary relationships among species. However, certain phenetic methods, such as neighbor-joining, have found their way into phylogenetics, as a reasonable approximation of phylogeny when more advanced methods (such as Bayesian inference) are too computationally expensive.

Phenetic techniques include various forms of clustering and ordination. These are sophisticated ways of reducing the variation displayed by organisms to a manageable level. In practice this means measuring dozens of variables, and then presenting them as two- or three-dimensional graphs. Much of the technical challenge in phenetics revolves around balancing the loss of information in such a reduction against the ease of interpreting the resulting graphs.

The method can be traced back to 1763 and Michel Adanson (in his _Familles des plantes_) because of two shared basic principles — overall similarity and equal weighting — and modern pheneticists are sometimes called neo-Adansonians.[2]


Difference from cladistics

Phenetic analyses are unrooted, that is, they do not distinguish between plesiomorphies, traits that are inherited from an ancestor, and apomorphies, traits that evolved anew in one or several lineages. A common problem with phenetic analysis is that basal evolutionary grades, which retain many plesiomorphies compared to more advanced lineages, appear to be monophyletic. Phenetic analyses are also liable to be misled by convergent evolution and adaptive radiation. Cladistic methods have attempted to solve those problems.

Consider for example songbirds. These can be divided into two groups – Corvida, which retains ancient characters in phenotype and genotype, and Passerida, which has more modern traits. But only the latter are a group of closest relatives; the former are numerous independent and ancient lineages which are about as distantly related to each other as each single one of them is to the Passerida. In a phenetic analysis, the large degree of overall similarity found among the Corvida will make them appear to be monophyletic too, but their shared traits were present in the ancestors of _all_ songbirds already. It is the loss of these ancestral traits rather than their presence that signifies which songbirds are more closely related to each other than to other songbirds. However, the requirement that taxa be monophyletic – rather than paraphyletic as in the case of the Corvida – is itself part of the cladistic view of Taxonomy, not necessarily followed to an absolute degree by other schools.

The two methodologies are not mutually exclusive. There is no reason why, e.g., species identified using phenetics cannot subsequently be subjected to cladistic analysis, to determine their evolutionary relationships. Phenetic methods can also be superior to cladistics when only the _distinctness_ of related taxa is important, as the computational requirements are lower.[3]

The history of pheneticism and cladism as rival taxonomic systems is analysed in David Hull's 1988 book _Science as a Process_.[4]


Today

Traditionally there was a great deal of heated debate between pheneticists and cladists, as both methods were initially proposed to resolve evolutionary relationships. Perhaps the "high-water mark" of phenetics were the DNA-DNA hybridization studies by Charles G. Sibley, Jon E. Ahlquist and Burt L. Monroe Jr., from which resulted the 1990 Sibley-Ahlquist taxonomy for birds. Highly controversial at its time, some of its findings (e.g. the Galloanserae) have been vindicated, while others (e.g. the all-inclusive "Ciconiiformes" or the "Corvida") have been rejected. However, with computers growing increasingly powerful and widespread, more refined cladistic algorithms became available and could put the suggestions of Willi Hennig to the test; as it turned out, the results of cladistic analyses turned out to be superior to those of phenetic methods – at least when it came to resolving phylogenies.

Many systematists continue to use phenetic methods, particularly in addressing species-level questions. While a major goal of taxonomy remains describing the 'tree of life' – the evolutionary path connecting all species – in fieldwork one needs to be able to separate one taxon from another. Classifying diverse groups of closely related organisms that differ very subtly is difficult using a cladistic approach. Phenetics provides numerical tools for examining overall patterns of variation, allowing researchers to identify discrete groups that can be classified as species.

Modern applications of phenetics are common in botany, and some examples can be found in most issues of the journal _Systematic Botany_. Indeed, due to the effects of horizontal gene transfer, polyploid complexes and other peculiarities of plant genomics, phenetic techniques in botany – though less informative altogether – may, in these special cases, be less prone to errors compared with cladistic analysis of DNA sequences.

In addition, many of the techniques developed by phenetic taxonomists have been adopted and extended by community ecologists, due to a similar need to deal with large amounts of data.[5]


See also

-   Folk taxonomy
-   Form classification
-   Linnaean taxonomy
-   Taxonomy


References

Category:Biological classification

[1] Sneath, P. H. A. & R. R. Sokal. 1973. _Numerical taxonomy — The principles and practice of numerical classification_. W. H. Freeman, San Francisco. xv + 573 p.

[2] Schuh, Randall. 2000. Biological Systematics, p. 6. Cornell U. Press.

[3]

[4]

[5] Legendre, Pierre & Louis Legendre. 1998. Numerical ecology. 2nd English edition. Elsevier Science BV, Amsterdam. xv + 853 pages.