_1_: RNA POLYMERASE, _2_: REPRESSOR, _3_: PROMOTER, _4_: OPERATOR, _5_: LACTOSE, _6_: LACZ, _7_: LACY, _8_: LACA. TOP: The gene is essentially turned off. There is no lactose to inhibit the repressor, so the repressor binds to the operator, which obstructs the RNA polymerase from binding to the promoter and making lactase. BOTTOM: The gene is turned on. Lactose is inhibiting the repressor, allowing the RNA polymerase to bind with the promoter and express the genes, which synthesize lactase. Eventually, the lactase will digest all of the lactose, until there is none to bind to the repressor. The repressor will then bind to the operator, stopping the manufacture of lactase.

In genetics, a PROMOTER is a region of DNA that leads to initiation of transcription of a particular gene. Promoters are located near the transcription start sites of genes, upstream on the DNA (towards the 5' region of the sense strand). Promoters can be about 100–1000 base pairs long.[1]


Overview

For transcription to take place, the enzyme that synthesizes RNA, known as RNA polymerase, must attach to the DNA near a gene. Promoters contain specific DNA sequences such as response elements that provide a secure initial binding site for RNA polymerase and for proteins called transcription factors that recruit RNA polymerase. These transcription factors have specific activator or repressor sequences of corresponding nucleotides that attach to specific promoters and regulate gene expression.

In bacteria: The promoter is recognized by RNA polymerase and an associated sigma factor, which in turn are often brought to the promoter DNA by an activator protein's binding to its own DNA binding site nearby.
In eukaryotes: The process is more complicated, and at least seven different factors are necessary for the binding of an RNA polymerase II to the promoter.

Promoters represent critical elements that can work in concert with other regulatory regions (enhancers, silencers, boundary elements/insulators) to direct the level of transcription of a given gene. A promoter is induced in response to changes in abundance or conformation of regulatory proteins in a cell, which enable activating transcription factors to recruit RNA polymerase.[2][3]


Identification of relative location

As promoters are typically immediately adjacent to the gene in question, positions in the promoter are designated relative to the transcriptional start site, where transcription of DNA begins for a particular gene (i.e., positions upstream are negative numbers counting back from -1, for example -100 is a position 100 base pairs upstream).


Relative location in the cell nucleus

In the cell nucleus, it seems that promoters are distributed preferentially at the edge of the chromosomal territories, likely for the co-expression of genes on different chromosomes.[4] Furthermore, in humans, promoters show certain structural features characteristic for each chromosome.[5]


Elements

Eukaryotic

-   Core promoter – the minimal portion of the promoter required to properly initiate transcription[6]
    -   Includes the transcription start site (TSS) and elements directly upstream
    -   A binding site for RNA polymerase
        -   RNA polymerase I: transcribes genes encoding 18S, 5.8S and 28S ribosomal RNAs
        -   RNA polymerase II: transcribes genes encoding messenger RNA and certain small nuclear RNAs and microRNA
        -   RNA polymerase III: transcribes genes encoding transfer RNA, 5s ribosomal RNAs and other small RNAs
    -   General transcription factor binding sites, e.g. TATA box, B recognition element.
    -   Many other elements/motifs may be present. There is no such thing as a set of "universal elements" found in every core promoter.[7]
-   Proximal promoter – the proximal sequence upstream of the gene that tends to contain primary regulatory elements
    -   Approximately 250 base pairs upstream of the start site
    -   Specific transcription factor binding sites
-   Distal promoter – the distal sequence upstream of the gene that may contain additional regulatory elements, often with a weaker influence than the proximal promoter
    -   Anything further upstream (but not an enhancer or other regulatory region whose influence is positional/orientation independent)
    -   Specific transcription factor binding sites

Bacterial

In bacteria, the promoter contains two short sequence elements approximately 10 (Pribnow Box) and 35 nucleotides _upstream_ from the transcription start site.

-   The sequence at -10 (the -10 element) has the consensus sequence TATAAT.
-   The sequence at -35 (the -35 element) has the consensus sequence TTGACA.
-   The above consensus sequences, while conserved on average, are not found intact in most promoters. On average, only 3 to 4 of the 6 base pairs in each consensus sequence are found in any given promoter. Few natural promoters have been identified to date that possess intact consensus sequences at both the -10 and -35; artificial promoters with complete conservation of the -10 and -35 elements have been found to transcribe at lower frequencies than those with a few mismatches with the consensus.
-   The optimal spacing between the -35 and -10 sequences is 17 bp.
-   Some promoters contain one or more upstream promoter element (UP element) subsites[8] (consensus sequence 5'-AAAAAARNR-3' when centered in the -42 region; consensus sequence 5'-AWWWWWTTTTT-3' when centered in the -52 region; W = A or T; R = A or G; N = any base).[9]

The above promoter sequences are recognized only by RNA polymerase holoenzyme containing sigma-70. RNA polymerase holoenzymes containing other sigma factors recognize different core promoter sequences.

   <-- upstream                                                          downstream -->
5'-XXXXXXXPPPPPPXXXXXXPPPPPPXXXXGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGXXXX-3'
           -35       -10       Gene to be transcribed

Probability of occurrence of each nucleotide

 for -10 sequence
 T    A    T    A    A   T
77%  76%  60%  61%  56%  82%

 for -35 sequence
 T    T    G    A    C    A
69%  79%  61%  56%  54%  54%

Eukaryotic

Eukaryotic promoters are diverse and can be difficult to characterize, however, recent studies show that they are divided in more than 10 classes.[10]

Ten_classes_of_eukaryotic_promoters_and_their_representative_K-DNA_patterns.jpg'''. The representative eukaryotic promoter classes are shown in the following sections: (A) AT-based class, (B) CG-based class, (C) ATCG-compact class, (D) ATCG-balanced class, (E) ATCG-middle class, (F) ATCG-less class, (G) AT-less class, (H) CG-spike class, (I) CG-less class and (J) ATspike class.[11]]]

Gene promoters are typically located upstream of the gene and can have regulatory elements several kilobases away from the transcriptional start site (enhancers). In eukaryotes, the transcriptional complex can cause the DNA to bend back on itself, which allows for placement of regulatory sequences far from the actual site of transcription. Eukaryotic RNA-polymerase-II-dependent promoters can contain a TATA element (consensus sequence TATAAA), which is recognized by the general transcription factor TATA-binding protein (TBP); and a B recognition element (BRE), which is recognized by the general transcription factor TFIIB.[12][13][14] The TATA element and BRE typically are located close to the transcriptional start site (typically within 30 to 40 base pairs).

Eukaryotic promoter regulatory sequences typically bind proteins called transcription factors that are involved in the formation of the transcriptional complex. An example is the E-box (sequence CACGTG), which binds transcription factors in the basic helix-loop-helix (bHLH) family (e.g. BMAL1-Clock, cMyc).[15] Some promoters that are targeted by multiple transcription factors might achieve a hyperactive state, leading to increased transcriptional activity.[16]

Bidirectional (mammalian)

Bidirectional promoters are short (<1 kbp) intergenic regions of DNA between the 5' ends of the genes in a bidirectional gene pair.[17] A “bidirectional gene pair” refers to two adjacent genes coded on opposite strands, with their 5' ends oriented toward one another.[18] The two genes are often functionally related, and modification of their shared promoter region allows them to be co-regulated and thus co-expressed.[19] Bidirectional promoters are a common feature of mammalian genomes.[20] About 11% of human genes are bidirectionally paired.[21]

Bidirectionally paired genes in the Gene Ontology database shared at least one database-assigned functional category with their partners 47% of the time.[22] Microarray analysis has shown bidirectionally paired genes to be co-expressed to a higher degree than random genes or neighboring unidirectional genes.[23] Although co-expression does not necessarily indicate co-regulation, methylation of bidirectional promoter regions has been shown to downregulate both genes, and demethylation to upregulate both genes.[24] There are exceptions to this, however. In some cases (about 11%), only one gene of a bidirectional pair is expressed.[25] In these cases, the promoter is implicated in suppression of the non-expressed gene. The mechanism behind this could be competition for the same polymerases, or chromatin modification. Divergent transcription could shift nucleosomes to upregulate transcription of one gene, or remove bound transcription factors to downregulate transcription of one gene.[26]

Some functional classes of genes are more likely to be bidirectionally paired than others. Genes implicated in DNA repair are five times more likely to be regulated by bidirectional promoters than by unidirectional promoters. Chaperone proteins are three times more likely, and mitochondrial genes are more than twice as likely. Many basic housekeeping and cellular metabolic genes are regulated by bidirectional promoters.[27] The overrepresentation of bidirectionally paired DNA repair genes associates these promoters with cancer. Forty-five percent of human somatic oncogenes seem to be regulated by bidirectional promoters – significantly more than non-cancer causing genes. Hypermethylation of the promoters between gene pairs WNT9A/CD558500, CTDSPL/BC040563, and KCNK15/BF195580 has been associated with tumors.[28]

Certain sequence characteristics have been observed in bidirectional promoters, including a lack of TATA boxes, an abundance of CpG islands, and a symmetry around the midpoint of dominant Cs and As on one side and Gs and Ts on the other. A motif with the consensus sequence of TCTCGCGAGA, also called the CGCG element, was recently shown to drive PolII-driven bidirectional transcription in CpG islands.[29] CCAAT boxes are common, as they are in many promoters that lack TATA boxes. In addition, the motifs NRF-1, GABPA, YY1, and ACTACAnnTCCC are represented in bidirectional promoters at significantly higher rates than in unidirectional promoters. The absence of TATA boxes in bidirectional promoters suggests that TATA boxes play a role in determining the directionality of promoters, but counterexamples of bidirectional promoters do possess TATA boxes and unidirectional promoters without them indicates that they cannot be the only factor.[30]

Although the term "bidirectional promoter" refers specifically to promoter regions of mRNA-encoding genes, luciferase assays have shown that over half of human genes do not have a strong directional bias. Research suggests that non-coding RNAs are frequently associated with the promoter regions of mRNA-encoding genes. It has been hypothesized that the recruitment and initiation of RNA polymerase II usually begins bidirectionally, but divergent transcription is halted at a checkpoint later during elongation. Possible mechanisms behind this regulation include sequences in the promoter region, chromatin modification, and the spatial orientation of the DNA.[31]


Subgenomic

A subgenomic promoter is a promoter added to a virus for a specific heterologous gene, resulting in the formation of mRNA for that gene alone. Many positive-sense RNA viruses produce these subgenomic mRNAs (sgRNA) as one of the common infection techniques used by these viruses and generally transcribe late viral genes. Subgenomic promoters range from 24 nucleotide (Sindbis virus) to over 100 nucleotides (Beet necrotic yellow vein virus) and are usually found upstream of the transcription start.[32]


Detection

A wide variety of algorithms have been developed to facilitate detection of promoters in genomic sequence, and promoter prediction is a common element of many gene prediction methods. A promoter region is located before the -35 and -10 Consensus sequences. The closer the promoter region is to the consensus sequences the more often transcription of that gene will take place. There is not a set pattern for promoter regions as there are for consensus sequences.


Evolutionary change

Superposition_between_promoter_distributions.png Changes in promoter sequences are critical in evolution as indicated by the relatively stable number of genes in many lineages. For instance, most vertebrates have roughly the same number of protein-coding genes (about 20,000) which are often highly conserved in sequence, hence much of evolutionary change must come from changes in gene expression.[33][34]

De novo origin of promoters

Given the short sequences of most promoter elements, promoters can rapidly evolve from random sequences. For instance, in _E. coli_, ~60% of random sequences can evolve expression levels comparable to the wild-type lac promoter with only one mutation, and that ~10% of random sequences can serve as active promoters even without evolution.[35]


Diabetes

Other recent studies suggest that promoters of genes may be the primary cause of diabetes.[36] Promoters of genes associated with diabetes by Genome-wide association studies (GWAS) show specific DNA patterns for each phenotype.[37] This observation indicates that the promoters of these genes use specific transcription factors for each diabetes phenotype.[38]


Binding

The initiation of the transcription is a multistep sequential process that involves several mechanisms: promoter location, initial reversible binding of RNA polymerase, conformational changes in RNA polymerase, conformational changes in DNA, binding of nucleoside triphosphate (NTP) to the functional RNA polymerase-promoter complex, and nonproductive and productive initiation of RNA synthesis.[39]

The promoter binding process is crucial in the understanding of the process of gene expression.

Location

Although RNA polymerase holoenzyme shows high affinity to non-specific sites of the DNA, this characteristic does not allow us to clarify the process of promoter location.[40] This process of promoter location has been attributed to the structure of the holoenzyme to DNA and sigma 4 to DNA complexes.[41]


Diseases associated with aberrant function

Most diseases are heterogeneous in cause, meaning that one "disease" is often many different diseases at the molecular level, though symptoms exhibited and response to treatment may be identical. How diseases of different molecular origin respond to treatments is partially addressed in the discipline of pharmacogenomics.

Not listed here are the many kinds of cancers involving aberrant transcriptional regulation owing to creation of chimeric genes through pathological chromosomal translocation. Importantly, intervention in the number or structure of promoter-bound proteins is one key to treating a disease without affecting expression of unrelated genes sharing elements with the target gene.[42] Some genes whose change is not desirable are capable of influencing the potential of a cell to become cancerous.[43]


CpG islands in promoters

In humans, about 70% of promoters located near the transcription start site of a gene (proximal promoters) contain a CpG island.[44][45] CpG islands are generally 200 to 2000 base pairs long, have a C:G base pair content >50%, and have regions of DNA where a cytosine nucleotide is followed by a guanine nucleotide and this occurs frequently in the linear sequence of bases along its 5' → 3' direction.

Distal promoters also frequently contain CpG islands, such as the promoter of the DNA repair gene _ERCC1_, where the CpG island-containing promoter is located about 5,400 nucleotides upstream of the coding region of the _ERCC1_ gene.[46] CpG islands also occur frequently in promoters for functional noncoding RNAs such as microRNAs.


Methylation of CpG islands stably silences genes

In humans, DNA methylation occurs at the 5' position of the pyrimidine ring of the cytosine residues within CpG sites to form 5-methylcytosines. The presence of multiple methylated CpG sites in CpG islands of promoters causes stable silencing of genes.[47] Silencing of a gene may be initiated by other mechanisms, but this is often followed by methylation of CpG sites in the promoter CpG island to cause the stable silencing of the gene.[48]


Promoter CpG hyper/hypo-methylation in cancer

Generally, in progression to cancer, hundreds of genes are silenced or activated. Although silencing of some genes in cancers occurs by mutation, a large proportion of carcinogenic gene silencing is a result of altered DNA methylation (see DNA methylation in cancer). DNA methylation causing silencing in cancer typically occurs at multiple CpG sites in the CpG islands that are present in the promoters of protein coding genes.

Altered expressions of microRNAs also silence or activate many genes in progression to cancer (see microRNAs in cancer). Altered microRNA expression occurs through hyper/hypo-methylation of CpG sites in CpG islands in promoters controlling transcription of the microRNAs.

Silencing of DNA repair genes through methylation of CpG islands in their promoters appears to be especially important in progression to cancer (see methylation of DNA repair genes in cancer).


Canonical sequences and wild-type

The usage of the term canonical sequence to refer to a promoter is often problematic, and can lead to misunderstandings about promoter sequences. Canonical implies perfect, in some sense.

In the case of a transcription factor binding site, there may be a single sequence that binds the protein most strongly under specified cellular conditions. This might be called canonical.

However, natural selection may favor less energetic binding as a way of regulating transcriptional output. In this case, we may call the most common sequence in a population the wild-type sequence. It may not even be the most advantageous sequence to have under prevailing conditions.

Recent evidence also indicates that several genes (including the proto-oncogene c-myc) have G-quadruplex motifs as potential regulatory signals.


Diseases that may be associated with variations

Some cases of many genetic diseases are associated with variations in promoters or transcription factors.

Examples include:

-   Asthma[49][50]
-   Beta thalassemia[51]
-   Rubinstein-Taybi syndrome[52]


Constitutive vs regulated

Some promoters are called constitutive as they are active in all circumstances in the cell, while others are regulated, becoming active in the cell only in response to specific stimuli.


Use of the term

When referring to a promoter some authors actually mean promoter + operator; i.e., the lac promoter is IPTG inducible, meaning that besides the lac promoter, the lac operator is also present. If the lac operator were not present the IPTG would not have an inducible effect.[53] Another example is the Tac-Promoter system (Ptac). Notice how tac is written as a tac promoter, while in fact tac is actually both a promoter and an operator.[54]


See also

-   Activator (genetics)
-   Enhancer (genetics)
-   Glossary of gene expression terms
-   Operon
-   Regulation of gene expression
-   Repressor
-   Transcription factor
-   Promoter bashing


References


External links

-   ORegAnno – Open Regulatory Annotation Database
-   Identifying a Protein Binding Sites on DNA molecule YouTube tutorial video
-   Pleiades Promoter Project – a research project with an aim to generate 160 fully characterized, human DNA promoters of less than 4 kb (MiniPromoters) to drive gene expression in defined brain regions of therapeutic interests.
-   ENCODE threads Explorer RNA and chromatin modification patterns around promoters. Nature (journal)

Category:Gene expression

[1]

[2] Chromatin remodeling: from transcription to cancer.Cancer Genet. 2014 Sep;207(9):352-7.

[3] Promoter organization of the interferon-A genes differentially affects virus-induced expression and responsiveness to TBK1 and IKKepsilon. J Biol Chem. 2006 Feb 24;281(8):4856-66.

[4]

[5]

[6]

[7]

[8]

[9]

[10]

[11]

[12]

[13]

[14]

[15]

[16]

[17]

[18]

[19]

[20]

[21]

[22]

[23]

[24]

[25]

[26]

[27]

[28]

[29]

[30]

[31]

[32]

[33]

[34]

[35]

[36]

[37]

[38]

[39]

[40]

[41]

[42]

[43]

[44]

[45]

[46]

[47]

[48]

[49]

[50]

[51]

[52]

[53] Lac operon

[54]