, transcriptome, proteome, and metabolome (lipidome).]]

The PROTEOME is the entire set of proteins that is, or can be, expressed by a genome, cell, tissue, or organism at a certain time. It is the set of expressed proteins in a given type of cell or organism, at a given time, under defined conditions. Proteomics is the study of the proteome.


Systems

The term has been applied to several different types of biological systems. A CELLULAR PROTEOME is the collection of proteins found in a particular cell type under a particular set of environmental conditions such as exposure to hormone stimulation. It can also be useful to consider an organism's COMPLETE PROTEOME, which can be conceptualized as the complete set of proteins from all of the various cellular proteomes. This is very roughly the protein equivalent of the genome. The term "proteome" has also been used to refer to the collection of proteins in certain sub-cellular biological systems. For example, all of the proteins in a virus can be called a viral proteome.


History

Marc Wilkins coined the term _proteome_ [1] in 1994 in a symposium on "2D Electrophoresis: from protein maps to genomes" held in Siena in Italy. It appeared in print in 1995,[2] with the publication of part of his PhD thesis. Wilkins used the term to describe the entire complement of proteins expressed by a genome, cell, tissue or organism.


Size and contents

The proteome can be larger than the genome, especially in eukaryotes, as more than one protein can be produced from one gene due to alternative splicing (e.g. human proteome consists 92,179 proteins out of which 71,173 are splicing variants).[3] On the other hand, not all genes are translated to proteins, and many known genes encode only RNA which is the final functional product. Moreover, complete proteome size vary depending the kingdom of life. For instance, eukaryotes, bacteria, archaea and viruses have on average 15,145, 3,200, 2,358 and 42 proteins respectively encoded in their genomes.[4]

The term “dark proteome" coined by Perdigão and colleagues, defines regions of proteins that have no detectable sequence homology to other proteins of known three-dimensional structure and therefore cannot be modeled by homology. For 546,000 Swiss-Prot proteins, 44–54% of the proteome in eukaryotes and viruses was found to be "dark", compared with only ∼14% in archaea and bacteria.[5]


Methods to study the proteome

Numerous methods are available to study proteins, sets of proteins, or the whole proteome. In fact, proteins are often studied indirectly, e.g. using computational methods and analyses of genomes. Only a few examples are given below.

Separation techniques and electrophoresis

Proteomics, the study of the proteome, has largely been practiced through the separation of proteins by two dimensional gel electrophoresis. In the first dimension, the proteins are separated by isoelectric focusing, which resolves proteins on the basis of charge. In the second dimension, proteins are separated by molecular weight using SDS-PAGE. The gel is dyed with Coomassie Brilliant Blue or silver to visualize the proteins. Spots on the gel are proteins that have migrated to specific locations.

Mass spectrometry

Mass spectrometry has augmented proteomics.[6] Peptide mass fingerprinting identifies a protein by cleaving it into short peptides and then deduces the protein's identity by matching the observed peptide masses against a sequence database. Tandem mass spectrometry, on the other hand, can get sequence information from individual peptides by isolating them, colliding them with a non-reactive gas, and then cataloguing the fragment ions produced.[7]

In May 2014, a draft map of the human proteome was published in _Nature_.[8] This map was generated using high-resolution Fourier-transform mass spectrometry. This study profiled 30 histologically normal human samples resulting in the identification of proteins coded by 17,294 genes. This accounts for around 84% of the total annotated protein-coding genes.

Protein complementation assays and interaction screens

Protein fragment complementation assays are often used to detect protein–protein interactions. The yeast two-hybrid assay is the most popular of them but there are numerous variations, both used _in vitro_ and _in vivo_.


See also

-   Metabolome
-   Cytome
-   Bioinformatics
-   List of omics topics in biology
-   Plant Proteome Database
-   Transcriptome
-   Interactome
-   Human Proteome Project
-   BioPlex


References


External links

-   PIR database
-   UniProt database
-

Category:Proteomics

[1]

[2]

[3]

[4]

[5]

[6]

[7]

[8]