S11|11 (disambiguation){{!}}11 (plural)}} PROTEASOMES are protein complexes which degrade unneeded or damaged proteins by proteolysis, a chemical reaction that breaks peptide bonds. Enzymes that help such reactions are called proteases.

Proteasomes are part of a major mechanism by which cells regulate the concentration of particular proteins and degrade misfolded proteins. Proteins are tagged for degradation with a small protein called ubiquitin. The tagging reaction is catalyzed by enzymes called ubiquitin ligases. Once a protein is tagged with a single ubiquitin molecule, this is a signal to other ligases to attach additional ubiquitin molecules. The result is a _polyubiquitin chain_ that is bound by the proteasome, allowing it to degrade the tagged protein.[1] The degradation process yields peptides of about seven to eight amino acids long, which can then be further degraded into shorter amino acid sequences and used in synthesizing new proteins.[2]

Proteasomes are found inside all eukaryotes and archaea, and in some bacteria. In eukaryotes, proteasomes are located both in the nucleus and in the cytoplasm.[3]

In structure, the proteasome is a cylindrical complex containing a "core" of four stacked rings forming a central pore. Each ring is composed of seven individual proteins. The inner two rings are made of seven _β subunits_ that contain three to seven protease active sites. These sites are located on the interior surface of the rings, so that the target protein must enter the central pore before it is degraded. The outer two rings each contain seven _α subunits_ whose function is to maintain a "gate" through which proteins enter the barrel. These α subunits are controlled by binding to "cap" structures or _regulatory particles_ that recognize polyubiquitin tags attached to protein substrates and initiate the degradation process. The overall system of ubiquitination and proteasomal degradation is known as the UBIQUITIN-PROTEASOME SYSTEM .[4]

The proteasomal degradation pathway is essential for many cellular processes, including the cell cycle, the regulation of gene expression, and responses to oxidative stress. The importance of proteolytic degradation inside cells and the role of ubiquitin in proteolytic pathways was acknowledged in the award of the 2004 Nobel Prize in Chemistry to Aaron Ciechanover, Avram Hershko and Irwin Rose.[5]


Discovery

Before the discovery of the ubiquitin proteasome system, protein degradation in cells was thought to rely mainly on lysosomes, membrane-bound organelles with acidic and protease-filled interiors that can degrade and then recycle exogenous proteins and aged or damaged organelles.[6] However, work by Joseph Etlinger and Alfred Goldberg in 1977 on ATP-dependent protein degradation in reticulocytes, which lack lysosomes, suggested the presence of a second intracellular degradation mechanism.[7] This was shown in 1978 to be composed of several distinct protein chains, a novelty among proteases at the time. [8] Later work on modification of histones led to the identification of an unexpected covalent modification of the histone protein by a bond between a lysine side chain of the histone and the C-terminal glycine residue of ubiquitin, a protein that had no known function.[9] It was then discovered that a previously identified protein associated with proteolytic degradation, known as ATP-dependent proteolysis factor 1 (APF-1), was the same protein as ubiquitin.[10] The proteolytic activities of this system were isolated as a multi-protein complex originally called the multi-catalytic proteinase complex by Sherwin Wilk and Marion Orlowski.[11] Later, the ATP-dependent proteolytic complex that was responsible for ubiquitin-dependent protein degradation was discovered and was called the 26S proteasome.[12][13]

Much of the early work leading up to the discovery of the ubiquitin proteasome system occurred in the late 1970s and early 1980s at the Technion in the laboratory of Avram Hershko, where Aaron Ciechanover worked as a graduate student. Hershko's year-long sabbatical in the laboratory of Irwin Rose at the Fox Chase Cancer Center provided key conceptual insights, though Rose later downplayed his role in the discovery.[14] The three shared the 2004 Nobel Prize in Chemistry for their work in discovering this system.[15]

Although electron microscopy data revealing the stacked-ring structure of the proteasome became available in the mid-1980s,[16] the first structure of the proteasome core particle was not solved by X-ray crystallography until 1994.[17]


Structure and organization

The proteasome subcomponents are often referred to by their Svedberg sedimentation coefficient (denoted _S_). The proteasome most exclusively used in mammals is the cytosolic 26S proteasome, which is about 2000 kilodaltons (kDa) in molecular mass containing one 20S protein subunit and two 19S regulatory cap subunits. The core is hollow and provides an enclosed cavity in which proteins are degraded; openings at the two ends of the core allow the target protein to enter. Each end of the core particle associates with a 19S regulatory subunit that contains multiple ATPase active sites and ubiquitin binding sites; it is this structure that recognizes polyubiquitinated proteins and transfers them to the catalytic core.[18] An alternative form of regulatory subunit called the 11S particle can associate with the core in essentially the same manner as the 19S particle; the 11S may play a role in degradation of foreign peptides such as those produced after infection by a virus.[19]

20S core particle

The number and diversity of subunits contained in the 20S core particle depends on the organism; the number of distinct and specialized subunits is larger in multicellular than unicellular organisms and larger in eukaryotes than in prokaryotes. All 20S particles consist of four stacked heptameric ring structures that are themselves composed of two different types of subunits; α subunits are structural in nature, whereas β subunits are predominantly catalytic. The α subunits are pseudoenzymes homologous to β subunits. They are assembled with their N-termini adjacent to that of the β subunits.[20] The outer two rings in the stack consist of seven α subunits each, which serve as docking domains for the regulatory particles and the alpha subunits N-termini () form a gate that blocks unregulated access of substrates to the interior cavity.[21] The inner two rings each consist of seven β subunits and in their N-termini contain the protease active sites that perform the proteolysis reactions.[22] Three distinct catalytic activities were identified in the purified complex: chymotrypsin-like, trypsin-like and peptidylglutamyl-peptide hydrolyzing.[23] The size of the proteasome is relatively conserved and is about 150 angstroms (Å) by 115 Å. The interior chamber is at most 53 Å wide, though the entrance can be as narrow as 13 Å, suggesting that substrate proteins must be at least partially unfolded to enter.[24]

In archaea such as _Thermoplasma acidophilum_, all the α and all the β subunits are identical, whereas eukaryotic proteasomes such as those in yeast contain seven distinct types of each subunit. In mammals, the β1, β2, and β5 subunits are catalytic; although they share a common mechanism, they have three distinct substrate specificities considered chymotrypsin-like, trypsin-like, and peptidyl-glutamyl peptide-hydrolyzing (PHGH).[25] Alternative β forms denoted β1i, β2i, and β5i can be expressed in hematopoietic cells in response to exposure to pro-inflammatory signals such as cytokines, in particular, interferon gamma. The proteasome assembled with these alternative subunits is known as the _immunoproteasome_, whose substrate specificity is altered relative to the normal proteasome.[26] Recently an alternative proteasome was identified in human cells that lack the α3 core subunit.[27] These proteasomes (known as the α4-α4 proteasomes) instead form 20S core particles containing an additional α4 subunit in place of the missing α3 subunit. These alternative 'α4-α4' proteasomes have been known previously to exist in yeast.[28] Although the precise function of these proteasome isoforms is still largely unknown, cells expressing these proteasomes show enhanced resistance to toxicity induced by metallic ions such as cadmium.[29][30]

19S regulatory particle

The 19S particle in eukaryotes consists of 19 individual proteins and is divisible into two subassemblies, a 9-subunit base that binds directly to the α ring of the 20S core particle, and a 10-subunit lid. Six of the nine base proteins are ATPase subunits from the AAA Family, and an evolutionary homolog of these ATPases exists in archaea, called PAN (Proteasome-Activating Nucleotidase).[31] The association of the 19S and 20S particles requires the binding of ATP to the 19S ATPase subunits, and ATP hydrolysis is required for the assembled complex to degrade folded and ubiquitinated proteins. Note that only the step of substrate unfolding requires energy from ATP hydrolysis, while ATP-binding alone can support all the other steps required for protein degradation (e.g., complex assembly, gate opening, translocation, and proteolysis).[32][33] In fact, ATP binding to the ATPases by itself supports the rapid degradation of unfolded proteins. However, while ATP hydrolysis is required for unfolding only, it is not yet clear whether this energy may be used in the coupling of some of these steps.[34][35]

In 2012, two independent efforts have elucidated the molecular architecture of the 26S proteasome by single particle electron microscopy.[36][37] In 2016, three independent efforts have determined the first near-atomic resolution structure of the human 26S proteasome in the absence of substrates by cryo-EM.[38][39][40] In 2018, a major effort has elucidated the detailed mechanisms of deubiquitylation, initiation of translocation and processive unfolding of substrates by determining seven atomic structures of substrate-engaged 26S proteasome simultaneously.[41] In the heart of the 19S, directly adjacent to the 20S, are the AAA-ATPases (AAA proteins) that assemble to a heterohexameric ring of the order Rpt1/Rpt2/Rpt6/Rpt3/Rpt4/Rpt5. This ring is a trimer of dimers: Rpt1/Rpt2, Rpt6/Rpt3, and Rpt4/Rpt5 dimerize via their N-terminal coiled-coils. These coiled-coils protrude from the hexameric ring. The largest regulatory particle non-ATPases Rpn1 and Rpn2 bind to the tips of Rpt1/2 and Rpt6/3, respectively. The ubiquitin receptor Rpn13 binds to Rpn2 and completes the base cub-complex. The lid covers one half of the AAA-ATPase hexamer (Rpt6/Rpt3/Rpt4) and, unexpectedly, directly contacts the 20S via Rpn6 and to lesser extent Rpn5. The subunits Rpn9, Rpn5, Rpn6, Rpn7, Rpn3, and Rpn12, which are structurally related among themselves and to subunits of the COP9 complex and eIF3 (hence called PCI subunits) assemble to a horseshoe-like structure enclosing the Rpn8/Rpn11 heterodimer. Rpn11, the deubiquinating enzyme, is placed at the mouth of the AAA-ATPase hexamer, ideally positioned to remove ubiquitin moieties immediately before translocation of substrates into the 20S. The second ubiquitin receptor identified to date, Rpn10, is positioned at the periphery of the lid, near subunits Rpn8 and Rpn9.

Conformational changes of 19S

The 19S regulatory particle within the 26S proteasome holoenzyme has been observed in six strongly differing conformational states in the absence of substrates to date.[42][43] A hallmark of the AAA-ATPase configuration in this predominant low-energy state is a staircase- or lockwasher-like arrangement of the AAA-domains.[44][45] In the presence of ATP but absence of substrate three alternative, less abundant conformations of the 19S are adopted primarily differing in the positioning of the lid with respect to the AAA-ATPase module.[46][47] In the presence of ATP-γS or a substrate, considerably more conformations have been observed displaying dramatic structural changes of the AAA-ATPase module.[48][49][50][51]. Some of the substrate-bound conformations bear high similarity to the substrate-free ones, but they are not entirely identical, particularly in the AAA-ATPase module.[52][53] Prior to the 26S assembly, the 19S regulatory particle in a free form has also been observed in seven conformational states.[54] Notably, all these conformers are somewhat different and present distinct features. Thus, the 19S regulatory particle can sample at least 20 conformational states under different physiological conditions.

Regulation of the 20S by the 19S

The 19S regulatory particle is responsible for stimulating the 20S to degrade proteins. A primary function of the 19S regulatory ATPases is to open the gate in the 20S that blocks the entry of substrates into the degradation chamber.[55] The mechanism by which the proteasomal ATPase open this gate has been recently elucidated.[56] 20S gate opening, and thus substrate degradation, requires the C-termini of the proteasomal ATPases, which contains a specific motif (i.e., HbYX motif). The ATPases C-termini bind into pockets in the top of the 20S, and tether the ATPase complex to the 20S proteolytic complex, thus joining the substrate unfolding equipment with the 20S degradation machinery. Binding of these C-termini into these 20S pockets by themselves stimulates opening of the gate in the 20S in much the same way that a "key-in-a-lock" opens a door.[57] The precise mechanism by which this "key-in-a-lock" mechanism functions has been structurally elucidated in the context of human 26S proteasome at near-atomic resolution, suggesting that the insertion of five C-termini of ATPase subunits Rpt1/2/3/5/6 into the 20S surface pockets are required to fully open the 20S gate.[58][59][60]

Other regulatory particles

20S proteasomes can also associate with a second type of regulatory particle, the 11S regulatory particle, a heptameric structure that does not contain any ATPases and can promote the degradation of short peptides but not of complete proteins. It is presumed that this is because the complex cannot unfold larger substrates. This structure is also known as PA28, REG, or PA26.[61] The mechanisms by which it binds to the core particle through the C-terminal tails of its subunits and induces α-ring conformational changes to open the 20S gate suggest a similar mechanism for the 19S particle.[62] The expression of the 11S particle is induced by interferon gamma and is responsible, in conjunction with the immunoproteasome β subunits, for the generation of peptides that bind to the major histocompatibility complex.[63]

Yet another type of non-ATPase regulatory particle is the Blm10 (yeast) or PA200/PSME4 (human). It opens only one α subunit in the 20S gate and itself folds into a dome with a very small pore over it.[64]


Assembly

The assembly of the proteasome is a complex process due to the number of subunits that must associate to form an active complex. The β subunits are synthesized with N-terminal "propeptides" that are post-translationally modified during the assembly of the 20S particle to expose the proteolytic active site. The 20S particle is assembled from two half-proteasomes, each of which consists of a seven-membered pro-β ring attached to a seven-membered α ring. The association of the β rings of the two half-proteasomes triggers threonine-dependent autolysis of the propeptides to expose the active site. These β interactions are mediated mainly by salt bridges and hydrophobic interactions between conserved alpha helices whose disruption by mutation damages the proteasome's ability to assemble.[65] The assembly of the half-proteasomes, in turn, is initiated by the assembly of the α subunits into their heptameric ring, forming a template for the association of the corresponding pro-β ring. The assembly of α subunits has not been characterized.[66]

Only recently, the assembly process of the 19S regulatory particle has been elucidated to considerable extent. The 19S regulatory particle assembles as two distinct subcomponents, the base and the lid. Assembly of the base complex is facilitated by four assembly chaperones, Hsm3/S5b, Nas2/p27, Rpn14/PAAF1, and Nas6/gankyrin (names for yeast/mammals).[67] These assembly chaperones bind to the AAA-ATPase subunits and their main function seems to be to ensure proper assembly of the heterohexameric AAA-ATPase ring. To date it is still under debate whether the base complex assembles separately, whether the assembly is templated by the 20S core particle, or whether alternative assembly pathways exist. In addition to the four assembly chaperones, the deubiquitinating enzyme Ubp6/Usp14 also promotes base assembly, but it is not essential.[68] The lid assembles separately in a specific order and does not require assembly chaperones.[69]


The protein degradation process

of ubiquitin, the highly conserved protein that serves as a molecular tag targeting proteins for degradation by the proteasome]]

Ubiquitination and targeting

Proteins are targeted for degradation by the proteasome with covalent modification of a lysine residue that requires the coordinated reactions of three enzymes. In the first step, a ubiquitin-activating enzyme (known as E1) hydrolyzes ATP and adenylylates a ubiquitin molecule. This is then transferred to E1's active-site cysteine residue in concert with the adenylylation of a second ubiquitin.[70] This adenylylated ubiquitin is then transferred to a cysteine of a second enzyme, ubiquitin-conjugating enzyme (E2). In the last step, a member of a highly diverse class of enzymes known as ubiquitin ligases (E3) recognizes the specific protein to be ubiquitinated and catalyzes the transfer of ubiquitin from E2 to this target protein. A target protein must be labeled with at least four ubiquitin monomers (in the form of a polyubiquitin chain) before it is recognized by the proteasome lid.[71] It is therefore the E3 that confers substrate specificity to this system.[72] The number of E1, E2, and E3 proteins expressed depends on the organism and cell type, but there are many different E3 enzymes present in humans, indicating that there is a huge number of targets for the ubiquitin proteasome system.

The mechanism by which a polyubiquitinated protein is targeted to the proteasome is not fully understood. A few high-resolution snapshots of the proteasome bound to a polyubiquitinated protein suggest that ubiquitin receptors might be coordinated with deubiquitinase Rpn11 for initial substrate targeting and engagement.[73] Ubiquitin-receptor proteins have an N-terminal ubiquitin-like (UBL) domain and one or more ubiquitin-associated (UBA) domains. The UBL domains are recognized by the 19S proteasome caps and the UBA domains bind ubiquitin via three-helix bundles. These receptor proteins may escort polyubiquitinated proteins to the proteasome, though the specifics of this interaction and its regulation are unclear.[74]

The ubiquitin protein itself is 76 amino acids long and was named due to its ubiquitous nature, as it has a highly conserved sequence and is found in all known eukaryotic organisms.[75] The genes encoding ubiquitin in eukaryotes are arranged in tandem repeats, possibly due to the heavy transcription demands on these genes to produce enough ubiquitin for the cell. It has been proposed that ubiquitin is the slowest-evolving protein identified to date.[76] Ubiquitin contains seven lysine residues to which another ubiquitin can be ligated, resulting in different types of polyubiquitin chains.[77] Chains in which each additional ubiquitin is linked to lysine 48 of the previous ubiquitin have a role in proteasome targeting, while other types of chains may be involved in other processes.[78][79]

ation pathway]]

Unfolding and translocation

After a protein has been ubiquitinated, it is recognized by the 19S regulatory particle in an ATP-dependent binding step.[80][81] The substrate protein must then enter the interior of the 20S particle to come in contact with the proteolytic active sites. Because the 20S particle's central channel is narrow and gated by the N-terminal tails of the α ring subunits, the substrates must be at least partially unfolded before they enter the core.[82] The passage of the unfolded substrate into the core is called _translocation_ and necessarily occurs after deubiquitination.[83][84] However, the order in which substrates are deubiquitinated and unfolded is not yet clear.[85] Which of these processes is the rate-limiting step in the overall proteolysis reaction depends on the specific substrate; for some proteins, the unfolding process is rate-limiting, while deubiquitination is the slowest step for other proteins.[86] The extent to which substrates must be unfolded before translocation is suggested to be around 20 amino acid residues by the atomic structure of the substrate-engaged 26S proteasome in the deubiquitylation-compatible state,[87] but substantial tertiary structure, and in particular nonlocal interactions such as disulfide bonds, are sufficient to inhibit degradation.[88] The presence of intrinsically disordered protein segments of sufficient size, either at the protein terminus or internally, has also been proposed to facilitate efficient initiation of degradation.[89][90]

The gate formed by the α subunits prevents peptides longer than about four residues from entering the interior of the 20S particle. The ATP molecules bound before the initial recognition step are hydrolyzed before translocation. While energy is needed for substrate unfolding, it is not required for translocation.[91][92] The assembled 26S proteasome can degrade unfolded proteins in the presence of a non-hydrolyzable ATP analog, but cannot degrade folded proteins, indicating that energy from ATP hydrolysis is used for substrate unfolding.[93] Passage of the unfolded substrate through the opened gate occurs via facilitated diffusion if the 19S cap is in the ATP-bound state.[94]

The mechanism for unfolding of globular proteins is necessarily general, but somewhat dependent on the amino acid sequence. Long sequences of alternating glycine and alanine have been shown to inhibit substrate unfolding, decreasing the efficiency of proteasomal degradation; this results in the release of partially degraded byproducts, possibly due to the decoupling of the ATP hydrolysis and unfolding steps.[95] Such glycine-alanine repeats are also found in nature, for example in silk fibroin; in particular, certain Epstein–Barr virus gene products bearing this sequence can stall the proteasome, helping the virus propagate by preventing antigen presentation on the major histocompatibility complex.[96]

s. The α subunits are represented as green spheres and the β subunits as protein backbones colored by individual polypeptide chain. The small pink spheres represent the location of the active-site threonine residue in each subunit. Light blue chemical structures are the inhibitor bortezomib bound to the active sites.]]

Proteolysis

The mechanism of proteolysis by the β subunits of the 20S core particle is through a threonine-dependent nucleophilic attack. This mechanism may depend on an associated water molecule for deprotonation of the reactive threonine hydroxyl. Degradation occurs within the central chamber formed by the association of the two β rings and normally does not release partially degraded products, instead reducing the substrate to short polypeptides typically 7–9 residues long, though they can range from 4 to 25 residues, depending on the organism and substrate. The biochemical mechanism that determines product length is not fully characterized.[97] Although the three catalytic β subunits have a common mechanism, they have slightly different substrate specificities, which are considered chymotrypsin-like, trypsin-like, and peptidyl-glutamyl peptide-hydrolyzing (PHGH)-like. These variations in specificity are the result of interatomic contacts with local residues near the active sites of each subunit. Each catalytic β subunit also possesses a conserved lysine residue required for proteolysis.[98]

Although the proteasome normally produces very short peptide fragments, in some cases these products are themselves biologically active and functional molecules. Certain transcription factors regulating the expression of specific genes, including one component of the mammalian complex NF-κB, are synthesized as inactive precursors whose ubiquitination and subsequent proteasomal degradation converts them to an active form. Such activity requires the proteasome to cleave the substrate protein internally, rather than processively degrading it from one terminus. It has been suggested that long loops on these proteins' surfaces serve as the proteasomal substrates and enter the central cavity, while the majority of the protein remains outside.[99] Similar effects have been observed in yeast proteins; this mechanism of selective degradation is known as _regulated ubiquitin/proteasome dependent processing_ (RUP).[100]

Ubiquitin-independent degradation

Although most proteasomal substrates must be ubiquitinated before being degraded, there are some exceptions to this general rule, especially when the proteasome plays a normal role in the post-translational processing of the protein. The proteasomal activation of NF-κB by processing p105 into p50 via internal proteolysis is one major example.[101] Some proteins that are hypothesized to be unstable due to intrinsically unstructured regions,[102] are degraded in a ubiquitin-independent manner. The most well-known example of a ubiquitin-independent proteasome substrate is the enzyme ornithine decarboxylase.[103] Ubiquitin-independent mechanisms targeting key cell cycle regulators such as p53 have also been reported, although p53 is also subject to ubiquitin-dependent degradation.[104] Finally, structurally abnormal, misfolded, or highly oxidized proteins are also subject to ubiquitin-independent and 19S-independent degradation under conditions of cellular stress.[105]


Evolution

(blue) and hslU (red) from _E. coli_. This complex of heat shock proteins is thought to resemble the ancestor of the modern proteasome.]] The 20S proteasome is both ubiquitous and essential in eukaryotes. Some prokaryotes, including many archaea and the bacterial order Actinomycetales, also share homologs of the 20S proteasome, whereas most bacteria possess heat shock genes hslV and hslU, whose gene products are a multimeric protease arranged in a two-layered ring and an ATPase.[106] The hslV protein has been hypothesized to resemble the likely ancestor of the 20S proteasome.[107] In general, HslV is not essential in bacteria, and not all bacteria possess it, whereas some protists possess both the 20S and the hslV systems.[108] Many bacteria also possess other homologs of the proteasome and an associated ATPase, most notably ClpP and ClpX. This redundancy explains why the HslUV system is not essential.

Sequence analysis suggests that the catalytic β subunits diverged earlier in evolution than the predominantly structural α subunits. In bacteria that express a 20S proteasome, the β subunits have high sequence identity to archaeal and eukaryotic β subunits, whereas the α sequence identity is much lower. The presence of 20S proteasomes in bacteria may result from lateral gene transfer, while the diversification of subunits among eukaryotes is ascribed to multiple gene duplication events.[109]


Cell cycle control

Cell cycle progression is controlled by ordered action of cyclin-dependent kinases (CDKs), activated by specific cyclins that demarcate phases of the cell cycle. Mitotic cyclins, which persist in the cell for only a few minutes, have one of the shortest life spans of all intracellular proteins.[110] After a CDK-cyclin complex has performed its function, the associated cyclin is polyubiquitinated and destroyed by the proteasome, which provides directionality for the cell cycle. In particular, exit from mitosis requires the proteasome-dependent dissociation of the regulatory component cyclin B from the mitosis promoting factor complex.[111] In vertebrate cells, "slippage" through the mitotic checkpoint leading to premature M phase exit can occur despite the delay of this exit by the spindle checkpoint.[112]

Earlier cell cycle checkpoints such as post-restriction point check between G₁ phase and S phase similarly involve proteasomal degradation of cyclin A, whose ubiquitination is promoted by the anaphase promoting complex (APC), an E3 ubiquitin ligase.[113] The APC and the Skp1/Cul1/F-box protein complex (SCF complex) are the two key regulators of cyclin degradation and checkpoint control; the SCF itself is regulated by the APC via ubiquitination of the adaptor protein, Skp2, which prevents SCF activity before the G1-S transition.[114]

Individual components of the 19S particle have their own regulatory roles. Gankyrin, a recently identified oncoprotein, is one of the 19S subcomponents that also tightly binds the cyclin-dependent kinase CDK4 and plays a key role in recognizing ubiquitinated p53, via its affinity for the ubiquitin ligase MDM2. Gankyrin is anti-apoptotic and has been shown to be overexpressed in some tumor cell types such as hepatocellular carcinoma.[115]

Regulation of plant growth

In plants, signaling by auxins, or phytohormones that order the direction and tropism of plant growth, induces the targeting of a class of transcription factor repressors known as Aux/IAA proteins for proteasomal degradation. These proteins are ubiquitinated by SCFTIR1, or SCF in complex with the auxin receptor TIR1. Degradation of Aux/IAA proteins derepresses transcription factors in the auxin-response factor (ARF) family and induces ARF-directed gene expression.[116] The cellular consequences of ARF activation depend on the plant type and developmental stage, but are involved in directing growth in roots and leaf veins. The specific response to ARF derepression is thought to be mediated by specificity in the pairing of individual ARF and Aux/IAA proteins.[117]

Apoptosis

Both internal and external signals can lead to the induction of apoptosis, or programmed cell death. The resulting deconstruction of cellular components is primarily carried out by specialized proteases known as caspases, but the proteasome also plays important and diverse roles in the apoptotic process. The involvement of the proteasome in this process is indicated by both the increase in protein ubiquitination, and of E1, E2, and E3 enzymes that is observed well in advance of apoptosis.[118][119][120] During apoptosis, proteasomes localized to the nucleus have also been observed to translocate to outer membrane blebs characteristic of apoptosis.[121]

Proteasome inhibition has different effects on apoptosis induction in different cell types. In general, the proteasome is not required for apoptosis, although inhibiting it is pro-apoptotic in most cell types that have been studied. Apoptosis is mediated through disrupting the regulated degradation of pro-growth cell cycle proteins.[122] However, some cell lines — in particular, primary cultures of quiescent and differentiated cells such as thymocytes and neurons — are prevented from undergoing apoptosis on exposure to proteasome inhibitors. The mechanism for this effect is not clear, but is hypothesized to be specific to cells in quiescent states, or to result from the differential activity of the pro-apoptotic kinase JNK.[123] The ability of proteasome inhibitors to induce apoptosis in rapidly dividing cells has been exploited in several recently developed chemotherapy agents such as bortezomib and .


Response to cellular stress

In response to cellular stresses – such as infection, heat shock, or oxidative damage – heat shock proteins that identify misfolded or unfolded proteins and target them for proteasomal degradation are expressed. Both Hsp27 and Hsp90—chaperone proteins have been implicated in increasing the activity of the ubiquitin-proteasome system, though they are not direct participants in the process.[124] Hsp70, on the other hand, binds exposed hydrophobic patches on the surface of misfolded proteins and recruits E3 ubiquitin ligases such as CHIP to tag the proteins for proteasomal degradation.[125] The CHIP protein (carboxyl terminus of Hsp70-interacting protein) is itself regulated via inhibition of interactions between the E3 enzyme CHIP and its E2 binding partner.[126]

Similar mechanisms exist to promote the degradation of oxidatively damaged proteins via the proteasome system. In particular, proteasomes localized to the nucleus are regulated by PARP and actively degrade inappropriately oxidized histones.[127] Oxidized proteins, which often form large amorphous aggregates in the cell, can be degraded directly by the 20S core particle without the 19S regulatory cap and do not require ATP hydrolysis or tagging with ubiquitin.[128] However, high levels of oxidative damage increases the degree of cross-linking between protein fragments, rendering the aggregates resistant to proteolysis. Larger numbers and sizes of such highly oxidized aggregates are associated with aging.[129]

Dysregulation of the ubiquitin proteasome system may contribute to several neural diseases. It may lead to brain tumors such as astrocytomas.[130] In some of the late-onset neurodegenerative diseases that share aggregation of misfolded proteins as a common feature, such as Parkinson's disease and Alzheimer's disease, large insoluble aggregates of misfolded proteins can form and then result in neurotoxicity, through mechanisms that are not yet well understood. Decreased proteasome activity has been suggested as a cause of aggregation and Lewy body formation in Parkinson's.[131] This hypothesis is supported by the observation that yeast models of Parkinson's are more susceptible to toxicity from α-synuclein, the major protein component of Lewy bodies, under conditions of low proteasome activity.[132] Impaired proteasomal activity may underlie cognitive disorders such as the autism spectrum disorders, and muscle and nerve diseases such as inclusion body myopathy.[133]


Role in the immune system

The proteasome plays a straightforward but critical role in the function of the adaptive immune system. Peptide antigens are displayed by the major histocompatibility complex class I (MHC) proteins on the surface of antigen-presenting cells. These peptides are products of proteasomal degradation of proteins originated by the invading pathogen. Although constitutively expressed proteasomes can participate in this process, a specialized complex composed of proteins, whose expression is induced by interferon gamma, are the primary producers of peptides which are optimal in size and composition for MHC binding. These proteins whose expression increases during the immune response include the 11S regulatory particle, whose main known biological role is regulating the production of MHC ligands, and specialized β subunits called β1i, β2i, and β5i with altered substrate specificity. The complex formed with the specialized β subunits is known as the _immunoproteasome_.[134] Another β5i variant subunit, β5t, is expressed in the thymus, leading to a thymus-specific "thymoproteasome" whose function is as yet unclear.[135]

The strength of MHC class I ligand binding is dependent on the composition of the ligand C-terminus, as peptides bind by hydrogen bonding and by close contacts with a region called the "B pocket" on the MHC surface. Many MHC class I alleles prefer hydrophobic C-terminal residues, and the immunoproteasome complex is more likely to generate hydrophobic C-termini.[136]

Due to its role in generating the activated form of NF-κB, an anti-apoptotic and pro-inflammatory regulator of cytokine expression, proteasomal activity has been linked to inflammatory and autoimmune diseases. Increased levels of proteasome activity correlate with disease activity and have been implicated in autoimmune diseases including systemic lupus erythematosus and rheumatoid arthritis.[137]

The proteasome is also involved in Intracellular antibody-mediated proteolysis of antibody-bound virions. In this neutralisation pathway, TRIM21 (a protein of the tripartite motif family) binds with immunoglobulin G to direct the virion to the proteasome where it is degraded.[138]


Proteasome inhibitors

(Boronated form of MG132), a proteasome inhibitor used in chemotherapy that is particularly effective against multiple myeloma]] bound to the core particle in a yeast proteasome. The bortezomib molecule is in the center colored by atom type (carbon = pink, nitrogen = blue, oxygen = red, boron = yellow), surrounded by the local protein surface. The blue patch is the catalytic threonine residue whose activity is blocked by the presence of bortezomib.]] Proteasome inhibitors have effective anti-tumor activity in cell culture, inducing apoptosis by disrupting the regulated degradation of pro-growth cell cycle proteins.[139] This approach of selectively inducing apoptosis in tumor cells has proven effective in animal models and human trials.

Lactacystin, a natural product synthesized by _Streptomyces_ bacteria, was the first non-peptidic proteasome inhibitor discovered[140] and is widely used as a research tool in biochemistry and cell biology. Lactacystin was licensed to Myogenics/Proscript, which was acquired by Millennium Pharmaceuticals, now part of Takeda Pharmaceuticals. Lactacystin covalently modifies the amino-terminal threonine of catalytic β subunits of the proteasome, particularly the β5 subunit responsible for the proteasome's chymotrypsin-like activity. This discovery helped to establish the proteasome as a mechanistically novel class of protease: an amino-terminal threonine protease.

Bortezomib (Boronated MG132), a molecule developed by Millennium Pharmaceuticals and marketed as Velcade, is the first proteasome inhibitor to reach clinical use as a chemotherapy agent.[141] Bortezomib is used in the treatment of multiple myeloma.[142] Notably, multiple myeloma has been observed to result in increased proteasome-derived peptide levels in blood serum that decrease to normal levels in response to successful chemotherapy.[143] Studies in animals have indicated that bortezomib may also have clinically significant effects in pancreatic cancer.[144][145] Preclinical and early clinical studies have been started to examine bortezomib's effectiveness in treating other B-cell-related cancers,[146] particularly some types of non-Hodgkin's lymphoma.[147] Clinical results also seem to justify use of proteasome inhibitor combined with chemotherapy, for B-cell acute lymphoblastic leukemia [148] Proteasome inhibitors can kill some types of cultured leukemia cells that are resistant to glucocorticoids.[149]

The molecule ritonavir, marketed as Norvir, was developed as a protease inhibitor and used to target HIV infection. However, it has been shown to inhibit proteasomes as well as free proteases; to be specific, the chymotrypsin-like activity of the proteasome is inhibited by ritonavir, while the trypsin-like activity is somewhat enhanced.[150] Studies in animal models suggest that ritonavir may have inhibitory effects on the growth of glioma cells.[151]

Proteasome inhibitors have also shown promise in treating autoimmune diseases in animal models. For example, studies in mice bearing human skin grafts found a reduction in the size of lesions from psoriasis after treatment with a proteasome inhibitor.[152] Inhibitors also show positive effects in rodent models of asthma.[153]

Labeling and inhibition of the proteasome is also of interest in laboratory settings for both _in vitro_ and _in vivo_ study of proteasomal activity in cells. The most commonly used laboratory inhibitors are lactacystin and the peptide aldehyde MG132 initially developed by Goldberg lab. Fluorescent inhibitors have also been developed to specifically label the active sites of the assembled proteasome.[154]


Clinical significance

The proteasome and its subunits are of clinical significance for at least two reasons: (1) a compromised complex assembly or a dysfunctional proteasome can be associated with the underlying pathophysiology of specific diseases, and (2) they can be exploited as drug targets for therapeutic interventions. More recently, more effort has been made to consider the proteasome for the development of novel diagnostic markers and strategies. An improved and comprehensive understanding of the pathophysiology of the proteasome should lead to clinical applications in the future.

The proteasomes form a pivotal component for the Ubiquitin-Proteasome System (UPS) [155] and corresponding cellular Protein Quality Control (PQC). Protein ubiquitination and subsequent proteolysis and degradation by the proteasome are important mechanisms in the regulation of the cell cycle, cell growth and differentiation, gene transcription, signal transduction and apoptosis.[156] Subsequently, a compromised proteasome complex assembly and function lead to reduced proteolytic activities and the accumulation of damaged or misfolded protein species. Such protein accumulation may contribute to the pathogenesis and phenotypic characteristics in neurodegenerative diseases,[157][158] cardiovascular diseases,[159][160][161] inflammatory responses and autoimmune diseases,[162] and systemic DNA damage responses leading to malignancies.[163]

Several experimental and clinical studies have indicated that aberrations and deregulations of the UPS contribute to the pathogenesis of several neurodegenerative and myodegenerative disorders, including Alzheimer's disease,[164] Parkinson's disease[165] and Pick's disease,[166] amyotrophic lateral sclerosis (ALS),[167] Huntington's disease,[168] Creutzfeldt–Jakob disease,[169] and motor neuron diseases, polyglutamine (PolyQ) diseases, muscular dystrophies[170] and several rare forms of neurodegenerative diseases associated with dementia.[171] As part of the Ubiquitin-Proteasome System (UPS), the proteasome maintains cardiac protein homeostasis and thus plays a significant role in cardiac ischemic injury,[172] ventricular hypertrophy[173] and heart failure.[174] Additionally, evidence is accumulating that the UPS plays an essential role in malignant transformation. UPS proteolysis plays a major role in responses of cancer cells to stimulatory signals that are critical for the development of cancer. Accordingly, gene expression by degradation of transcription factors, such as p53, c-Jun, c-Fos, NF-κB, c-Myc, HIF-1α, MATα2, STAT3, sterol-regulated element-binding proteins and androgen receptors are all controlled by the UPS and thus involved in the development of various malignancies.[175] Moreover, the UPS regulates the degradation of tumor suppressor gene products such as adenomatous polyposis coli (APC) in colorectal cancer, retinoblastoma (Rb). and von Hippel-Lindau tumor suppressor (VHL), as well as a number of proto-oncogenes (Raf, Myc, Myb, Rel, Src, Mos, Abl). The UPS is also involved in the regulation of inflammatory responses. This activity is usually attributed to the role of proteasomes in the activation of NF-κB which further regulates the expression of pro inflammatory cytokines such as TNF-α, IL-β, IL-8, adhesion molecules (ICAM-1, VCAM-1, P-selectin) and prostaglandins and nitric oxide (NO).[176] Additionally, the UPS also plays a role in inflammatory responses as regulators of leukocyte proliferation, mainly through proteolysis of cyclines and the degradation of CDK inhibitors.[177] Lastly, autoimmune disease patients with SLE, Sjogren's syndrome and rheumatoid arthritis (RA) predominantly exhibit circulating proteasomes which can be applied as clinical biomarkers.[178]


See also

-   The Proteolysis Map
-   Exosome
-   Endoplasmic reticulum-associated protein degradation
-   JUNQ and IPOD


References


Further reading

-   -   The Yeast 26S Proteasome with list of subunits and pictures
-   -   -   -


External links

-   Proteasome subunit nomenclature guide
-   [http://www.pdbe.org/emsearch/proteasome* 3D proteasome structures in the EM Data Bank(EMDB)]

Category:Proteins Category:Protein complexes Category:Organelles Category:Apoptosis

[1]

[2]

[3]

[4]

[5]

[6]

[7]

[8]

[9]

[10]

[11]

[12]

[13]

[14]

[15]

[16]

[17]

[18]

[19]

[20]

[21]

[22]

[23]

[24]

[25]

[26]

[27]

[28]

[29]

[30]

[31]

[32]

[33]

[34]

[35]

[36]

[37]

[38]

[39]

[40]

[41]

[42]

[43]

[44]

[45]

[46]

[47]

[48]

[49]

[50]

[51]

[52]

[53]

[54]

[55]

[56]

[57]

[58]

[59]

[60]

[61]

[62]

[63]

[64]

[65]

[66]

[67]

[68]

[69]

[70]

[71]

[72]

[73]

[74]

[75]

[76]

[77]

[78]

[79]

[80]

[81]

[82]

[83]

[84]

[85]

[86]

[87]

[88]

[89]

[90]

[91]

[92]

[93]

[94]

[95]

[96]

[97]

[98]

[99]

[100]

[101]

[102]

[103]

[104]

[105]

[106]

[107]

[108]

[109]

[110]

[111]

[112]

[113]

[114]

[115]

[116]

[117]

[118]

[119]

[120]

[121]

[122]

[123]

[124]

[125]

[126]

[127]

[128]

[129]

[130]

[131]

[132]

[133]

[134]

[135]

[136]

[137]

[138]

[139]

[140]

[141] United States Food and Drug Administration press release 13 May 2003. Access date 29 December 2006. See also FDA Velcade information page.

[142]

[143]

[144]

[145]

[146]

[147]

[148]

[149]

[150]

[151]

[152]

[153]

[154]

[155]

[156]

[157]

[158]

[159]

[160]

[161]

[162]

[163]

[164]

[165]

[166]

[167]

[168]

[169]

[170]

[171]

[172]

[173]

[174]

[175]

[176]

[177]

[178]