RNAS ASSOCIATED WITH GENES ASSOCIATED WITH TWISTER AND HAMMERHEAD RIBOZYMES (RAGATH) refers to a bioinformatics strategy that was devised to find self-cleaving ribozymes in bacteria.[1] It also refers to candidate RNAs, or RAGATH RNA MOTIFS, discovered using this strategy.

With the discovery of the twister ribozyme,[2] it was recognized that many genetic elements in bacteria are often located nearby to twister ribozymes and also to the previously discovered hammerhead ribozymes.[3] These genetic elements include several gene classes, many of which are characteristic of Mu-like phages. The nearby elements also include twister and hammerhead ribozymes. In other words, twister and hammerhead ribozymes are often located in bacteria nearby to other twister or hammerhead ribozymes.

Given these observations, researchers hypothesized that other classes of self-cleaving ribozymes would also associate with these genetic elements. Therefore, searches were conducted on the non-coding regions nearby to the associated genetic elements to find conserved RNA structures using a previously established method.[4] Such RNA structures would then be candidates as self-cleaving ribozymes.

Using this method, previously unknown self-cleaving ribozyme classes were found: the twister sister, pistol and hatchet ribozymes. Unusual examples of hammerhead and HDV ribozymes were also found. Twelve additional conserved RNA structures did not appear to function as ribozymes, and the biological and biochemical functions of these RNAs remain unknown. All conserved RNAs were named "RAGATH RNA motifs", and the unsolved RNAs are numbered RAGATH-4 through RAGATH-15. Additional RAGATH motifs that did not self-cleave 'in vitro' were also later published.[5]


References

Category:RNA Ribozymes

[1]

[2]

[3]

[4]

[5]