Generally, in progression to cancer, hundreds of genes are silenced or activated. Although silencing of some genes in cancers occurs by mutation, a large proportion of carcinogenic gene silencing is a result of altered DNA methylation (see DNA methylation in cancer). DNA methylation causing silencing in cancer typically occurs at multiple CpG sites in the CpG islands that are present in the promoters of protein coding genes.

Altered expressions of microRNAs also silence or activate many genes in progression to cancer (see microRNAs in cancer). Altered microRNA expression occurs through hyper/hypo-methylation of CpG sites in CpG islands in promoters controlling transcription of the microRNAs.

Silencing of DNA repair genes through methylation of CpG islands in their promoters appears to be especially important in progression to cancer (see methylation of DNA repair genes in cancer).


CpG islands in promoters

In humans, about 70% of promoters located near the transcription start site of a gene (proximal promoters) contain a CpG island.[1][2] CpG islands are generally 200 to 2000 base pairs long, have a C:G base pair content >50%, and have regions of DNA where a cytosine nucleotide is followed by a guanine nucleotide and this occurs frequently in the linear sequence of bases along its 5' â†’ 3' direction.[3][4]

Genes may also have distant promoters (distal promoters) and these frequently contain CpG islands as well. An example is the promoter of the DNA repair gene _ERCC1_, where the CpG island-containing promoter is located about 5,400 nucleotides upstream of the coding region of the _ERCC1_ gene.[5] CpG islands also occur frequently in promoters for functional noncoding RNAs such as microRNAs.[6]


Transcription silencing due to methylation of CpG islands

In humans, DNA methylation occurs at the 5' position of the pyrimidine ring of the cytosine residues within CpG sites to form 5-methylcytosines. The presence of multiple methylated CpG sites in CpG islands of promoters causes stable inhibition (silencing) of genes.[7] Silencing of transcription of a gene may be initiated by other mechanisms, but this is often followed by methylation of CpG sites in the promoter CpG island to cause the stable silencing of the gene.[8]


Transcription silencing/activation in cancers

In cancers, loss of expression of genes occurs about 10 times more frequently by transcription silencing (caused by promoter hypermethylation of CpG islands) than by mutations. As Vogelstein et al. point out, in a colorectal cancer there are usually about 3 to 6 driver mutations and 33 to 66 hitchhiker or passenger mutations.[9] In contrast, in colon tumors compared to adjacent normal-appearing colonic mucosa, there are about 600 to 800 heavily methylated CpG islands in promoters of genes in the tumors while these CpG islands are not methylated in the adjacent mucosa.[10][11][12]

Using gene set enrichment analysis, 569 out of 938 gene sets were hypermethylated and 369 were hypomethylated in cancers. Hypomethylation of CpG islands in promoters results in increased transcription of the genes or gene sets affected.[13]

One study[14] listed 147 specific genes with colon cancer-associated hypermethylated promoters and 27 with hypomethylated promoters, along with the frequency with which these hyper/hypo-methylations were found in colon cancers. At least 10 of those genes had hypermethylated promoters in nearly 100% of colon cancers. They also indicated 11 microRNAs whose promoters were hypermethylated in colon cancers at frequencies between 50% and 100% of cancers. MicroRNAs (miRNAs) are small endogenous RNAs that pair with sequences in messenger RNAs to direct post-transcriptional repression. On average, each microRNA represses or inhibits transcriptional expression of several hundred target genes. Thus microRNAs with hypermethylated promoters may be allowing enhanced transcription of hundreds to thousands of genes in a cancer.[15]


Transcription inhibition and activation by nuclear microRNAs

For more than 20 years, microRNAs have been known to act in the cytoplasm to degrade transcriptional expression of specific target gene messenger RNAs (see microRNA history). However, recently, Gagnon et al.[16] showed that as many as 75% of microRNAs may be shuttled back into the nucleus of cells. Some nuclear microRNAs have been shown to mediate transcriptional gene activation or transcriptional gene inhibition.[17]


DNA repair genes with hyper/hypo-methylated promoters in cancers

DNA repair genes are frequently repressed in cancers due to hypermethylation of CpG islands within their promoters. In head and neck squamous cell carcinomas at least 15 DNA repair genes have frequently hypermethylated promoters; these genes are _XRCC1, MLH3, PMS1, RAD51B, XRCC3, RAD54B, BRCA1, SHFM1, GEN1, FANCE, FAAP20, SPRTN, SETMAR, HUS1,_ and _PER1_.[18] About seventeen types of cancer are frequently deficient in one or more DNA repair genes due to hypermethylation of their promoters.[19][20] As summarized in one review article, promoter hypermethylation of the DNA repair gene _MGMT_ occurs in 93% of bladder cancers, 88% of stomach cancers, 74% of thyroid cancers, 40%-90% of colorectal cancers and 50% of brain cancers.[21] Promoter hypermethylation of _LIG4_ occurs in 82% of colorectal cancers. This review article also indicates promoter hypermethylation of _NEIL1_ occurs in 62% of head and neck cancers and in 42% of non-small-cell lung cancers; promoter hypermetylation of _ATM_ occurs in 47% of non-small-cell lung cancers; promoter hypermethylation of _MLH1_ occurs in 48% of squamous cell carcinomas; and promoter hypermethylation of _FANCB_ occurs in 46% of head and neck cancers.[22]

On the other hand, the promoters of two genes, _PARP1_ and _FEN1_, were hypomethylated and these genes were over-expressed in numerous cancers. _PARP1_ and _FEN1_ are essential genes in the error-prone and mutagenic DNA repair pathway microhomology-mediated end joining. If this pathway is over-expressed, the excess mutations it causes can lead to cancer. PARP1 is over-expressed in tyrosine kinase-activated leukemias,[23] in neuroblastoma,[24] in testicular and other germ cell tumors,[25] and in Ewing's sarcoma,[26] FEN1 is over-expressed in the majority of cancers of the breast,[27] prostate,[28] stomach,[29][30] neuroblastomas,[31] pancreatic,[32] and lung.[33]

DNA damage appears to be the primary underlying cause of cancer.[34][35] If accurate DNA repair is deficient, DNA damages tend to accumulate. Such excess DNA damage can increase mutational errors during DNA replication due to error-prone translesion synthesis. Excess DNA damage can also increase epigenetic alterations due to errors during DNA repair. Such mutations and epigenetic alterations can give rise to cancer (see malignant neoplasms). Thus, CpG island hyper/hypo-methylation in the promoters of DNA repair genes are likely central to progression to cancer.[36][37]


See also

-   Eukaryotic transcription
-   Gene expression
-   Transcriptional regulation


References

Category:Gene expression Category:Non-coding RNA Category:Epigenetics Category:Cancer epigenetics Category:DNA Category:Medical regulation

[1]

[2]

[3]

[4]

[5]

[6]

[7]

[8]

[9]

[10]

[11]

[12]

[13]

[14]

[15]

[16]

[17]

[18]

[19] Carol Bernstein and Harris Bernstein (2015). Epigenetic Reduction of DNA Repair in Progression to Cancer, Advances in DNA Repair, Prof. Clark Chen (Ed.), , InTech, Available from: http://www.intechopen.com/books/advances-in-dna-repair/epigenetic-reduction-of-dna-repair-in-progression-to-cancer

[20]

[21]

[22]

[23]

[24]

[25]

[26]

[27]

[28]

[29]

[30]

[31]

[32]

[33]

[34]

[35]

[36]

[37]