A REVERSE TRANSCRIPTASE (RT) is an enzyme used to generate complementary DNA (cDNA) from an RNA template, a process termed _reverse transcription_. Reverse transcriptases are used by retroviruses to replicate their genomes, by retrotransposon mobile genetic elements to proliferate within the host genome, by eukaryotic cells to extend the telomeres at the ends of their linear chromosomes, and by some non-retroviruses such as the hepatitis B virus, a member of the _Hepadnaviridae_, which are dsDNA-RT viruses.

Retroviral RT has three sequential biochemical activities: RNA-dependent DNA polymerase activity, ribonuclease H, and DNA-dependent DNA polymerase activity. Collectively, these activities enable the enzyme to convert single-stranded RNA into double-stranded cDNA. In retroviruses and retrotransposons, this cDNA can then integrate into the host genome, from which new RNA copies can be made via host-cell transcription. The same sequence of reactions is widely used in the laboratory to convert RNA to DNA for use in molecular cloning, RNA sequencing, polymerase chain reaction (PCR), or genome analysis.


History

Reverse transcriptases were discovered by Howard Temin at the University of Wisconsin–Madison in Rous sarcoma virions[1] and independently isolated by David Baltimore in 1970 at MIT from two RNA tumour viruses: murine leukemia virus and again Rous sarcoma virus.[2] For their achievements, they shared the 1975 Nobel Prize in Physiology or Medicine (with Renato Dulbecco).

Well-studied reverse transcriptases include:

-   HIV-1 reverse transcriptase from human immunodeficiency virus type 1 () has two subunits, which have respective molecular weights of 66 and 51 kDa.[3]
-   M-MLV reverse transcriptase from the Moloney murine leukemia virus is a single 75 kDa monomer.[4]
-   AMV reverse transcriptase from the avian myeloblastosis virus also has two subunits, a 63 kDa subunit and a 95 kDa subunit.[5]
-   Telomerase reverse transcriptase that maintains the telomeres of eukaryotic chromosomes.


Function in viruses

s of the RNase H active site and the polymerase active site are shown in ball-and-stick form.]] The enzymes are encoded and used by viruses that use reverse transcription as a step in the process of replication. Reverse-transcribing RNA viruses, such as retroviruses, use the enzyme to reverse-transcribe their RNA genomes into DNA, which is then integrated into the host genome and replicated along with it. Reverse-transcribing DNA viruses, such as the hepadnaviruses, can allow RNA to serve as a template in assembling and making DNA strands. HIV infects humans with the use of this enzyme. Without reverse transcriptase, the viral genome would not be able to incorporate into the host cell, resulting in failure to replicate.

Process of reverse transcription

Reverse transcriptase creates double-stranded DNA from an RNA template.

In virus species with reverse transcriptase lacking DNA-dependent DNA polymerase activity, creation of double-stranded DNA can possibly be done by host-encoded DNA polymerase δ, mistaking the viral DNA-RNA for a primer and synthesizing a double-stranded DNA by similar mechanism as in primer removal, where the newly synthesized DNA displaces the original RNA template.

The process of reverse transcription is extremely error-prone, and it is during this step that mutations may occur. Such mutations may cause drug resistance.

Retroviral reverse transcription

Retroviruses, also referred to as class VI ssRNA-RT viruses, are RNA reverse-transcribing viruses with a DNA intermediate. Their genomes consist of two molecules of positive-sense single-stranded RNA with a 5' cap and 3' polyadenylated tail. Examples of retroviruses include the human immunodeficiency virus (HIV) and the human T-lymphotropic virus (HTLV). Creation of double-stranded DNA occurs in the cytosol[6] as a series of these steps:

1.  Lysyl tRNA acts as a primer and hybridizes to a complementary part of the virus RNA genome called the primer binding site or PBS.
2.  Reverse transcriptase then adds DNA nucleotides onto the 3' end of the primer, synthesizing DNA complementary to the U5 (non-coding region) and R region (a direct repeat found at both ends of the RNA molecule) of the viral RNA.
3.  A domain on the reverse transcriptase enzyme called RNAse H degrades the U5 and R regions on the 5’ end of the RNA.
4.  The tRNA primer then "jumps" to the 3’ end of the viral genome, and the newly synthesised DNA strands hybridizes to the complementary R region on the RNA.
5.  The complementary DNA (cDNA) added in (2) is further extended.
6.  The majority of viral RNA is degraded by RNAse H, leaving only the PP sequence.
7.  Synthesis of the second DNA strand begins, using the remaining PP fragment of viral RNA as a primer.
8.  The tRNA primer leaves and a "jump" happens. The PBS from the second strand hybridizes with the complementary PBS on the first strand.
9.  Both strands are extended to form a complete double-stranded DNA copy of the original viral RNA genome, which can then be incorporated into the host's genome by the enzyme integrase.

Creation of double-stranded DNA also involves _strand transfer_, in which there is a translocation of short DNA product from initial RNA-dependent DNA synthesis to acceptor template regions at the other end of the genome, which are later reached and processed by the reverse transcriptase for its DNA-dependent DNA activity.[7]

Retroviral RNA is arranged in 5’ terminus to 3’ terminus. The site where the primer is annealed to viral RNA is called the primer-binding site (PBS). The RNA 5’end to the PBS site is called U5, and the RNA 3’ end to the PBS is called the leader. The tRNA primer is unwound between 14 and 22 nucleotides and forms a base-paired duplex with the viral RNA at PBS. The fact that the PBS is located near the 5’ terminus of viral RNA is unusual because reverse transcriptase synthesize DNA from 3’ end of the primer in the 5’ to 3’ direction (with respect to the newly synthesized DNA strand). Therefore, the primer and reverse transcriptase must be relocated to 3’ end of viral RNA. In order to accomplish this reposition, multiple steps and various enzymes including DNA polymerase, ribonuclease H(RNase H) and polynucleotide unwinding are needed.[8][9]

The HIV reverse transcriptase also has ribonuclease activity that degrades the viral RNA during the synthesis of cDNA, as well as DNA-dependent DNA polymerase activity that copies the sense cDNA strand into an _antisense_ DNA to form a double-stranded viral DNA intermediate (vDNA).[10]


In cellular life

Self-replicating stretches of eukaryotic genomes known as retrotransposons utilize reverse transcriptase to move from one position in the genome to another via an RNA intermediate. They are found abundantly in the genomes of plants and animals. Telomerase is another reverse transcriptase found in many eukaryotes, including humans, which carries its own RNA template; this RNA is used as a template for DNA replication.[11]

Initial reports of reverse transcriptase in prokaryotes came as far back as 1971 (Beljanski et al., 1971a, 1972). These have since been broadly described as part of bacterial Retrons, distinct sequences that code for reverse transcriptase, and are used in the synthesis of msDNA. In order to initiate synthesis of DNA, a primer is needed. In bacteria, the primer is synthesized during replication.[12]

Valerian Dolja of Oregon State argues that viruses, due to their diversity, have played an evolutionary role in the development of cellular life, with reverse transcriptase playing a central role.[13]


Structure

The reverse transcriptase employs a "right hand" structure similar to that found in other viral nucleic acid polymerases.[14][15] In addition to the transcription function, retroviral reverse transcriptases have a domain belonging to the RNase H family, which is vital to their replication. By degrading the RNA template, it allows the other strand of DNA to be synthesized.[16] Some fragments from the digestion also serves as the primer for the DNA polymerase (either the same enzyme or a host protein), responsible for making the other (plus) strand.[17]


Replication fidelity

There are three different replication systems during the life cycle of a retrovirus. First of all, the reverse transcriptase synthesizes viral DNA from viral RNA, and then from newly made complementary DNA strand. The second replication process occurs when host cellular DNA polymerase replicates the integrated viral DNA. Lastly, RNA polymerase II transcribes the proviral DNA into RNA, which will be packed into virions. Therefore, mutation can occur during one or all of these replication steps.[18]

Reverse transcriptase has a high error rate when transcribing RNA into DNA since, unlike most other DNA polymerases, it has no proofreading ability. This high error rate allows mutations to accumulate at an accelerated rate relative to proofread forms of replication. The commercially available reverse transcriptases produced by Promega are quoted by their manuals as having error rates in the range of 1 in 17,000 bases for AMV and 1 in 30,000 bases for M-MLV.[19]

Other than creating single-nucleotide polymorphisms, reverse transcriptases have also been shown to be involved in processes such as transcript fusions, exon shuffling and creating artificial antisense transcripts.[20][21] It has been speculated that this _TEMPLATE SWITCHING_ activity of reverse transcriptase, which can be demonstrated completely _in vivo_, may have been one of the causes for finding several thousand unannotated transcripts in the genomes of model organisms.[22]


Applications

(AZT), a drug used to inhibit HIV reverse transcriptase]]

Antiviral drugs

As HIV uses reverse transcriptase to copy its genetic material and generate new viruses (part of a retrovirus proliferation circle), specific drugs have been designed to disrupt the process and thereby suppress its growth. Collectively, these drugs are known as reverse-transcriptase inhibitors and include the nucleoside and nucleotide analogues zidovudine (trade name Retrovir), lamivudine (Epivir) and tenofovir (Viread), as well as non-nucleoside inhibitors, such as nevirapine (Viramune).

Molecular biology

Reverse transcriptase is commonly used in research to apply the polymerase chain reaction technique to RNA in a technique called reverse transcription polymerase chain reaction (RT-PCR). The classical PCR technique can be applied only to DNA strands, but, with the help of reverse transcriptase, RNA can be transcribed into DNA, thus making PCR analysis of RNA molecules possible. Reverse transcriptase is used also to create cDNA libraries from mRNA. The commercial availability of reverse transcriptase greatly improved knowledge in the area of molecular biology, as, along with other enzymes, it allowed scientists to clone, sequence, and characterise RNA.

Reverse transcriptase has also been employed in insulin production. By inserting eukaryotic mRNA for insulin production along with reverse transcriptase into bacteria, the mRNA could be inserted into the prokaryote's genome. Large amounts of insulin can then be created, sidestepping the need to harvest pig pancreas and other such traditional sources. Directly inserting eukaryotic DNA into bacteria would not work because it carries introns, so would not translate successfully using the bacterial ribosomes. Processing in the eukaryotic cell during mRNA production removes these introns to provide a suitable template. Reverse transcriptase converted this edited RNA back into DNA so it could be incorporated in the genome.


See also

-   cDNA library
-   DNA polymerase
-   msDNA
-   Reverse transcribing virus
-   RNA polymerase
-   Telomerase
-   Retrotransposon marker


References


External links

-   -   animation of reverse transcriptase action and three reverse transcriptase inhibitors
-   Molecule of the month (September 2002) at the RCSB PDB
-   HIV Replication 3D Medical Animation. (Nov 2008). Video by Boehringer Ingelheim.
-   {{ cite web | url= http://www.rcsb.org/pdb/101/motm.do?momID=33 | title= Molecule of the Month: Reverse Transcriptase (Sep 2002)|access-date= 2013-01-13 | author= Goodsell DS | publisher= Research Collaboratory for Structural Bioinformatics (RCSB) Protein Data Bank (PDB) }}
-   PDBe-KB provides an overview of all the structure information available in the PDB for Human immunodeficiency virus Reverse transcriptase

Category:Genetics Category:EC 2.7.7 Category:Molecular biology Category:Viral enzymes Category:Telomeres Category:1970 in biology Category:RNA reverse-transcribing viruses

[1]

[2]

[3]

[4]

[5]

[6] Bio-Medicine.org - Retrovirus Retrieved on 17 Feb, 2009

[7]

[8]

[9]

[10]

[11]

[12]

[13]

[14]

[15]

[16]

[17]

[18]

[19]

[20]

[21]

[22]