Peptide_syn.pngic protein molecules whose sequence is controlled by the sequence of messenger RNA molecules. This is required by all living cells and associated viruses.]]

RIBOSOMES ([1]) comprise a complex macromolecular machine, found within all living cells, that serves as the site of biological protein synthesis (translation). Ribosomes link amino acids together in the order specified by messenger RNA (mRNA) molecules. Ribosomes consist of two major components: the small ribosomal subunits, which read the mRNA, and the large subunits, which join amino acids to form a polypeptide chain. Each subunit consists of one or more ribosomal RNA (rRNA) molecules and a variety of ribosomal proteins (r-protein or rProtein[2][3][4]). The ribosomes and associated molecules are also known as the _translational apparatus_.


Overview

The sequence of DNA, which encodes the sequence of the amino acids in a protein, is copied into a messenger RNA chain. It may be copied many times into RNA chains. Ribosomes can bind to a messenger RNA chain and use its sequence for determining the correct sequence of amino acids for generating a given protein. Amino acids are selected, collected, and carried to the ribosome by transfer RNA (tRNA) molecules, which enter one part of the ribosome and bind to the messenger RNA chain. It is during this binding that the correct translation of nucleic acid sequence to amino acid sequence occurs. For each coding triplet in the messenger RNA there is a distinct transfer RNA that matches and which carries the correct amino acid for that coding triplet. The attached amino acids are then linked together by another part of the ribosome. Once the protein is produced, it can then fold to produce a specific functional three-dimensional structure although during synthesis some proteins start folding into their correct form.

A ribosome is made from complexes of RNAs and proteins and is therefore a ribonucleoprotein. Each ribosome is divided into two subunits:

1.  a smaller subunit which binds to a larger subunit and the mRNA pattern, and
2.  a larger subunit which binds to the tRNA, the amino acids, and the smaller subunit.

When a ribosome finishes reading an mRNA molecule, these two subunits split apart. Ribosomes are ribozymes, because the catalytic peptidyl transferase activity that links amino acids together is performed by the ribosomal RNA. Ribosomes are often associated with the intracellular membranes that make up the rough endoplasmic reticulum.

Ribosomes from bacteria, archaea and eukaryotes in the three-domain system, resemble each other to a remarkable degree, evidence of a common origin. They differ in their size, sequence, structure, and the ratio of protein to RNA. The differences in structure allow some antibiotics to kill bacteria by inhibiting their ribosomes, while leaving human ribosomes unaffected. In bacteria and archaea, more than one ribosome may move along a single mRNA chain at one time, each "reading" its sequence and producing a corresponding protein molecule.

The mitochondrial ribosomes of eukaryotic cells, are produced from mitochondrial genes, and functionally resemble many features of those in bacteria, reflecting the likely evolutionary origin of mitochondria.[5][6]


Discovery

Ribosomes were first observed in the mid-1950s by Romanian-American cell biologist George Emil Palade, using an electron microscope, as dense particles or granules.[7] The term "ribosome" was proposed by scientist Richard B. Roberts in the end of 1950s:

Albert Claude, Christian de Duve, and George Emil Palade were jointly awarded the Nobel Prize in Physiology or Medicine, in 1974, for the discovery of the ribosome.[8] The Nobel Prize in Chemistry 2009 was awarded to Venkatraman Ramakrishnan, Thomas A. Steitz and Ada E. Yonath for determining the detailed structure and mechanism of the ribosome.[9]


Structure

ribosome_shape.png The ribosome is a highly complex cellular machine. It is largely made up of specialized RNA known as ribosomal RNA (rRNA) as well as dozens of distinct proteins (the exact number varies slightly between species). The ribosomal proteins and rRNAs are arranged into two distinct ribosomal pieces of different size, known generally as the large and small subunit of the ribosome. Ribosomes consist of two subunits that fit together (Figure 2) and work as one to translate the mRNA into a polypeptide chain during protein synthesis (Figure 1). Because they are formed from two subunits of non-equal size, they are slightly longer in the axis than in diameter.

Bacterial ribosomes

Prokaryotic ribosomes are around 20 nm (200 Å) in diameter and are composed of 65% rRNA and 35% ribosomal proteins.[10] Eukaryotic ribosomes are between 25 and 30 nm (250–300 Å) in diameter with an rRNA-to-protein ratio that is close to 1.[11] Crystallographic work [12] has shown that there are no ribosomal proteins close to the reaction site for polypeptide synthesis. This suggests that the protein components of ribosomes do not directly participate in peptide bond formation catalysis, but rather that these proteins act as a scaffold that may enhance the ability of rRNA to synthesize protein (See: Ribozyme).

010_small_subunit-1FKA.gif''.[13] Proteins are shown in blue and the single RNA chain in brown.]]

The ribosomal subunits of bacteria and eukaryotes are quite similar.[14]

The unit of measurement used to describe the ribosomal subunits and the rRNA fragments is the Svedberg unit, a measure of the rate of sedimentation in centrifugation rather than size. This accounts for why fragment names do not add up: for example, bacterial 70S ribosomes are made of 50S and 30S subunits.

Bacteria have 70S ribosomes, each consisting of a small (30S) and a large (50S) subunit. _E. coli_, for example, has a 16S RNA subunit (consisting of 1540 nucleotides) that is bound to 21 proteins. The large subunit is composed of a 5S RNA subunit (120 nucleotides), a 23S RNA subunit (2900 nucleotides) and 31 proteins.[15]

    {| class="wikitable float-right" style="text-align:center"

|+ Ribosome of _E. coli_ (a bacterium)[16] |- ! width="25%"| ribosome ! width="25%"| subunit ! width="25%"| rRNAs ! width="25%"| r-proteins |- | rowspan="3" | 70S || rowspan="2" | 50S || 23S (2904 nt) || rowspan="2" | 31 |- | 5S (120 nt) |- | 30S || 16S (1542 nt) || 21 |} Affinity label for the tRNA binding sites on the _E. coli_ ribosome allowed the identification of A and P site proteins most likely associated with the peptidyltransferase activity; labelled proteins are L27, L14, L15, L16, L2; at least L27 is located at the donor site, as shown by E. Collatz and A.P. Czernilofsky.[17][18] Additional research has demonstrated that the S1 and S21 proteins, in association with the 3′-end of 16S ribosomal RNA, are involved in the initiation of translation.[19]

Eukaryotic ribosomes

Eukaryotes have 80S ribosomes located in their cytosol, each consisting of a small (40S) and large (60S) subunit. Their 40S subunit has an 18S RNA (1900 nucleotides) and 33 proteins.[20][21] The large subunit is composed of a 5S RNA (120 nucleotides), 28S RNA (4700 nucleotides), a 5.8S RNA (160 nucleotides) subunits and 46 proteins.[22][23][24]

    {| class="wikitable float-right" style="text-align:center"

|+ eukaryotic cytosolic ribosomes (_R. norvegicus_)[25] |- ! width="25%"| ribosome ! width="25%"| subunit ! width="25%"| rRNAs ! width="25%"| r-proteins |- | rowspan="4" | 80S || rowspan="3" | 60S || 28S (4718 nt) || rowspan="3" | 49 |- | 5.8S (160 nt) |- | 5S (120 nt) |- | 40S || 18S (1874 nt) || 33 |} During 1977, Czernilofsky published research that used affinity labeling to identify tRNA-binding sites on rat liver ribosomes. Several proteins, including L32/33, L36, L21, L23, L28/29 and L13 were implicated as being at or near the peptidyl transferase center.[26]

Plastoribosomes and mitoribosomes

In eukaryotes, ribosomes are present in mitochondria (sometimes called mitoribosomes) and in plastids such as chloroplasts (also called plastoribosomes). They also consist of large and small subunits bound together with proteins into one 70S particle.[27] These ribosomes are similar to those of bacteria and these organelles are thought to have originated as symbiotic bacteria[28] Of the two, chloroplastic ribosomes are closer to bacterial ones than mitochrondrial ones are. Many pieces of ribosomal RNA in the mitochrondria are shortened, and in the case of 5S rRNA, replaced by other structures in animals and fungi.[29] In particular, _Leishmania tarentolae_ has a minimalized set of mitochondrial rRNA.[30]

The cryptomonad and chlorarachniophyte algae may contain a nucleomorph that resembles a vestigial eukaryotic nucleus.[31] Eukaryotic 80S ribosomes may be present in the compartment containing the nucleomorph.

Making use of the differences

The differences between the bacterial and eukaryotic ribosomes are exploited by pharmaceutical chemists to create antibiotics that can destroy a bacterial infection without harming the cells of the infected person. Due to the differences in their structures, the bacterial 70S ribosomes are vulnerable to these antibiotics while the eukaryotic 80S ribosomes are not.[32] Even though mitochondria possess ribosomes similar to the bacterial ones, mitochondria are not affected by these antibiotics because they are surrounded by a double membrane that does not easily admit these antibiotics into the organelle.[33] The same cannot be said of chloroplasts, where antibiotic resistance in ribosomal proteins is a trait to be introduced as a marker in genetic engineering.[34]

Common properties

The various ribosomes share a core structure, which is quite similar despite the large differences in size. Much of the RNA is highly organized into various tertiary structural motifs, for example pseudoknots that exhibit coaxial stacking. The extra RNA in the larger ribosomes is in several long continuous insertions, such that they form loops out of the core structure without disrupting or changing it.[35] All of the catalytic activity of the ribosome is carried out by the RNA; the proteins reside on the surface and seem to stabilize the structure.[36]

High-resolution structure

10_large_subunit.gif''. Proteins are shown in blue and the two RNA chains in brown and yellow.[37] The small patch of green in the center of the subunit is the active site.]] The general molecular structure of the ribosome has been known since the early 1970s. In the early 2000s, the structure has been achieved at high resolutions, of the order of a few ångströms.

The first papers giving the structure of the ribosome at atomic resolution were published almost simultaneously in late 2000. The 50S (large prokaryotic) subunit was determined from the archaeon _Haloarcula marismortui_[38] and the bacterium _Deinococcus radiodurans_,[39] and the structure of the 30S subunit was determined from _Thermus thermophilus_.[40] These structural studies were awarded the Nobel Prize in Chemistry in 2009. In May 2001 these coordinates were used to reconstruct the entire _T. thermophilus_ 70S particle at 5.5 Å resolution.[41]

Two papers were published in November 2005 with structures of the _Escherichia coli_ 70S ribosome. The structures of a vacant ribosome were determined at 3.5 Å resolution using X-ray crystallography.[42] Then, two weeks later, a structure based on cryo-electron microscopy was published,[43] which depicts the ribosome at 11–15 Å resolution in the act of passing a newly synthesized protein strand into the protein-conducting channel.

The first atomic structures of the ribosome complexed with tRNA and mRNA molecules were solved by using X-ray crystallography by two groups independently, at 2.8 Å[44] and at 3.7 Å.[45] These structures allow one to see the details of interactions of the _Thermus thermophilus_ ribosome with mRNA and with tRNAs bound at classical ribosomal sites. Interactions of the ribosome with long mRNAs containing Shine-Dalgarno sequences were visualized soon after that at 4.5–5.5 Å resolution.[46]

In 2011, the first complete atomic structure of the eukaryotic 80S ribosome from the yeast _Saccharomyces cerevisiae_ was obtained by crystallography.[47] The model reveals the architecture of eukaryote-specific elements and their interaction with the universally conserved core. At the same time, the complete model of a eukaryotic 40S ribosomal structure in _Tetrahymena thermophila_ was published and described the structure of the 40S subunit, as well as much about the 40S subunit's interaction with eIF1 during translation initiation.[48] Similarly, the eukaryotic 60S subunit structure was also determined from _Tetrahymena thermophila_ in complex with eIF6.[49]


Function

Ribosomes are minute particles consisting of RNA and associated proteins that function to synthesize proteins. Proteins are needed for many cellular functions such as repairing damage or directing chemical processes. Ribosomes can be found floating within the cytoplasm or attached to the endoplasmic reticulum. Basically, their main function is to convert genetic code into an amino acid sequence and to build protein polymers from amino acid monomers.

Ribosomes act as catalysts in two extremely important biological processes called peptidyl transfer and peptidyl hydrolysis.[50] The "PT center is responsible for producing protein bonds during protein elongation".[51]

Translation

Ribosomes are the workplaces of protein biosynthesis, the process of translating mRNA into protein. The mRNA comprises a series of codons which are decoded by the ribosome so as to make the protein. Using the mRNA as a template, the ribosome traverses each codon (3 nucleotides) of the mRNA, pairing it with the appropriate amino acid provided by an aminoacyl-tRNA. Aminoacyl-tRNA contains a complementary anticodon on one end and the appropriate amino acid on the other. For fast and accurate recognition of the appropriate tRNA, the ribosome utilizes large conformational changes (conformational proofreading) .[52] The small ribosomal subunit, typically bound to an aminoacyl-tRNA containing the first amino acid methionine, binds to an AUG codon on the mRNA and recruits the large ribosomal subunit. The ribosome contains three RNA binding sites, designated A, P and E. The A-site binds an aminoacyl-tRNA or termination release factors;[53][54] the P-site binds a peptidyl-tRNA (a tRNA bound to the poly-peptide chain); and the E-site (exit) binds a free tRNA. Protein synthesis begins at a start codon AUG near the 5' end of the mRNA. mRNA binds to the P site of the ribosome first. The ribosome recognizes the start codon by using the Shine-Dalgarno sequence of the mRNA in prokaryotes and Kozak box in eukaryotes.

Although catalysis of the peptide bond involves the C2 hydroxyl of RNA's P-site adenosine in a proton shuttle mechanism, other steps in protein synthesis (such as translocation) are caused by changes in protein conformations. Since their catalytic core is made of RNA, ribosomes are classified as "ribozymes,"[55] and it is thought that they might be remnants of the RNA world.[56]

Ribosomer_i_arbete.png

In Figure 5, both ribosomal subunits (small and large) assemble at the start codon (towards the 5' end of the mRNA). The ribosome uses tRNA that matches the current codon (triplet) on the mRNA to append an amino acid to the polypeptide chain. This is done for each triplet on the mRNA, while the ribosome moves towards the 3' end of the mRNA. Usually in bacterial cells, several ribosomes are working parallel on a single mRNA, forming what is called a _polyribosome_ or _polysome_.

Cotranslational folding

The ribosome is known to actively participate in the protein folding.[57][58] The structures obtained in this way are usually identical to the ones obtained during protein chemical refolding, however, the pathways leading to the final product may be different.[59][60] In some cases, the ribosome is crucial in obtaining the functional protein form. For example, one of the possible mechanisms of folding of the deeply knotted proteins relies on the ribosome pushing the chain through the attached loop.[61]

Addition of translation-independent amino acids

Presence of a ribosome quality control protein Rqc2 is associated with mRNA-independent protein elongation.[62][63] This elongation is a result of ribosomal addition (via tRNAs brought by Rqc2) of _CAT tails_: ribosomes extend the _C_-terminus of a stalled protein with random, translation-independent sequences of _A_lanines and _T_hreonines.[64][65]


Ribosome locations

Ribosomes are classified as being either "free" or "membrane-bound".

Protein_translation.gif.]] Free and membrane-bound ribosomes differ only in their spatial distribution; they are identical in structure. Whether the ribosome exists in a free or membrane-bound state depends on the presence of an ER-targeting signal sequence on the protein being synthesized, so an individual ribosome might be membrane-bound when it is making one protein, but free in the cytosol when it makes another protein.

Ribosomes are sometimes referred to as organelles, but the use of the term _organelle_ is often restricted to describing sub-cellular components that include a phospholipid membrane, which ribosomes, being entirely particulate, do not. For this reason, ribosomes may sometimes be described as "non-membranous organelles".

Free ribosomes

Free ribosomes can move about anywhere in the cytosol, but are excluded from the cell nucleus and other organelles. Proteins that are formed from free ribosomes are released into the cytosol and used within the cell. Since the cytosol contains high concentrations of glutathione and is, therefore, a reducing environment, proteins containing disulfide bonds, which are formed from oxidized cysteine residues, cannot be produced within it.

Membrane-bound ribosomes

When a ribosome begins to synthesize proteins that are needed in some organelles, the ribosome making this protein can become "membrane-bound". In eukaryotic cells this happens in a region of the endoplasmic reticulum (ER) called the "rough ER". The newly produced polypeptide chains are inserted directly into the ER by the ribosome undertaking vectorial synthesis and are then transported to their destinations, through the secretory pathway. Bound ribosomes usually produce proteins that are used within the plasma membrane or are expelled from the cell via _exocytosis_.[66]


Biogenesis

In bacterial cells, ribosomes are synthesized in the cytoplasm through the transcription of multiple ribosome gene operons. In eukaryotes, the process takes place both in the cell cytoplasm and in the nucleolus, which is a region within the cell nucleus. The assembly process involves the coordinated function of over 200 proteins in the synthesis and processing of the four rRNAs, as well as assembly of those rRNAs with the ribosomal proteins.


Origin

The ribosome may have first originated in an RNA world, appearing as a self-replicating complex that only later evolved the ability to synthesize proteins when amino acids began to appear.[67] Studies suggest that ancient ribosomes constructed solely of rRNA could have developed the ability to synthesize peptide bonds.[68][69][70] In addition, evidence strongly points to ancient ribosomes as self-replicating complexes, where the rRNA in the ribosomes had informational, structural, and catalytic purposes because it could have coded for tRNAs and proteins needed for ribosomal self-replication.[71] Hypothetical cellular organisms with self-replicating RNA but without DNA are called ribocytes (or ribocells).[72][73]

As amino acids gradually appeared in the RNA world under prebiotic conditions,[74][75] their interactions with catalytic RNA would increase both the range and efficiency of function of catalytic RNA molecules.[76] Thus, the driving force for the evolution of the ribosome from an ancient self-replicating machine into its current form as a translational machine may have been the selective pressure to incorporate proteins into the ribosome’s self-replicating mechanisms, so as to increase its capacity for self-replication.[77]


Specialized ribosome

Many textbooks suggest that there are only two kinds of ribosomes, namely prokaryotic and eukaryotic ribosomes. However, ribosomes are surprisingly heterogeneous with different compositions in different species. Heterogeneous ribosomes have different structures and thus different activities compared to typical ribosomes in major model organisms.

Journal.pone.0067609.g005.png (REN) gene and FF (Firefly) luciferase. Highlighted in green are the RPs on LSU that increase the expression of LAMB3 reporter with premature termination codons (PTC).[78]]]

Heterogeneity in ribosome composition has been proposed to be involved in translational control of protein synthesis.[79] Vincent Mauro and Gerald Edelman proposed the ribosome filter hypothesis to explain the regulatory functions of ribosomes. Emerging evidence has shown that specialized ribosomes specific to different cell populations can affect how genes are translated.[80] Some ribosomal proteins exchange from the assembled complex with cytosolic copies [81] suggesting that the structure of the _in vivo_ ribosome can be modified without synthesizing an entire new ribosome.

Ribosomal proteins

A group of highly acidic ribosomal proteins (RPs), also known as P proteins, are known to be present on the 60S subunit in multiple copies in the ribosomes stalk and P proteins mediate selective translation.[82] These P proteins can be found in yeast and mammalian cells. If P proteins are not present in yeast this can cause the yeast to have a cold-sensitive phenotype.[83] If P proteins are not present in human cells, this could cause autophagy induction.[84]

Certain ribosomal proteins are absolutely critical while others are not. For instance, Rpl28 and Rpl5 mutant flies are alive but have abnormally large wings.[85] Rpl38 also appears to be critical in mammals only under very specific conditions: in mice Rpl38 is required for the translation of a subset of Hox mRNA and a mutation of Rpl38 lead to a homeotic transformation with a short tail.[86]

Other ways heterogeneity can arise from POST-TRANSLATIONAL MODIFICATIONS to RPs include acetylation and methylation in species like yeast,[87] _Arabidopsis_,[88] and human cells.[89][90][91] But there is no overall change in protein synthesis.

Modifications to core ribosomal proteins (RPs) can also give rise to the formation of heterogeneous ribosomes. For instance, in yeast, Rpl28 ubiquitination levels vary with the cell cycle. Ribosomes with the polyubiquitinated Rpl28 carry out protein synthesis at a higher rate _in vitro_ compared with ribosomes with monoubiquitinated Rpl28.[92]

1-s2.0-S0300908414003952-fx1_lrg.jpg

Ribosome-associated factors

Heterogeneous ribosomes can also arise from other factors binding to the ribosome surface. For example, ribosome-associated factor (RACK1) is so tightly associated with the ribosome that its binding is resistant to high-salt washes _in vitro_. RACK1 is required for efficient translation of mRNA with short open reading frames.[93]

rRNA heterogeneity

Viral IRES (internal ribosome entry site) translations can also be mediated by specialized ribosomes.  Specifically, 40S ribosomal units without RPS25 in yeast and mammalian cells are unable to begin translation from two viral IRESes, namely the hepatitis C virus IRES and the cricket paralysis virus (CrPV) intergenic region. The CrPV only needs RPS25 to begin translation and mediate ribosome recruitment. Usually if RPS25 is not present in a certain IRES, imitation from these IRESes can be defective.[94][95][96]

rRNA modifications

Heterogeneity of ribosomal RNA modifications plays an important role in structural maintenance and/or function, such as modulating translation and most mRNA modifications are found in highly conserved regions.[97][98] The most common rRNA modifications are pseudouridylation and 2’-O methylation of ribose.[99]


See also

-   Aminoglycosides
-   Biological machines
-   Eukaryotic translation
-   Posttranslational modification
-   Prokaryotic translation
-   Protein dynamics
-   RNA tertiary structure
-   Translation (genetics)
-   Wobble base pair


References


External links

-   Lab computer simulates ribosome in motion
-   Role of the Ribosome, Gwen V. Childs, copied here
-   Ribosome in _Proteopedia_ - The free, collaborative 3D encyclopedia of proteins & other molecules
-   Ribosomal proteins families in ExPASy
-   Molecule of the Month © RCSB Protein Data Bank:
    -   Ribosome
    -   Elongation Factors
    -   Palade
-   3D electron microscopy structures of ribosomes at the EM Data Bank(EMDB)

Category:Ribozymes Category:Protein biosynthesis Ribosome

[1]

[2]

[3]

[4]

[5]

[6]

[7]

[8]

[9]

[10]

[11]

[12]

[13]

[14]

[15]

[16]

[17]

[18]

[19]

[20]

[21]

[22]

[23]

[24]

[25]

[26]

[27]

[28]

[29]

[30]

[31]

[32]

[33]

[34]

[35]

[36]

[37]

[38]

[39]

[40]

[41]

[42]

[43]

[44]

[45]

[46]

[47]

[48]

[49]

[50]  Boundless Microbiology|website=courses.lumenlearning.com|access-date=2018-09-27}}

[51]

[52]

[53]

[54]

[55]

[56]

[57]

[58]

[59]

[60]

[61]

[62]

[63]

[64]

[65]

[66]

[67]

[68]

[69]

[70]

[71]

[72]

[73]

[74]

[75]

[76]

[77]

[78]

[79]

[80]

[81]

[82]

[83]

[84]

[85]

[86]

[87]

[88]

[89]

[90]

[91]

[92]

[93]

[94]

[95]

[96]

[97]

[98]

[99]