An RNA VIRUS is a virus that has RNA (ribonucleic acid) as its genetic material.[1] This nucleic acid is usually single-stranded RNA (ssRNA) but may be double-stranded RNA (dsRNA).[2] Notable human diseases caused by RNA viruses include Ebola virus disease, SARS, rabies, common cold, influenza, hepatitis C, hepatitis E, West Nile fever, polio and measles.

The International Committee on Taxonomy of Viruses (ICTV) classifies RNA viruses as those that belong to _Group III_, _Group IV_ or _Group V_ of the Baltimore classification system of classifying viruses and does not consider viruses with DNA intermediates in their life cycle as RNA viruses.[3] Viruses with RNA as their genetic material which also include DNA intermediates in their replication cycle are called retroviruses, and comprise _Group VI_ of the Baltimore classification. Notable human retroviruses include HIV-1 and HIV-2, the cause of the disease AIDS.

Another term for RNA viruses that explicitly excludes retroviruses is RIBOVIRUS.[4]


Characteristics

Single-stranded RNA viruses and RNA Sense

RNA viruses can be further classified according to the sense or polarity of their RNA into negative-sense and positive-sense, or ambisense RNA viruses. Positive-sense viral RNA is similar to mRNA and thus can be immediately translated by the host cell. Negative-sense viral RNA is complementary to mRNA and thus must be converted to positive-sense RNA by an RNA-dependent RNA polymerase before translation. As such, purified RNA of a positive-sense virus can directly cause infection though it may be less infectious than the whole virus particle. Purified RNA of a negative-sense virus is not infectious by itself as it needs to be transcribed into positive-sense RNA; each virion can be transcribed to several positive-sense RNAs. Ambisense RNA viruses resemble negative-sense RNA viruses, except they also translate genes from the negative strand.[5]

Double-stranded RNA viruses

Structure_of_the_reovirus_virion.png The double-stranded (ds)RNA viruses represent a diverse group of viruses that vary widely in host range (humans, animals, plants, fungi, and bacteria), genome segment number (one to twelve), and virion organization (Triangulation number, capsid layers, spikes, turrets, etc.). Members of this group include the rotaviruses, renowned globally as the most common cause of gastroenteritis in young children, and picobirnaviruses, renowned worldwide as the most commonly occurring virus in fecal samples of both humans and animals with or without signs of diarrhea. Bluetongue virus is an economically important pathogen of cattle and sheep. In recent years, remarkable progress has been made in determining, at atomic and subnanometeric levels, the structures of a number of key viral proteins and of the virion capsids of several dsRNA viruses, highlighting the significant parallels in the structure and replicative processes of many of these viruses.[6]

Mutation rates

RNA viruses generally have very high mutation rates compared to DNA viruses,[7] because viral RNA polymerases lack the proofreading ability of DNA polymerases.[8] This is one reason why it is difficult to make effective vaccines to prevent diseases caused by RNA viruses.[9] Retroviruses also have a high mutation rate even though their DNA intermediate integrates into the host genome (and is thus subject to host DNA proofreading once integrated), because errors during reverse transcription are embedded into both strands of DNA before integration.[10] Some genes of RNA virus are important to the viral replication cycles and mutations are not tolerated. For example, the region of the hepatitis C virus genome that encodes the core protein is highly conserved,[11] because it contains an RNA structure involved in an internal ribosome entry site.[12]


Replication

Animal RNA viruses are classified by the ICTV. There are three distinct groups of RNA viruses depending on their genome and mode of replication:

-   Double-stranded RNA viruses (Group III) contain from one to a dozen different RNA molecules, each coding for one or more viral proteins.
-   Positive-sense ssRNA viruses (Group IV) have their genome directly utilized as mRNA, with host ribosomes translating it into a single protein that is modified by host and viral proteins to form the various proteins needed for replication. One of these includes RNA-dependent RNA polymerase (RNA replicase), which copies the viral RNA to form a double-stranded replicative form. In turn this dsRNA directs the formation of new viral RNA.
-   Negative-sense ssRNA viruses (Group V) must have their genome copied by an RNA replicase to form positive-sense RNA. This means that the virus must bring along with it the RNA replicase enzyme. The positive-sense RNA molecule then acts as viral mRNA, which is translated into proteins by the host ribosomes.

Retroviruses (Group VI) have a single-stranded RNA genome but, in general, are not considered RNA viruses because they use DNA intermediates to replicate. Reverse transcriptase, a viral enzyme that comes from the virus itself after it is uncoated, converts the viral RNA into a complementary strand of DNA, which is copied to produce a double-stranded molecule of viral DNA. After this DNA is integrated into the host genome using the viral enzyme integrase, expression of the encoded genes may lead to the formation of new virions.


Classification

Classification of the RNA viruses has proven to be a difficult problem. This is in part due to the high mutation rates these genomes undergo. Classification is based principally on the type of genome (double-stranded, negative- or positive-single-strand) and gene number and organisation. Currently there are 5 orders and 47 families of RNA viruses recognised. There are also many unassigned species and genera.

Related to but distinct from the RNA viruses are the viroids and the RNA satellite viruses. These are not currently classified as RNA viruses and are described on their own pages.

A study of several thousand RNA viruses has shown the presence of at least five main taxa: a levivirus and relatives group; a picornavirus supergroup; an alphavirus supergroup plus a flavivirus supergroup; the dsRNA viruses; and the -ve strand viruses.[13] The lentivirus group appears to be basal to all the remaining RNA viruses. The next major division lies between the picornasupragroup and the remaining viruses. The dsRNA viruses appear to have evolved from a +ve RNA ancestor and the -ve RNA viruses from within the dsRNA viruses. The closest reltion to the -ve stranded RNA viruses are the Reoviridae.

Positive strand RNA viruses

This is the single largest group of RNA viruses[14] with 30 families. Attempts have been made to group these families in higher orders. These proposals were based on an analysis of the RNA polymerases and are still under consideration. To date, the suggestions proposed have not been broadly accepted because of doubts over the suitability of a single gene to determine the taxonomy of the clade.

The proposed classification of positive-strand RNA viruses is based on the RNA-dependent RNA polymerase. Three groups have been recognised:[15]

1.  Bymoviruses, comoviruses, nepoviruses, nodaviruses, picornaviruses, potyviruses, sobemoviruses and a subset of luteoviruses (beet western yellows virus and potato leafroll virus)—the picorna like group (Picornavirata).
2.  Carmoviruses, dianthoviruses, flaviviruses, pestiviruses, statoviruses, tombusviruses, single-stranded RNA bacteriophages, hepatitis C virus and a subset of luteoviruses (barley yellow dwarf virus)—the flavi like group (Flavivirata).
3.  Alphaviruses, carlaviruses, furoviruses, hordeiviruses, potexviruses, rubiviruses, tobraviruses, tricornaviruses, tymoviruses, apple chlorotic leaf spot virus, beet yellows virus and hepatitis E virus—the alpha like group (Rubivirata).

A division of the alpha-like (Sindbis-like) supergroup on the basis of a novel domain located near the N termini of the proteins involved in viral replication has been proposed.[16] The two groups proposed are: the 'altovirus' group (alphaviruses, furoviruses, hepatitis E virus, hordeiviruses, tobamoviruses, tobraviruses, tricornaviruses and probably rubiviruses); and the 'typovirus' group (apple chlorotic leaf spot virus, carlaviruses, potexviruses and tymoviruses).

The alpha like supergroup can be further divided into three clades: the rubi-like, tobamo-like, and tymo-like viruses.[17]

Additional work has identified five groups of positive-stranded RNA viruses containing four, three, three, three, and one order(s), respectively.[18] These fourteen orders contain 31 virus families (including 17 families of plant viruses) and 48 genera (including 30 genera of plant viruses). This analysis suggests that alphaviruses and flaviviruses can be separated into two families—the Togaviridae and Flaviridae, respectively—but suggests that other taxonomic assignments, such as the pestiviruses, hepatitis C virus, rubiviruses, hepatitis E virus, and arteriviruses, may be incorrect. The coronaviruses and toroviruses appear to be distinct families in distinct orders and not distinct genera of the same family as currently classified. The luteoviruses appear to be two families rather than one, and apple chlorotic leaf spot virus appears not to be a closterovirus but a new genus of the Potexviridae.

Evolution

The evolution of the picornaviruses based on an analysis of their RNA polymerases and helicases appears to date to the divergence of the eukaryotes.[19] Their putative ancestors include the bacterial group II retroelements, the family of HtrA proteases and DNA bacteriophages.

Partitiviruses are related to and may have evolved from a totivirus ancestor.[20]

Hypoviruses and barnaviruses appear to share an ancestry with the potyvirus and sobemovirus lineages respectively.[21]

Double-stranded RNA viruses

This analysis also suggests that the dsRNA viruses are not closely related to each other but instead belong to four additional classes—Birnaviridae, Cystoviridae, Partitiviridae, and Reoviridae — and one additional order (Totiviridae) of one of the classes of positive ssRNA viruses in the same subphylum as the positive-strand RNA viruses.

One study has suggested that there are two large clades: One includes the Caliciviridae, Flaviviridae, and Picornaviridae families and a second that includes the Alphatetraviridae, Birnaviridae and Cystoviridae, Nodaviridae, and Permutotretraviridae families.[22]

Negative strand RNA viruses

These viruses have multiple types of genome ranging from a single RNA molecule up to eight segments. Despite their diversity it appears that they may have originated in arthropods and to have diversified from there.[23]

Satellite viruses

A number of satellite viruses - viruses that require the assistance of another virus to complete their life cycle - are also known. Their taxonomy has yet to be settled. The following four genera have been proposed for positive sense single stranded RNA satellite viruses that infect plants - Albetovirus, Aumaivirus, Papanivirus and Virtovirus.[24] A family - Sarthroviridae which includes the genus Macronovirus - has been proposed for the positive sense single stranded RNA satellite viruses that infect arthropods.


Group III—dsRNA viruses

There are twelve families and a number of unassigned genera and species recognised in this group.[25]

-   Family Amalgaviridae
-   Family Birnaviridae
-   Family Chrysoviridae
-   Family Cystoviridae
-   Family Endornaviridae
-   Family Hypoviridae
-   Family Megabirnaviridae
-   Family Partitiviridae
-   Family Picobirnaviridae
-   Family Reoviridae—includes Rotavirus
-   Family Totiviridae
-   Family Quadriviridae
-   Genus Botybirnavirus
-   Unassigned species
    -   _Botrytis porri RNA virus 1_
    -   _Circulifer tenellus virus 1_
    -   _Colletotrichum camelliae filamentous virus 1_
    -   _Cucurbit yellows associated virus_
    -   _Sclerotinia sclerotiorum debilitation-associated virus_
    -   _Spissistilus festinus virus 1_


Group IV—positive-sense ssRNA viruses

There are three orders and 34 families recognised in this group. In addition, there are a number of unclassified species and genera.

-   Order Nidovirales
    -   Family Arteriviridae
    -   Family Coronaviridae — includes Coronavirus, SARS
    -   Family Mesoniviridae
    -   Family Roniviridae
-   Order Picornavirales
    -   Family Dicistroviridae
    -   Family Iflaviridae
    -   Family Marnaviridae
    -   Family Picornaviridae—includes Poliovirus, Rhinovirus (a common cold virus), Hepatitis A virus
    -   Family Secoviridae includes subfamily Comovirinae
    -   Genus Bacillariornavirus
    -   Genus Dicipivirus
    -   Genus Labyrnavirus
    -   Genus Sequiviridae
    -   Species Kelp fly virus
-   Order Tymovirales
    -   Family Alphaflexiviridae
    -   Family Betaflexiviridae
    -   Family Gammaflexiviridae
    -   Family Tymoviridae
-   Unassigned
    -   Family Alphatetraviridae
    -   Family Alvernaviridae
    -   Family Astroviridae
    -   Family Barnaviridae
    -   Family Benyviridae
    -   Family Bromoviridae
    -   Family Caliciviridae — includes Norwalk virus
    -   Family Carmotetraviridae
    -   Family Closteroviridae
    -   Family Flaviviridae — includes Yellow fever virus, West Nile virus, Hepatitis C virus, Dengue fever virus, Zika virus
    -   Family Fusariviridae
    -   Family Hepeviridae
    -   Family Hypoviridae
    -   Family Leviviridae
    -   Family Luteoviridae — includes Barley yellow dwarf virus
    -   Family Polycipiviridae
    -   Family Narnaviridae
    -   Family Nodaviridae
    -   Family Permutotetraviridae
    -   Family Potyviridae
    -   Family Sarthroviridae
    -   Family Statovirus
    -   Family Togaviridae — includes Rubella virus, Ross River virus, Sindbis virus, Chikungunya virus
    -   Family Tombusviridae
    -   Family Virgaviridae[26]
    -   Unassigned genera
        -   Genus _Blunervirus_
        -   Genus _Cilevirus_
        -   Genus _Higrevirus_
        -   Genus _Idaeovirus_
        -   Genus _Negevirus_
        -   Genus _Ourmiavirus_
        -   Genus _Polemovirus_
        -   Genus ''Sinaivirus
        -   Genus _Sobemovirus_
    -   Unassigned species
        -   Acyrthosiphon pisum virus
        -   Bastrovirus
        -   Blackford virus
        -   Blueberry necrotic ring blotch virus
        -   Cadicistrovirus
        -   Chara australis virus
        -   Extra small virus
        -   Goji berry chlorosis virus
        -   Hepelivirus
        -   Jingmen tick virus
        -   Le Blanc virus
        -   Nedicistrovirus
        -   Nesidiocoris tenuis virus 1
        -   Niflavirus
        -   Nylanderia fulva virus 1
        -   Orsay virus
        -   Osedax japonicus RNA virus 1
        -   Picalivirus
        -   Plasmopara halstedii virus
        -   Rosellinia necatrix fusarivirus 1
        -   Santeuil virus
        -   Secalivirus
        -   Solenopsis invicta virus 3
        -   Wuhan large pig roundworm virus

Satellite viruses

-   Family Sarthroviridae
-   Genus Albetovirus
-   Genus Aumaivirus
-   Genus Papanivirus
-   Genus Virtovirus
-   Chronic bee paralysis virus

An unclassified astrovirus/hepevirus-like virus has also been described.[27]


Group V—negative-sense ssRNA viruses

With the exception of the Hepatitis D virus, this group of viruses have been placed into a single phylum - Negarnaviricota. This phylum has been divided into two subphyla - Haploviricotina and Polyploviricotina. Within the subphylum Haploviricotina four classes are currently recognised: Chunqiuviricetes, Milneviricetes, Monjiviricetes and Yunchangviricetes. In the subphylum Polyploviricotina two classes are recognised: Ellioviricetes and Insthoviricetes.

Six classes, seven orders and twenty four families are currently recognised in this group. A number of unassigned species and genera are yet to be classified.[28]

-   Phylum _Negarnaviricota_[29]
    -   Subphylum _Haploviricotina_
        -   Class _Chunqiuviricetes_
            -   Order _Muvirales_
                -   Family _Qinviridae_
        -   Class _Milneviricetes_
            -   Order _Serpentovirales_
                -   Family _Aspiviridae_
        -   Class _Monjiviricetes_
            -   Order _Jingchuvirales_
                -   Family _Chuviridae_
            -   Order _Mononegavirales_
                -   Family _Bornaviridae_ — Borna disease virus
                -   Family _Filoviridae_ — includes Ebola virus, Marburg virus
                -   Family ''Mymonaviridae
                -   Family ''Nyamiviridae[30]
                -   Family _Paramyxoviridae_ — includes Measles virus, Mumps virus, Nipah virus, Hendra virus, and NDV
                -   Family _Pneumoviridae_ — includes RSV and Metapneumovirus
                -   Family _Rhabdoviridae_ — includes Rabies virus
                -   Family ''Sunviridae
                -   Genus Anphevirus
                -   Genus Arlivirus
                -   Genus Chengtivirus
                -   Genus Crustavirus
                -   Genus Wastrivirus
        -   Class _Yunchangviricetes_
            -   Order _Goujianvirales_
                -   Family _Yueviridae_
    -   Subphylum _Polyploviricotina_
        -   Class _Ellioviricetes_
            -   Order _Bunyavirales_
                -   Family _Arenaviridae_ — includes Lassa virus
                -   Family _Cruliviridae_
                -   Family _Feraviridae_
                -   Family _Fimoviridae_
                -   Family _Hantaviridae_
                -   Family _Jonviridae_
                -   Family _Nairoviridae_
                -   Family _Peribunyaviridae_
                -   Family _Phasmaviridae_
                -   Family _Phenuiviridae_
                -   Family _Tospoviridae_
                -   Genus _Tilapineviridae_
        -   Class _Insthoviricetes_
            -   Order _Articulavirales_
                -   Family _Amnoonviridae_ — includes Taastrup virus
                -   Family _Orthomyxoviridae_ — includes Influenza viruses

-   Unassigned genera:
    -   Genus _Deltavirus_ — includes Hepatitis D virus


Gallery

Image:Lassa virus.JPG|Lassa virus (_Arenaviridae_) Image:Lymphocytic choriomeningitis virus.jpg|Lymphocytic choriomeningitis virus (_Arenaviridae_) Image:Sin Nombre virus Hanta TEM 1137 lores.jpg|Hantavirus (_Bunyaviridae_) Image:Marburg virus.jpg|Marburg Virus (_Filoviridae_) Image:Ebola virions.png|Ebola virus (_Filoviridae_) Image:Influenza virus particle 8430 lores.jpg|Influenza (_Orthomyxoviridae_) Image:Measles virus.JPG|Measles (_Paramyxoviridae_) Image:Mumps virus, negative stained TEM 8758 lores.jpg|Mumps virus (_Paramyxoviridae_) Image:Respiratory Syncytial Virus (RSV) EM PHIL 2175 lores.jpg|Human respiratory syncytial virus (_Paramyxoviridae_) Image:Parainfluenza virus TEM PHIL 271 lores.jpg|Parainfluenza (_Paramyxoviridae_) Image:Rabies Virus EM PHIL 1876.JPG|Rabies (_Rhabdoviridae_) Image:Vesicular stomatitis virus (VSV) EM 18 lores.jpg|Vesicular stomatitis virus (_Rhabdoviridae_)


Notes

The majority of fungal viruses are double-stranded RNA viruses. A small number of positive-strand RNA viruses have been described. One report has suggested the possibility of a negative stranded virus.[31]


See also

-   Virus classification
-   List of viruses
-   Viral replication
-   Positive/negative-sense
-   Animal viruses
-   Double-stranded RNA viruses
-   Retrovirus
-   DNA viruses
-   Norovirus cis-acting replication element
-   Viroid


References


External links

-   -   Animal viruses

RNA_viruses Category:RNA

[1] MeSH, retrieved on 12 April 2008.

[2]

[3]

[4]

[5]

[6]

[7]

[8]

[9]

[10]

[11]

[12]

[13] Wolf YI, Kazlauskas D, Iranzo J, Lucía-Sanz A, Kuhn JH, Krupovic M, Dolja VV, Koonin EV (2018) Origins and Evolution of the Global RNA Virome. MBio 9(6) pii: e02329-18

[14]

[15]

[16]

[17]

[18]

[19]

[20] Ghabrial SA (1998) Origin, adaptation and evolutionary pathways of fungal viruses. _Virus Genes_ 16(1):119-131

[21]

[22]

[23] Li CX, Shi M, Tian JH, Lin XD, Kang YJ, Chen LJ, Qin XC, Xu J, Holmes EC, Zhang YZ (2015) Unprecedented genomic diversity of RNA viruses in arthropods reveals the ancestry of negative-sense RNA viruses. Elife 4. doi: 10.7554/eLife.05378.

[24] Krupovic M, Kuhn JH, Fischer MG (2016) A classification system for virophages and satellite viruses. Arch Virol 161(1):233-247 doi: 10.1007/s00705-015-2622-9

[25]

[26]

[27] Pankovics P, Boros Á, Kiss T, Engelmann P, Reuter G (2019) Genetically highly divergent RNA virus with astrovirus-like (5'-end) and hepevirus-like (3'-end) genome organization in carnivorous birds, European roller (_Coracias garrulus_). Infect Genet Evol

[28]

[29]

[30]

[31]