In the genetic code, a STOP CODON (or TERMINATION CODON) is a nucleotide triplet within messenger RNA that signals a termination of translation into proteins.[1] Proteins are based on polypeptides, which are unique sequences of amino acids. Most codons in messenger RNA (from DNA) correspond to the addition of an amino acid to a growing polypeptide chain, which may ultimately become a protein. Stop codons signal the termination of this process by binding release factors, which cause the ribosomal subunits to disassociate, releasing the amino acid chain. While start codons need nearby sequences or initiation factors to start translation, a stop codon alone is sufficient to initiate termination.


Introduction

In the standard genetic code, there are three different stop codons:

-   in RNA:
    -   UAG ("amber")
    -   UAA ("ochre")
    -   UGA ("opal")
-   in DNA:
    -   TAG ("amber")
    -   TAA ("ochre")
    -   TGA ("opal" or "umber")

In 2007, the UGA codon was identified as the codon coding for selenocysteine (Sec) and found in 25 selenoproteins located in the active site of the protein. Translation of this codon is enabled by the proximity of the SECIS element (SElenoCysteine Incorporation Sequence).[2] The UAG codon can translate into pyrrolysine (Pyl) in a similar manner.

Distribution of stop codons within the genome of an organism is non-random and can correlate with GC-content.[3][4] For example, the _E. coli_ K-12 genome contains 2705 TAA (63%), 1257 TGA (29%), and 326 TAG (8%) stop codons (GC content 50.8%).[5] Also the substrates for the stop codons release factor 1 or release factor 2 are strongly correlated to the abundance of stop codons.[6] Large scale study of bacteria with a broad range of GC-contents shows that while the frequency of occurrence of TAA is negatively correlated to the GC-content and the frequency of occurrence of TGA is positively correlated to the GC-content, the frequency of occurrence of the TAG stop codon, which is often the minimally used stop codon in a genome, is not influenced by the GC-content.[7]

Nonsense mutations are changes in DNA sequence that introduce a premature stop codon, causing any resulting protein to be abnormally shortened. This often causes a loss of function in the protein, as critical parts of the amino acid chain are no longer created. Because of this terminology, stop codons have also been referred to as NONSENSE CODONS.

Recognition of stop codons in bacteria have been associated with the so-called 'tripeptide anticodon',[8] a highly conserved amino acid motif in RF1 (PxT) and RF2 (SPF). Even though this is supported by structural studies, it was shown that the tripeptide anticodon hypothesis is an oversimplification.[9]


Amber, ochre, and opal nomenclature

Stop codons were historically given many different names, as they each corresponded to a distinct class of mutants that all behaved in a similar manner. These mutants were first isolated within bacteriophages (T4 and lambda), viruses that infect the bacteria _Escherichia coli_. Mutations in viral genes weakened their infectious ability, sometimes creating viruses that were able to infect and grow within only certain varieties of _E. coli_.

amber mutations (UAG): were the first set of nonsense mutations to be discovered, isolated by Richard Epstein and Charles Steinberg and named after their friend Harris Bernstein (whose last name means "amber" in German).[10]

    Viruses with amber mutations are characterized by their ability to infect only certain strains of bacteria, known as AMBER SUPPRESSORS. These bacteria carry their own mutation that allows a recovery of function in the mutant viruses. For example, a mutation in the tRNA that recognizes the amber stop codon allows translation to "read through" the codon and produce a full-length protein, thereby recovering the normal form of the protein and "suppressing" the amber mutation.[11]
    Thus, amber mutants are an entire class of virus mutants that can grow in bacteria that contain amber suppressor mutations. Similar suppressors are known for ochre and opal stop codons as well.

ochre mutation (UAA): was the second stop codon mutation to be discovered. Given a color name to match the name of amber mutants, ochre mutant viruses had a similar property in that they recovered infectious ability within certain suppressor strains of bacteria. The set of ochre suppressors was distinct from amber suppressors, so ochre mutants were inferred to correspond to a different nucleotide triplet. Through a series of mutation experiments comparing these mutants with each other and other known amino acid codons, Sydney Brenner concluded that the amber and ochre mutations corresponded to the nucleotide triplets "UAG" and "UAA".[12]

opal mutations or umber mutations (UGA): the third and last stop codon in the standard genetic code was discovered soon after, corresponding to the nucleotide triplet "UGA".[13] Nonsense mutations that created this premature stop codon were later called _opal mutations_ or _umber mutations_.


Hidden stops

HIDDEN STOPS are non-stop codons that would be read as stop codons if they were frameshifted +1 or âˆ’1. These prematurely terminate translation if the corresponding frame-shift (such as due to a ribosomal RNA slip) occurs before the hidden stop. It is hypothesised that this decreases resource waste on nonfunctional proteins and the production of potential cytotoxins. Researchers at Louisiana State University propose the _ambush hypothesis_, that hidden stops are selected for. Codons that can form hidden stops are used in genomes more frequently compared to synonymous codons that would otherwise code for the same amino acid. Unstable rRNA in an organism correlates with a higher frequency of hidden stops.[14] This hypothesis however could not be validated with a larger data set.[15]

Stop-codons and hidden stops together are collectively referred as stop-signals. Researchers at University of Memphis found that the ratios of the stop-signals on the three reading frames of a genome (referred to as translation stop-signals ratio or TSSR) of genetically related bacteria, despite their great differences in gene contents, are much alike. This nearly identical Genomic-TSSR value of genetically related bacteria may suggest that bacterial genome expansion is limited by their unique stop-signals bias of that bacterial species.[16]


Translational readthrough

STOP CODON SUPPRESSION or TRANSLATIONAL READTHROUGH occurs when in translation a stop codon is interpreted as a sense codon, that is, when a (standard) amino acid is 'encoded' by the stop codon. Mutated tRNAs can be the cause of readthrough, but also certain nucleotide motifs close to the stop codon. Translational readthrough is very common in viruses and bacteria, and has also been found as a gene regulatory principle in humans, yeasts, bacteria and drosophila.[17][18] This kind of endogenous translational readthough constitutes a variation of the genetic code, because a stop codon codes for an amino acid. In the case of human malate dehydrogenase, the stop codon is read through with a frequency of about 4%.[19] The amino acid inserted at the stop codon depends on the identity of the stop codon itself: Gln, Tyr, and Lys have been found for the UAA and UAG codons, while Cys, Trp, and Arg for the UGA codon have been identified by mass spectrometry.[20]


Nonstop mutations

A NONSTOP MUTATION is a point mutation that occurs within a stop codon. Nonstop mutations cause the continued translation of an mRNA strand into what should be an untranslated region. Most polypeptides resulting from a gene with a nonstop mutation are nonfunctional due to their extreme length. Nonstop mutations differ from nonsense mutations in that they do not create a stop codon but, instead, delete one.

Nonstop mutations have been linked with several congenital diseases including congenital adrenal hyperplasia,[21] variable anterior segment dysgenesis,[22] and mitochondrial neurogastrointestinal encephalomyopathy.[23]


Use as a watermark

In 2010 when Craig Venter unveiled the first fully functioning, reproducing cell controlled by synthetic DNA he described how his team used frequent stop codons to create watermarks in RNA and DNA to help confirm the results were indeed synthetic (and not contaminated or otherwise), using it to encode authors' names and website addresses.[24]


See also

-   Genetic code
-   Start codon
-   Terminator gene


References

Category:Molecular genetics Category:Gene expression Category:Protein biosynthesis

[1]

[2]

[3]

[4]

[5]

[6]

[7]

[8]

[9]

[10]

[11]

[12]

[13]

[14]

[15]

[16]

[17]

[18]

[19]

[20]

[21]

[22]

[23]

[24]