SUPEROXIDE DISMUTASE (SOD, ) is an enzyme that alternately catalyzes the dismutation (or partitioning) of the superoxide (O₂^(−)) radical into either ordinary molecular oxygen (O₂) or hydrogen peroxide (H₂O₂). Superoxide is produced as a by-product of oxygen metabolism and, if not regulated, causes many types of cell damage.[1] Hydrogen peroxide is also damaging and is degraded by other enzymes such as catalase. Thus, SOD is an important antioxidant defense in nearly all living cells exposed to oxygen. One exception is _Lactobacillus plantarum_ and related lactobacilli, which use a different mechanism to prevent damage from reactive (O₂^(−)).


Chemical reaction

SODs catalyze the disproportionation of superoxide:

    2 HO₂ → O₂ + H₂O₂

In this way, O₂^(−) is converted into two less damaging species.

The pathway by which SOD-catalyzed dismutation of superoxide may be written, for Cu,Zn SOD, with the following reactions :

-   Cu²⁺-SOD + O₂^(−) → Cu⁺-SOD + O₂ (reduction of copper; oxidation of superoxide)
-   Cu⁺-SOD + O₂^(−) + 2H⁺ → Cu²⁺-SOD + H₂O₂ (oxidation of copper; reduction of superoxide)

The general form, applicable to all the different metal-coordinated forms of SOD, can be written as follows:

-   M^((n+1)+)-SOD + O₂^(−) → M^(n+)-SOD + O₂
-   M^(n+)-SOD + O₂^(−) + 2H⁺ → M^((n+1)+)-SOD + H₂O₂.

where M = Cu (n=1) ; Mn (n=2) ; Fe (n=2) ; Ni (n=2).

In a series of such reactions, the oxidation state and the charge of the metal cation oscillates between n and n+1: +1 and +2 for Cu, or +2 and +3 for the other metals .


Types

General

Irwin Fridovich and Joe McCord at Duke University discovered the enzymatic activity of superoxide dismutase in 1968.[2] SODs were previously known as a group of metalloproteins with unknown function; for example, CuZnSOD was known as erythrocuprein (or hemocuprein, or cytocuprein) or as the veterinary anti-inflammatory drug "Orgotein".[3] Likewise, Brewer (1967) identified a protein that later became known as superoxide dismutase as an indophenol oxidase by protein analysis of starch gels using the phenazine-tetrazolium technique.[4]

There are three major families of superoxide dismutase, depending on the protein fold and the metal cofactor: the Cu/Zn type (which binds both copper and zinc), Fe and Mn types (which bind either iron or manganese), and the Ni type (which binds nickel).

  --------------------------------------------------------------------- ---------------------------------------------- -----------------------------------------
  2SOD_ribbon_colorPencil_WhBkgd.png of bovine Cu-Zn SOD subunit[5]]]   Crystal_Structure_of_Human_Manganese_SOD.png   094-SuperoxideDismutase-Mn_Fe_2mers.png
  --------------------------------------------------------------------- ---------------------------------------------- -----------------------------------------

-   Copper and zinc – most commonly used by eukaryotes, including humans. The cytosols of virtually all eukaryotic cells contain an SOD enzyme with copper and zinc (Cu-Zn-SOD). For example, Cu-Zn-SOD available commercially is normally purified from bovine red blood cells. The bovine Cu-Zn enzyme is a homodimer of molecular weight 32,500. It was the first SOD whose atomic-detail crystal structure was solved, in 1975.[6] It is an 8-stranded "Greek key" beta-barrel, with the active site held between the barrel and two surface loops. The two subunits are tightly joined back-to-back, mostly by hydrophobic and some electrostatic interactions. The ligands of the copper and zinc are six histidine and one aspartate side-chains; one histidine is bound between the two metals.[7]
-   Iron_Superoxide_Dismutase_Active_Site.pngIron or manganese – used by prokaryotes and protists, and in mitochondria and chloroplasts
    -   Iron – Many bacteria contain a form of the enzyme with iron (Fe-SOD); some bacteria contain Fe-SOD, others Mn-SOD, and some (such as _E. coli_) contain both. Fe-SOD can also be found in the chloroplasts of plants. The 3D structures of the homologous Mn and Fe superoxide dismutases have the same arrangement of alpha-helices, and their active sites contain the same type and arrangement of amino acid side-chains. They are usually dimers, but occasionally tetramers.
    -   Manganese – Nearly all mitochondria, and many bacteria, contain a form with manganese (Mn-SOD): For example, the Mn-SOD found in human mitochondria. The ligands of the manganese ions are 3 histidine side-chains, an aspartate side-chain and a water molecule or hydroxy ligand, depending on the Mn oxidation state (respectively II and III).[8]
-   Nickel – prokaryotic. This has a hexameric (6-copy) structure built from right-handed 4-helix bundles, each containing N-terminal hooks that chelate a Ni ion. The Ni-hook contains the motif His-Cys-X-X-Pro-Cys-Gly-X-Tyr; it provides most of the interactions critical for metal binding and catalysis and is, therefore, a likely diagnostic of NiSODs.[9][10]

  ------------ ----------------- ----------------------------------------- --------------------------------------- --------- ------------------------------------------------------ ---------------- ---------------------- --------- --------------------- ------------ -------- -------------- ------------------ ------------ ------------------- ------------------------------------------------------------------- ------------------------------------- --------- ----------------------------------------------------------------------------------------- ---------------- ---------------------- --------- --------------------- ------------- -------- -------------- ------------------ ------------ ------------------- ---------------------------------------------------------------- ------------------------------------- --------- ----------------------------------------------------------------------------------------- ---------------- ---------------------- --------- --------------------- ------------- -------- -------------- ------------------ ------------ ----------------- ------------------------------------ -------------------------------------------- --------- --------------------------------------------------------------------------- ---------------- ---------------------- --------- ----------- ------------- -------- -------------- ------------------
  {{Pfam_box   Symbol = Sod_Cu   Name = Copper/zinc superoxide dismutase   image = 1sdy CuZnSOD dimer ribbon.png   width =   caption = Yeast Cu,Zn superoxide dismutase dimer[11]   Pfam = PF00080   InterPro = IPR001424   SMART =   PROSITE = PDOC00082   SCOP =1sdy   TCDB =   OPM family =   OPM protein = }}   {{Pfam_box   Symbol = Sod_Fe_N   Name = Iron/manganese superoxide dismutases, alpha-hairpin domain   image = 1n0j H mit MnSOD D1 rib.png   width =   caption = Structure of domain1 (color), human mitochondrial Mn superoxide dismutase[12]   Pfam = PF00081   InterPro = IPR001189   SMART =   PROSITE = PDOC00083   SCOP = 1n0j   TCDB =   OPM family =   OPM protein = }}   {{Pfam_box   Symbol = Sod_Fe_C   Name = Iron/manganese superoxide dismutases, C-terminal domain   image = 1n0j H mit MnSOD D2 rib.png   width =   caption = Structure of domain2 (color), human mitochondrial Mn superoxide dismutase[13]   Pfam = PF02777   InterPro = IPR001189   SMART =   PROSITE = PDOC00083   SCOP = 1n0j   TCDB =   OPM family =   OPM protein = }}   {{Pfam_box   Symbol = Sod_Ni   Name = Nickel superoxide dismutase   image = 94-SuperoxideDismutase-Ni 6mer.png   width =   caption = Structure of _Streptomyces_ Ni superoxide dismutase hexamer[14]   Pfam = PF09055   InterPro = IPR014123   SMART =   PROSITE =   SCOP = 1q0d   TCDB =   OPM family =   OPM protein = }}
  ------------ ----------------- ----------------------------------------- --------------------------------------- --------- ------------------------------------------------------ ---------------- ---------------------- --------- --------------------- ------------ -------- -------------- ------------------ ------------ ------------------- ------------------------------------------------------------------- ------------------------------------- --------- ----------------------------------------------------------------------------------------- ---------------- ---------------------- --------- --------------------- ------------- -------- -------------- ------------------ ------------ ------------------- ---------------------------------------------------------------- ------------------------------------- --------- ----------------------------------------------------------------------------------------- ---------------- ---------------------- --------- --------------------- ------------- -------- -------------- ------------------ ------------ ----------------- ------------------------------------ -------------------------------------------- --------- --------------------------------------------------------------------------- ---------------- ---------------------- --------- ----------- ------------- -------- -------------- ------------------

In higher plants, SOD isozymes have been localized in different cell compartments. Mn-SOD is present in mitochondria and peroxisomes. Fe-SOD has been found mainly in chloroplasts but has also been detected in peroxisomes, and CuZn-SOD has been localized in cytosol, chloroplasts, peroxisomes, and apoplast.[15][16]

Human

Three forms of superoxide dismutase are present in humans, in all other mammals, and most chordates. SOD1 is located in the cytoplasm, SOD2 in the mitochondria, and SOD3 is extracellular. The first is a dimer (consists of two units), whereas the others are tetramers (four subunits). SOD1 and SOD3 contain copper and zinc, whereas SOD2, the mitochondrial enzyme, has manganese in its reactive centre. The genes are located on chromosomes 21, 6, and 4, respectively (21q22.1, 6q25.3 and 4p15.3-p15.1).

  ------------------- ---------------------- ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- ---------------------------------- --------- ---------------- --------------- ------------------------ ------------------- --------------- -------------------- ------------------ ------- --------------------- ----------------- --------- ------------- ----------------------------- ------------------- ---------------------------- ---------------------------------------------------------------------------------- --------------------- --------- ---------------- --------------- ----------------------------------- ------------------- --------------- -------------------- ------------------ ------- --------------------- ---------------- --------- ----------- ----------------------------- ------------------- ---------------------------- ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- ----------------------- --------- ---------------- --------------- ------------------------------- ------------------- --------------- -------------------- ------------------ ------- --------------------- ---------------- --------- ------------ ----------------------------------
  {{infobox protein   Name = SOD1, soluble   caption = Crystal structure of the human SOD1 enzyme (rainbow-color N-terminus = blue, C-terminus = red) complexed with copper (orange sphere) and zinc (grey sphere).[17]   image = 2c9v CuZn rib n site.png   width =   HGNCid = 11179   Symbol = SOD1   AltSymbols = ALS, ALS1   EntrezGene = 6647   OMIM = 147450   RefSeq = NM_000454   UniProt = P00441   PDB =   ECnumber = 1.15.1.1   Chromosome = 21   Arm = q   Band = 22.1   LocusSupplementaryData = }}   {{infobox protein   Name = SOD2, mitochondrial   caption = Active site of human mitochondrial Mn superoxide dismutase (SOD2).[18]   image = SODsite.gif   width =   HGNCid = 11180   Symbol = SOD2   AltSymbols = Mn-SOD; IPO-B; MVCD6   EntrezGene = 6648   OMIM = 147460   RefSeq = NM_000636   UniProt = P04179   PDB =   ECnumber = 1.15.1.1   Chromosome = 6   Arm = q   Band = 25   LocusSupplementaryData = }}   {{infobox protein   Name = SOD3, extracellular   caption = Crystallographic structure of the tetrameric human SOD3 enzyme (cartoon diagram) complexed with copper and zinc cations (orange and grey spheres respectively).[19]   image = SOD3_2JLP.png   width =   HGNCid = 11181   Symbol = SOD3   AltSymbols = EC-SOD; MGC20077   EntrezGene = 6649   OMIM = 185490   RefSeq = NM_003102   UniProt = P08294   PDB =   ECnumber = 1.15.1.1   Chromosome = 4   Arm = p   Band = ter   LocusSupplementaryData = -q21 }}
  ------------------- ---------------------- ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- ---------------------------------- --------- ---------------- --------------- ------------------------ ------------------- --------------- -------------------- ------------------ ------- --------------------- ----------------- --------- ------------- ----------------------------- ------------------- ---------------------------- ---------------------------------------------------------------------------------- --------------------- --------- ---------------- --------------- ----------------------------------- ------------------- --------------- -------------------- ------------------ ------- --------------------- ---------------- --------- ----------- ----------------------------- ------------------- ---------------------------- ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- ----------------------- --------- ---------------- --------------- ------------------------------- ------------------- --------------- -------------------- ------------------ ------- --------------------- ---------------- --------- ------------ ----------------------------------

Plants

In higher plants, superoxide dismutase enzymes (SODs) act as antioxidants and protect cellular components from being oxidized by reactive oxygen species (ROS).[20] ROS can form as a result of drought, injury, herbicides and pesticides, ozone, plant metabolic activity, nutrient deficiencies, photoinhibition, temperature above and below ground, toxic metals, and UV or gamma rays.[21][22] To be specific, molecular O₂ is reduced to O₂^(−) (a ROS called superoxide) when it absorbs an excited electron released from compounds of the electron transport chain. Superoxide is known to denature enzymes, oxidize lipids, and fragment DNA.[23] SODs catalyze the production of O₂ and H₂O₂ from superoxide (O₂^(−)), which results in less harmful reactants.

When acclimating to increased levels of oxidative stress, SOD concentrations typically increase with the degree of stress conditions. The compartmentalization of different forms of SOD throughout the plant makes them counteract stress very effectively. There are three well-known and -studied classes of SOD metallic coenzymes that exist in plants. First, Fe SODs consist of two species, one homodimer (containing 1-2 g Fe) and one tetramer (containing 2-4 g Fe). They are thought to be the most ancient SOD metalloenzymes and are found within both prokaryotes and eukaryotes. Fe SODs are most abundantly localized inside plant chloroplasts, where they are indigenous. Second, Mn SODs consist of a homodimer and homotetramer species each containing a single Mn(III) atom per subunit. They are found predominantly in mitochondrion and peroxisomes. Third, Cu-Zn SODs have electrical properties very different from those of the other two classes. These are concentrated in the chloroplast, cytosol, and in some cases the extracellular space. Note that Cu-Zn SODs provide less protection than Fe SODs when localized in the chloroplast.[24][25][26]

Bacteria

Human white blood cells use enzymes such as NADPH oxidase to generate superoxide and other reactive oxygen species to kill bacteria. During infection, some bacteria (e.g., _Burkholderia pseudomallei_) therefore produce superoxide dismutase to protect themselves from being killed.[27]


Biochemistry

SOD out-competes damaging reactions of superoxide, thus protecting the cell from superoxide toxicity. The reaction of superoxide with non-radicals is spin-forbidden. In biological systems, this means that its main reactions are with itself (dismutation) or with another biological radical such as nitric oxide (NO) or with a transition-series metal. The superoxide anion radical (O₂^(−)) spontaneously dismutes to O₂ and hydrogen peroxide (H₂O₂) quite rapidly (~10⁵ M^(−1)s^(−1) at pH 7). SOD is necessary because superoxide reacts with sensitive and critical cellular targets. For example, it reacts with the NO radical, and makes toxic peroxynitrite.

Because the uncatalysed dismutation reaction for superoxide requires two superoxide molecules to react with each other, the dismutation rate is second-order with respect to initial superoxide concentration. Thus, the half-life of superoxide, although very short at high concentrations (e.g., 0.05 seconds at 0.1mM) is actually quite long at low concentrations (e.g., 14 hours at 0.1 nM). In contrast, the reaction of superoxide with SOD is first order with respect to superoxide concentration. Moreover, superoxide dismutase has the largest k_(cat)/K_(M) (an approximation of catalytic efficiency) of any known enzyme (~7 x 10⁹ M^(−1)s^(−1)),[28] this reaction being limited only by the frequency of collision between itself and superoxide. That is, the reaction rate is "diffusion-limited".

The high efficiency of superoxide dismutase seems necessary: even at the subnanomolar concentrations achieved by the high concentrations of SOD within cells, superoxide inactivates the citric acid cycle enzyme aconitase, can poison energy metabolism, and releases potentially toxic iron. Aconitase is one of several iron-sulfur-containing (de)hydratases in metabolic pathways shown to be inactivated by superoxide.[29]


Stability and folding mechanism

SOD1 is an extremely stable protein. In the holo form (both copper and zinc bound) the melting point is > 90 °C. In the apo form (no copper or zinc bound) the melting point is ~ 60 °C.[30] By differential scanning calorimetry (DSC), holo SOD1 unfolds by a two-state mechanism: from dimer to two unfolded monomers.[31] In chemical denaturation experiments, holo SOD1 unfolds by a three-state mechanism with observation of a folded monomeric intermediate.[32]


Physiology

Superoxide is one of the main reactive oxygen species in the cell. As a consequence, SOD serves a key antioxidant role. The physiological importance of SODs is illustrated by the severe pathologies evident in mice genetically engineered to lack these enzymes. Mice lacking SOD2 die several days after birth, amid massive oxidative stress.[33] Mice lacking SOD1 develop a wide range of pathologies, including hepatocellular carcinoma,[34] an acceleration of age-related muscle mass loss,[35] an earlier incidence of cataracts, and a reduced lifespan. Mice lacking SOD3 do not show any obvious defects and exhibit a normal lifespan, though they are more sensitive to hyperoxic injury.[36] Knockout mice of any SOD enzyme are more sensitive to the lethal effects of superoxide-generating compounds, such as paraquat and diquat (herbicides).

_Drosophila_ lacking SOD1 have a dramatically shortened lifespan, whereas flies lacking SOD2 die before birth.

SOD knockdowns in the worm _C. elegans_ do not cause major physiological disruptions. However, the lifespan of _C. elegans_ can be extended by superoxide/catalase mimetics suggesting that oxidative stress is a major determinant of the rate of aging.[37]

Knockout or null mutations in SOD1 are highly detrimental to aerobic growth in the budding yeast _Saccharomyces cerevisiae_ and result in a dramatic reduction in post-diauxic lifespan. In wild-type _S. cerevisiae_, DNA damage rates increased 3-fold with age, but more than 5-fold in mutants deleted for either the _SOD1_ or _SOD2_ genes.[38] Reactive oxygen species levels increase with age in these mutant strains and show a similar pattern to the pattern of DNA damage increase with age. Thus it appears that superoxide dismutase plays a substantial role in preserving genome integrity during aging in _S. cerevisiae_. SOD2 knockout or null mutations cause growth inhibition on respiratory carbon sources in addition to decreased post-diauxic lifespan.

In the fission yeast _Schizosaccharomyces pombe_, deficiency of mitochondrial superoxide dismutase SOD2 accelerates chronological aging.[39]

Several prokaryotic SOD null mutants have been generated, including _E. coli_. The loss of periplasmic CuZnSOD causes loss of virulence and might be an attractive target for new antibiotics.


Role in disease

Mutations in the first SOD enzyme (SOD1) can cause familial amyotrophic lateral sclerosis (ALS, a form of motor neuron disease).[40][41][42][43] The most common mutation in the U.S. is A4V, while the most intensely studied is G93A. The other two isoforms of SOD have not been linked to many human diseases, however, in mice inactivation of SOD2 causes perinatal lethality[44] and inactivation of SOD1 causes hepatocellular carcinoma.[45] Mutations in SOD1 can cause familial ALS (several pieces of evidence also show that wild-type SOD1, under conditions of cellular stress, is implicated in a significant fraction of sporadic ALS cases, which represent 90% of ALS patients.),[46] by a mechanism that is presently not understood, but not due to loss of enzymatic activity or a decrease in the conformational stability of the SOD1 protein. Overexpression of SOD1 has been linked to the neural disorders seen in Down syndrome.[47] In patients with thalassemia, SOD will increase as a form of compensation mechanism. However, in the chronic stage, SOD does not seem to be sufficient and tends to decrease due to the destruction of proteins from the massive reaction of oxidant-antioxidant.[48]

In mice, the extracellular superoxide dismutase (SOD3, ecSOD) contributes to the development of hypertension.[49][50] Diminished SOD3 activity has been linked to lung diseases such as Acute Respiratory Distress Syndrome (ARDS) or Chronic obstructive pulmonary disease (COPD).[51][52][53]

Superoxide dismutase is also not expressed in neural crest cells in the developing fetus. Hence, high levels of free radicals can cause damage to them and induce dysraphic anomalies (neural tube defects).


Pharmacological activity

SOD has powerful antinflammatory activity. For example, SOD is a highly effective experimental treatment of chronic inflammation in colitis. Treatment with SOD decreases reactive oxygen species generation and oxidative stress and, thus, inhibits endothelial activation. Therefore, such antioxidants may be important new therapies for the treatment of inflammatory bowel disease.[54]

Likewise, SOD has multiple pharmacological activities. E.g., it ameliorates cis-platinum-induced nephrotoxicity in rodents.[55] As "Orgotein" or "ontosein", a pharmacologically-active purified bovine liver SOD, it is also effective in the treatment of urinary tract inflammatory disease in man.[56] For a time, bovine liver SOD even had regulatory approval in several European countries for such use. This was cut short by concerns about prion disease.

An SOD-mimetic agent, TEMPOL, is currently in clinical trials for radioprotection and to prevent radiation-induced dermatitis.[57] TEMPOL and similar SOD-mimetic nitroxides exhibit a multiplicity of actions in diseases involving oxidative stress.[58]


Cosmetic uses

SOD may reduce free radical damage to skin—for example, to reduce fibrosis following radiation for breast cancer. Studies of this kind must be regarded as tentative, however, as there were not adequate controls in the study including a lack of randomization, double-blinding, or placebo.[59] Superoxide dismutase is known to reverse fibrosis, possibly through de-differentiation of myofibroblasts back to fibroblasts.[60]


Commercial sources

SOD is commercially obtained from marine phytoplankton, bovine liver, horseradish, cantaloupe, and certain bacteria. For therapeutic purpose, SOD is usually injected locally. There is no evidence that ingestion of unprotected SOD or SOD-rich foods can have any physiological effects, as all ingested SOD is broken down into amino acids before being absorbed. However, ingestion of SOD bound to wheat proteins could improve its therapeutic activity, at least in theory.[61]


See also

-   Catalase
-   Glutathione peroxidase
-   Jiaogulan
-   NADPH oxidase, an enzyme which _produces_ superoxide
-   Peroxidase


References


External links

-   (ALS)

-   The ALS Online Database
-   A short but substantive overview of SOD and its literature.
-   Damage-Based Theories of Aging Includes a discussion of the roles of SOD1 and SOD2 in aging.
-   Physicians' Comm. For Responsible Med.
-   SOD and Oxidative Stress Pathway Image
-   Historical information on SOD research"The evolution of _Free Radical Biology & Medicine_: A 20-year history" and "_Free Radical Biology & Medicine_ The last 20 years: The most highly cited papers"
-   JM McCord discusses the discovery of SOD
-   PDBe-KB provides an overview of all the structure information available in the PDB for Human Superoxide dismutase [Cu-Zn]
-   PDBe-KB provides an overview of all the structure information available in the PDB for Human Superoxide dismutase [Mn], mitochondrial
-   PDBe-KB provides an overview of all the structure information available in the PDB for Human Extracellular superoxide dismutase [Cu-Zn]

Category:Antioxidants Category:Metalloproteins Category:Oxidoreductases Category:EC 1.15.1 Category:Copper enzymes Category:Aging-related enzymes Category:Iron enzymes Category:Zinc enzymes Category:Nickel enzymes Category:Manganese enzymes

[1]

[2]

[3]

[4]

[5] ;

[6] .

[7]

[8] ;

[9]

[10] ;

[11] ;

[12]

[13]

[14]

[15]

[16]

[17] ;

[18] ;

[19] ;

[20]

[21]

[22]

[23]

[24]

[25]

[26]

[27]

[28]

[29]

[30]

[31]

[32]

[33]

[34]

[35]

[36]

[37]

[38]

[39]

[40]

[41]

[42]

[43]

[44]

[45]

[46]

[47]

[48]

[49]

[50]

[51]

[52]

[53]

[54]

[55]

[56]

[57]

[58]

[59]

[60]

[61]